# Killer Tools

## R

* [animation](https://cran.r-project.org/web/packages/animation/index.html): A Gallery of Animations in Statistics and Utilities to Create Animations
* [AnnoProbe](https://github.com/jmzeng1314/AnnoProbe): 表达芯片数据分析伴侣
* [annotater](https://github.com/luisDVA/annotater): The goal of annotater is to annotate package load calls in character strings and R/Rmd files, so we can have an idea of the overall purpose of the libraries we’re loading.
* [AnnotationHub](https://bioconductor.org/packages/release/bioc/html/AnnotationHub.html): Client to access AnnotationHub resources
* [aplot](https://cran.r-project.org/web/packages/aplot/index.html): Decorate a 'ggplot' with Associated Information
* [arules](https://cran.r-project.org/web/packages/arules/index.html): Mining Association Rules and Frequent Itemsets
* [arulesViz](https://cran.r-project.org/web/packages/arulesViz/index.html): Visualizing Association Rules and Frequent Itemsets
* [bbplot](https://github.com/bbc/bbplot): R package that helps create and export ggplot2 charts in the style used by the BBC News data team
* [beepr](https://cran.r-project.org/web/packages/beepr/index.html): Easily Play Notification Sounds on any Platform
* [bestglm](https://cran.r-project.org/web/packages/bestglm/index.html): Best Subset GLM and Regression Utilities
* [bibliometrix](https://cran.r-project.org/web/packages/bibliometrix/index.html): Comprehensive Science Mapping Analysis
* [BiocFileCache](https://bioconductor.org/packages/release/bioc/html/BiocFileCache.html): Manage Files Across Sessions
* [BioInstaller](https://cran.r-project.org/web/packages/BioInstaller/index.html): Integrator of Bioinformatics Resources
* [Biostrings](https://bioconductor.org/packages/release/bioc/html/Biostrings.html): Efficient manipulation of biological strings
* [blastula](https://cran.r-project.org/web/packages/blastula/index.html): Easily Send HTML Email Messages
* [blogdown](https://cran.r-project.org/web/packages/blogdown/index.html): Create Blogs and Websites with R Markdown
* [bookdown](https://cran.rstudio.com/web/packages/bookdown/index.html): Authoring Books and Technical Documents with R Markdown
* [calendR](https://cran.r-project.org/web/packages/calendR/index.html): Ready to Print Monthly and Yearly Calendars Made with 'ggplot2'
* [canvasXpress](https://cran.r-project.org/web/packages/canvasXpress/index.html): Visualization Package for CanvasXpress in R
* [car](https://cran.r-project.org/web/packages/car/index.html): Companion to Applied Regression
* [caret](https://cran.r-project.org/web/packages/caret/index.html): Classification and Regression Training; Misc functions for training and plotting classification and regression models.
* [circlize](https://github.com/jokergoo/circlize): circular visualization in R
* [clusterProfiler](https://bioconductor.org/packages/release/bioc/html/clusterProfiler.html): statistical analysis and visualization of functional profiles for genes and gene clusters
* [ChIPseeker](https://www.bioconductor.org/packages/release/bioc/html/ChIPseeker.html): ChIPseeker for ChIP peak Annotation, Comparison, and Visualization
* [ChromHeatMap](http://bioconductor.org/packages/release/bioc/html/ChromHeatMap.html): Heat map plotting by genome coordinate
* [chromoMap](https://cran.r-project.org/web/packages/chromoMap/index.html): Interactive Visualization and Mapping of Chromosomes
* [cola](https://www.bioconductor.org/packages/release/bioc/html/cola.html): A Framework for Consensus Partitioning
* [colorfindr](https://github.com/zumbov2/colorfindr): Extracts colors from various image types, plots treemaps and 3D scatterplots of color compositions, create color palettes
* [colorfulVennPlot](https://cran.r-project.org/web/packages/colorfulVennPlot/index.html): Plot and add custom coloring to Venn diagrams for 2-dimensional, 3-dimensional and 4-dimensional data
* [ComplexHeatmap](http://bioconductor.org/packages/release/bioc/html/ComplexHeatmap.html): Make Complex Heatmaps
* [ComplexUpset](https://cran.r-project.org/web/packages/ComplexUpset/index.html): Create Complex UpSet Plots Using 'ggplot2' Components
* [concaveman](https://cran.r-project.org/web/packages/concaveman/index.html): A Very Fast 2D Concave Hull Algorithm
* [cowplot](https://cran.r-project.org/web/packages/cowplot/index.html): Streamlined Plot Theme and Plot Annotations for 'ggplot2'
* [crayon](https://cran.r-project.org/web/packages/crayon/index.html): Colored Terminal Output
* [CrossICC](https://www.bioconductor.org/packages/release/bioc/html/CrossICC.html): An Interactive Consensus Clustering Framework for Multi-platform Data Analysis
* [curatedTCGAData](https://bioconductor.org/packages/release/data/experiment/html/curatedTCGAData.html): Curated Data From The Cancer Genome Atlas (TCGA) as MultiAssayExperiment Objects
* [customLayout](https://cran.r-project.org/web/packages/customLayout/index.html): Arrange Elements on the R's Drawing Area or Inside the PowerPoint's Slide
* [data.table](https://cran.r-project.org/web/packages/data.table/index.html): Extension of 'data.frame'
* [dendextend](https://cran.r-project.org/web/packages/dendextend/index.html): Extending 'dendrogram' Functionality in R
* [DescrTab2](https://cran.r-project.org/web/packages/DescrTab2/index.html): Publication Quality Descriptive Statistics Tables
* [DNAshapeR](https://www.bioconductor.org/packages/release/bioc/html/DNAshapeR.html): High-throughput prediction of DNA shape features
* [distill](https://cran.r-project.org/web/packages/distill/index.html): 'R Markdown' Format for Scientific and Technical Writing
* [drake](https://cran.r-project.org/web/packages/drake/index.html): A Pipeline Toolkit for Reproducible Computation at Scale
* [dynamicTreeCut](https://cran.r-project.org/web/packages/dynamicTreeCut/index.html): Methods for Detection of Clusters in Hierarchical Clustering Dendrograms
* [e1071](https://cran.r-project.org/web/packages/e1071/index.html): Functions for latent class analysis, short time Fourier transform, fuzzy clustering, support vector machines, shortest path computation, bagged clustering, naive Bayes classifier, ...
* [easyPubMed](https://cran.r-project.org/web/packages/easyPubMed/index.html): Search and Retrieve Scientific Publication Records from PubMed
* [EasyStat](https://github.com/taowenmicro/EasyStat): 差异分析完整解决方案
* [EBImage](https://bioconductor.org/packages/release/bioc/html/EBImage.html): Image processing and analysis toolbox for R
* [emojifont](https://cran.r-project.org/web/packages/emojifont/index.html): Emoji and Font Awesome in Graphics
* [enrichplot](https://bioconductor.org/packages/release/bioc/html/enrichplot.html): Visualization of Functional Enrichment Result
* [EnhancedVolcano](https://bioconductor.org/packages/release/bioc/html/EnhancedVolcano.html): Publication-ready volcano plots with enhanced colouring and labeling
* [eoffice](https://cran.r-project.org/web/packages/eoffice/index.html): Export or Graph and Tables to 'Microsoft' Office and Import Figures and Tables
* [esquisse](https://cran.r-project.org/web/packages/esquisse/index.html): Explore and Visualize Your Data Interactively
* [export](https://github.com/tomwenseleers/export): easily export active R graphs and statistical output in publication quality to Microsoft Office (Word, PowerPoint and Excel), HTML and Latex
* [extrafont](https://cran.r-project.org/web/packages/extrafont/index.html): Tools for using fonts
* [ExPosition](https://cran.r-project.org/web/packages/ExPosition/index.html): Exploratory Analysis with the Singular Value Decomposition
* [factoextra](https://cran.r-project.org/web/packages/factoextra/index.html): Extract and Visualize the Results of Multivariate Data Analyses
* [FactoMineR](https://cran.r-project.org/web/packages/FactoMineR/index.html): Multivariate Exploratory Data Analysis and Data Mining
* [fgsea](https://bioconductor.org/packages/release/bioc/html/fgsea.html): Fast Gene Set Enrichment Analysis
* [fields](https://mran.microsoft.com/snapshot/2017-02-04/web/packages/fields/index.html): Tools for Spatial Data
* [filehash](https://cran.r-project.org/web/packages/filehash/index.html): Simple Key-Value Database
* [future](https://cran.r-project.org/web/packages/future/index.html): Unified Parallel and Distributed Processing in R for Everyone
* [future.apply](https://cran.r-project.org/web/packages/future.apply/index.html): Apply Function to Elements in Parallel using Futures
* [gapminder](https://cran.r-project.org/web/packages/gapminder/index.html): Data from Gapminder
* [gdata](https://cran.r-project.org/web/packages/gdata/index.html): Various R Programming Tools for Data Manipulation
* [GenomicRanges](https://bioconductor.org/packages/release/bioc/html/GenomicRanges.html): Representation and manipulation of genomic intervals
* [geoR](https://cran.r-project.org/web/packages/geoR/index.html): Analysis of Geostatistical Data
* [GetoptLong](https://cran.r-project.org/web/packages/GetoptLong/index.html): Parsing Command-Line Arguments and Variable Interpolation
* [gg.gap](https://cran.r-project.org/web/packages/gg.gap/index.html): Define Segments in y-Axis for 'ggplot2'
* [ggalluvial](https://cran.r-project.org/web/packages/ggalluvial/index.html): Alluvial Plots in 'ggplot2'
* [GGally](https://cran.r-project.org/web/packages/GGally/index.html): Extension to 'ggplot2'
* [ggalt](https://cran.r-project.org/web/packages/ggalt/index.html): Extra Coordinate Systems, 'Geoms', Statistical Transformations, Scales and Fonts for 'ggplot2'
* [gganatogram](https://github.com/jespermaag/gganatogram): Create anatogram images for different organisms using ggplot2
* [gganimate](https://cran.r-project.org/web/packages/gganimate/index.html): A Grammar of Animated Graphics
* [ggannotate](https://github.com/MattCowgill/ggannotate): Interactively annotate ggplots
* [ggbeeswarm](https://github.com/eclarke/ggbeeswarm): Beeswarm-style plots with ggplot2
* [ggbio](https://bioconductor.org/packages/release/bioc/html/ggbio.html): Visualization tools for genomic data
* [ggchicklet](https://github.com/hrbrmstr/ggchicklet): Create Chicklet (Rounded Segmented Column) Charts
* [ggcor](https://github.com/houyunhuang/ggcor): Extended tools for correlation analysis and visualization
* [ggcorrplot](https://cran.r-project.org/web/packages/ggcorrplot/index.html): Visualization of a Correlation Matrix using 'ggplot2'
* [ggDCA](https://cran.r-project.org/web/packages/ggDCA/index.html): Calculate and Plot Decision Curve
* [ggdendro](https://cran.r-project.org/web/packages/ggdendro/index.html): Create Dendrograms and Tree Diagrams Using 'ggplot2'
* [ggforce](https://cran.r-project.org/web/packages/ggforce/index.html): Accelerating 'ggplot2'
* [ggfortify](https://cran.r-project.org/web/packages/ggfortify/index.html): Data Visualization Tools for Statistical Analysis Results
* [gggenes](https://cran.r-project.org/web/packages/gggenes/index.html): Draw Gene Arrow Maps in 'ggplot2'
* [gggibbous](https://cran.r-project.org/web/packages/gggibbous/index.html): Moon Charts, a Pie Chart Alternative
* [gghalves](https://cran.r-project.org/web/packages/gghalves/index.html): Compose Half-Half Plots Using Your Favourite Geoms
* [ggiraph](https://cran.r-project.org/web/packages/ggiraph/index.html): Make 'ggplot2' Graphics Interactive
* [ggimage](https://cran.r-project.org/web/packages/ggimage/index.html): Use Image in 'ggplot2'
* [ggmap](https://cran.r-project.org/web/packages/ggmap/index.html): Spatial Visualization with ggplot2
* [ggmcmc](https://cran.r-project.org/web/packages/ggmcmc/index.html): Tools for Analyzing MCMC Simulations from Bayesian Inference
* [ggmsa](https://cran.r-project.org/web/packages/ggmsa/index.html): Plot Multiple Sequence Alignment using 'ggplot2'
* [ggnetwork](https://cran.r-project.org/web/packages/ggnetwork/index.html): Geometries to Plot Networks with 'ggplot2'
* [ggord](https://github.com/fawda123/ggord): a take on ordination plots using ggplot2
* [ggparliament](https://github.com/RobWHickman/ggparliament): Simple parliament plots using ggplot2
* [ggplot.multistats](https://cran.r-project.org/web/packages/ggplot.multistats/index.html): Multiple Summary Statistics for Binned Stats/Geometries
* [ggplotify](https://cran.r-project.org/web/packages/ggplotify/index.html): Convert Plot to 'grob' or 'ggplot' Object
* [ggpmisc](https://cran.r-project.org/web/packages/ggpmisc/index.html): Miscellaneous Extensions to 'ggplot2'
* [ggpol](https://cran.r-project.org/web/packages/ggpol/index.html): Visualizing Social Science Data with 'ggplot2'
* [ggpubr](https://cran.r-project.org/web/packages/ggpubr/index.html): 'ggplot2' Based Publication Ready Plots
* [ggradar](https://github.com/ricardo-bion/ggradar): radar charts with ggplot2
* [ggraph](https://cran.r-project.org/web/packages/ggraph/index.html): An Implementation of Grammar of Graphics for Graphs and Networks
* [ggrepel](https://cran.r-project.org/web/packages/ggrepel/index.html): Automatically Position Non-Overlapping Text Labels with 'ggplot2'
* [ggridges](https://cran.r-project.org/web/packages/ggridges/index.html): Ridgeline Plots in 'ggplot2'
* [ggsci](https://cran.r-project.org/web/packages/ggsci/index.html): Scientific Journal and Sci-Fi Themed Color Palettes for 'ggplot2'
* [ggseqlogo](https://cran.r-project.org/web/packages/ggseqlogo/index.html): A 'ggplot2' Extension for Drawing Publication-Ready Sequence Logos
* [ggsn](https://cran.r-project.org/web/packages/ggsn/index.html): North Symbols and Scale Bars for Maps Created with 'ggplot2' or 'ggmap'
* [ggstance](https://cran.r-project.org/web/packages/ggstance/index.html): Horizontal 'ggplot2' Components
* [ggstatsplot](https://cran.r-project.org/web/packages/ggstatsplot/index.html): 'ggplot2' Based Plots with Statistical Details
* [ggsymbol](https://github.com/xiangpin/ggsymbol): symbols layer based on ggplot2 and ggstar
* [ggtext](https://cran.r-project.org/web/packages/ggtext/index.html): Improved Text Rendering Support for 'ggplot2'
* [ggThemeAssist](https://cran.r-project.org/web/packages/ggThemeAssist/index.html): Add-in to Customize 'ggplot2' Themes
* [ggthemes](https://cran.r-project.org/web/packages/ggthemes/index.html): Extra Themes, Scales and Geoms for 'ggplot2'
* [ggTimeSeries](https://cran.r-project.org/web/packages/ggTimeSeries/index.html): Time Series Visualisations Using the Grammar of Graphics
* [ggtree](https://www.bioconductor.org/packages/release/bioc/html/ggtree.html): an R package for visualization of tree and annotation data
* [ggvenn](https://cran.r-project.org/web/packages/ggvenn/index.html): Draw Venn Diagram by 'ggplot2'
* [ggVennDiagram](https://cran.r-project.org/web/packages/ggVennDiagram/index.html): A 'ggplot2' Implement of Venn Diagram
* [ggvwline](https://github.com/houyunhuang/ggvwline): Extension to ggplot2 for drawing variable-width lines
* [ggwaffle](https://github.com/liamgilbey/ggwaffle): Creating waffle charts in a ggplot friendly way
* [glmSparseNet](https://www.bioconductor.org/packages/release/bioc/html/glmSparseNet.html): Network Centrality Metrics for Elastic-Net Regularized Models
* [glue](https://cran.r-project.org/web/packages/glue/index.html): Interpreted String Literals
* [GOplot](http://wencke.github.io/): Visualization of Functional Analysis Data
* [gplots](https://cran.r-project.org/web/packages/gplots/index.html): Various R Programming Tools for Plotting Data
* [grDevices](https://www.rdocumentation.org/packages/grDevices/versions/3.6.2): The R Graphics Devices and Support for Colours and Fonts
* [gtools](https://cran.r-project.org/web/packages/gtools/index.html): Various R Programming Tools
* [hchinamap](https://cran.r-project.org/web/packages/hchinamap/index.html): Mapping China and Its Provinces
* [here](https://cran.r-project.org/web/packages/here/index.html): A Simpler Way to Find Your Files
* [HilbertCurve](https://www.bioconductor.org/packages/release/bioc/html/HilbertCurve.html): Making 2D Hilbert Curve
* [Hmisc](https://cran.r-project.org/web/packages/Hmisc/index.html): Harrell Miscellaneous
* [hrbrthemes](https://cran.r-project.org/web/packages/hrbrthemes/index.html): Additional Themes, Theme Components and Utilities for 'ggplot2'
* [httr](https://cran.r-project.org/web/packages/httr/index.html): Tools for Working with URLs and HTTP
* [igraph](https://cran.r-project.org/web/packages/igraph/index.html): Network Analysis and Visualization
* [iheatmapr](https://cran.r-project.org/web/packages/iheatmapr/index.html): Interactive, Complex Heatmaps
* [infer](https://cran.r-project.org/web/packages/infer/index.html): Tidy Statistical Inference
* [kableExtra](https://cran.r-project.org/web/packages/kableExtra/index.html): Construct Complex Table with 'kable' and Pipe Syntax
* [karyoploteR](https://bioconductor.org/packages/release/bioc/html/karyoploteR.html): Plot customizable linear genomes displaying arbitrary data
* [latex2exp](https://cran.r-project.org/web/packages/latex2exp/index.html): Use LaTeX Expressions in Plots
* [learnr](https://cran.r-project.org/web/packages/learnr/index.html): Interactive Tutorials for R
* [legendMap](https://github.com/3wen/legendMap): Provides functions to add a North arrow and/or a scale bar to maps done using ggplot2
* [magick](https://cran.r-project.org/web/packages/magick/index.html): Advanced Graphics and Image-Processing in R
* [Map2NCBI](https://cran.r-project.org/web/packages/Map2NCBI/index.html): Mapping Markers to the Nearest Genomic Feature
* [maps](https://cran.r-project.org/web/packages/maps/index.html): Draw Geographical Maps
* [maptools](https://cran.r-project.org/web/packages/maptools/index.html): Tools for Handling Spatial Objects
* [Mfuzz](https://www.bioconductor.org/packages/release/bioc/html/Mfuzz.html): Soft clustering of time series gene expression data
* [mice](https://cran.r-project.org/web/packages/mice/index.html): Multivariate Imputation by Chained Equations
* [mindr](https://cran.r-project.org/web/packages/mindr/index.html): Convert Files Between Markdown or R Markdown Files and Mind Maps
* [mixOmics](http://www.bioconductor.org/packages/release/bioc/html/mixOmics.html): Omics Data Integration Project
* [mlr](https://cran.r-project.org/web/packages/mlr/index.html): Machine Learning in R
* [moderndive](https://cran.r-project.org/web/packages/moderndive/index.html): Tidyverse-Friendly Introductory Linear Regression
* [msa](https://bioconductor.org/packages/release/bioc/html/msa.html): Multiple Sequence Alignment
* [msigdbr](https://cran.r-project.org/web/packages/msigdbr/index.html): MSigDB Gene Sets for Multiple Organisms in a Tidy Data Format
* [multcomp](https://cran.r-project.org/web/packages/multcomp/index.html): Simultaneous Inference in General Parametric Models
* [multiMiR](https://bioconductor.org/packages/release/bioc/html/multiMiR.html): Integration of multiple microRNA-target databases with their disease and drug associations
* [network](https://cran.r-project.org/web/packages/network/index.html): Classes for Relational Data
* [networkD3](https://cran.r-project.org/web/packages/networkD3/index.html): D3 JavaScript Network Graphs from R
* [NMF](https://cran.r-project.org/web/packages/NMF/index.html): Algorithms and Framework for Nonnegative Matrix Factorization (NMF)
* [ontologyIndex](https://cran.r-project.org/web/packages/ontologyIndex/index.html): Functions for Reading Ontologies into R
* [optparse](https://cran.r-project.org/web/packages/optparse/index.html): Command Line Option Parser
* [pacman](https://cran.r-project.org/web/packages/pacman/index.html): Package Management Tool
* [paletteer](https://cran.r-project.org/web/packages/paletteer/index.html): Comprehensive Collection of Color Palettes
* [patchwork](https://cran.r-project.org/web/packages/patchwork/index.html): The Composer of ggplots
* [pathview](https://www.bioconductor.org/packages/release/bioc/html/pathview.html): a tool set for pathway based data integration and visualization
* [patternplot](https://cran.r-project.org/web/packages/patternplot/index.html): Versatile Pie Charts, Ring Charts, Bar Charts and Box Plots using Patterns, Colors and Images
* [pdftools](https://cran.r-project.org/web/packages/pdftools/index.html): Text Extraction, Rendering and Converting of PDF Documents
* [phytools](https://cran.r-project.org/web/packages/phytools/index.html): Phylogenetic Tools for Comparative Biology (and Other Things)
* [Pi](https://www.bioconductor.org/packages/release/bioc/html/Pi.html): Leveraging Genetic Evidence to Prioritise Drug Targets at the Gene and Pathway Level
* [pkgdown](https://cran.r-project.org/web/packages/pkgdown/index.html): Make Static HTML Documentation for a Package
* [precrec](https://cran.r-project.org/web/packages/precrec/index.html): Calculate Accurate Precision-Recall and ROC (Receiver Operator Characteristics) Curves
* [prettydoc](https://cran.r-project.org/web/packages/prettydoc/index.html): Creating Pretty Documents from R Markdown
* [pROC](https://cran.r-project.org/web/packages/pROC/index.html): Display and Analyze ROC Curves
* [qgraph](https://cran.r-project.org/web/packages/qgraph/index.html): Graph Plotting Methods, Psychometric Data Visualization and Graphical Model Estimation
* [qqman](https://cran.r-project.org/web/packages/qqman/index.html): Q-Q and Manhattan Plots for GWAS Data
* [qvalue](http://www.bioconductor.org/packages/release/bioc/html/qvalue.html): Q-value estimation for false discovery rate control
* [R.utils](https://cran.r-project.org/web/packages/R.utils/index.html): Various Programming Utilities
* [randomcoloR](https://cran.r-project.org/web/packages/randomcoloR/index.html): Generate Attractive Random Colors
* [randomForest](https://cran.r-project.org/web/packages/randomForest/index.html): Breiman and Cutler's Random Forests for Classification and Regression
* [rattle](https://cran.r-project.org/web/packages/rattle/index.html): Graphical User Interface for Data Science in R
* [rayshader](https://cran.r-project.org/web/packages/rayshader/index.html): Create Maps and Visualize Data in 2D and 3D
* [RCircos](https://cran.r-project.org/web/packages/RCircos/index.html): Circos 2D Track Plot
* [RColorBrewer](https://cran.r-project.org/web/packages/RColorBrewer/index.html): ColorBrewer Palettes
* [remotes](https://cran.r-project.org/web/packages/remotes/index.html): R Package Installation from Remote Repositories, Including 'GitHub'
* [reprex](https://cran.r-project.org/web/packages/reprex/index.html): Prepare Reproducible Example Code via the Clipboard
* [rgdal](https://cran.r-project.org/web/packages/rgdal/index.html): Bindings for the 'Geospatial' Data Abstraction Library
* [rgl](https://cran.r-project.org/web/packages/rgl/index.html): 3D Visualization Using OpenGL
* [rhub](https://github.com/r-hub/rhub): a multi-platform build and check service for R packages
* [rio](https://cran.r-project.org/web/packages/rio/index.html): A Swiss-Army Knife for Data I/O
* [riskRegression](https://cran.r-project.org/web/packages/riskRegression/index.html): Risk Regression Models and Prediction Scores for Survival Analysis with Competing Risks
* [RIdeogram](https://cran.r-project.org/web/packages/RIdeogram/index.html): Drawing SVG Graphics to Visualize and Map Genome-Wide Data on Idiograms
* [riverplot](https://cran.r-project.org/web/packages/riverplot/index.html): Sankey or Ribbon Plots
* [ROCit](https://cran.r-project.org/web/packages/ROCit/index.html): Performance Assessment of Binary Classifier with Visualization
* [ROCR](https://cran.r-project.org/web/packages/ROCR/index.html): Visualizing the Performance of Scoring Classifiers
* [ropls](https://bioconductor.org/packages/release/bioc/html/ropls.html): PCA, PLS(-DA) and OPLS(-DA) for multivariate analysis and feature selection of omics data
* [rpivotTable](https://cran.r-project.org/web/packages/rpivotTable/index.html): Build Powerful Pivot Tables and Dynamically Slice & Dice your Data
* [RSelenium](https://cran.r-project.org/web/packages/RSelenium/index.html): R Bindings for 'Selenium WebDriver'
* [rticles](https://cran.r-project.org/web/packages/rticles/index.html): Article Formats for R Markdown
* [rtracklayer](https://bioconductor.org/packages/release/bioc/html/rtracklayer.html): R interface to genome annotation files and the UCSC genome browser
* [RVA](https://cran.r-project.org/web/packages/RVA/index.html): RNAseq Visualization Automation
* [rvcheck](https://cran.rstudio.com/web/packages/rvcheck/index.html): R/Package Version Check, Check latest release version of R and R package (both in 'CRAN', 'Bioconductor' or 'Github')
* [rvest](https://cran.r-project.org/web/packages/rvest/index.html): Easily Harvest (Scrape) Web Pages
* [sankeywheel](https://cran.r-project.org/web/packages/sankeywheel/index.html): Create Dependency Wheels and Sankey Diagrams
* [scales](https://cran.r-project.org/web/packages/scales/index.html): Scale Functions for Visualization
* [scatterpie](https://cran.r-project.org/web/packages/scatterpie/index.html): Scatter Pie Plot
* [scholar](https://cran.r-project.org/web/packages/scholar/index.html): Analyse Citation Data from Google Scholar
* [seqinr](https://cran.r-project.org/web/packages/seqinr/index.html): Biological Sequences Retrieval and Analysis
* [sf](https://cran.r-project.org/web/packages/sf/index.html): Simple Features for R
* [shadowtext](https://cran.r-project.org/web/packages/shadowtext/index.html): Shadow Text Grob and Layer
* [shiny](https://cran.r-project.org/web/packages/shiny/index.html): Web Application Framework for R
* [showtext](https://cran.r-project.org/web/packages/showtext/index.html): Using Fonts More Easily in R Graphs
* [simplifyEnrichment](https://bioconductor.org/packages/release/bioc/html/simplifyEnrichment.html): Simplify Functional Enrichment Results
* [SnowballC](https://cran.r-project.org/web/packages/SnowballC/index.html): Snowball Stemmers Based on the C 'libstemmer' UTF-8 Library
* [sp](https://cran.r-project.org/web/packages/sp/index.html): Classes and Methods for Spatial Data
* [spatstat](https://cran.r-project.org/web/packages/spatstat/index.html): Spatial Point Pattern Analysis, Model-Fitting, Simulation, Tests
* [splines2](https://cran.r-project.org/web/packages/splines2/index.html): Regression Spline Functions and Classes
* [stm](https://cran.r-project.org/web/packages/stm/index.html): Estimation of the Structural Topic Model
* [STRINGdb](https://www.bioconductor.org/packages/release/bioc/html/STRINGdb.html): Search Tool for the Retrieval of Interacting proteins database
* [survival](https://cran.r-project.org/web/packages/survival/index.html): Survival Analysis
* [survivalAnalysis](https://cran.r-project.org/web/packages/survivalAnalysis/index.html): High-Level Interface for Survival Analysis and Associated Plots
* [survminer](https://cran.r-project.org/web/packages/survminer/index.html): Drawing Survival Curves using 'ggplot2'
* [survRM2](https://cran.r-project.org/web/packages/survRM2/index.html): Comparing Restricted Mean Survival Time
* [Sushi](https://bioconductor.org/packages/release/bioc/html/Sushi.html): Tools for visualizing genomics data
* [systemPipeR](https://bioconductor.org/packages/release/bioc/html/systemPipeR.html): NGS workflow and report generation environment
* [TCGAbiolinks](https://bioconductor.org/packages/release/bioc/html/TCGAbiolinks.html): An R/Bioconductor package for integrative analysis with GDC data
* [TCGAutils](https://bioconductor.org/packages/release/bioc/html/TCGAutils.html): A suite of helper functions for checking and manipulating TCGA data including data obtained from the curatedTCGAData experiment package. These functions aim to simplify and make working with TCGA data more manageable.
* [textir](https://cran.r-project.org/web/packages/textir/index.html): Inverse Regression for Text Analysis
* [tidyfst](https://cran.r-project.org/web/packages/tidyfst/index.html): Tidy Verbs for Fast Data Manipulation
* [tidymodels](https://cran.r-project.org/web/packages/tidymodels/index.html): Easily Install and Load the 'Tidymodels' Packages
* [tidytext](https://cran.r-project.org/web/packages/tidytext/index.html): Text Mining using 'dplyr', 'ggplot2', and Other Tidy Tools
* [tidytidbits](https://cran.r-project.org/web/packages/tidytidbits/index.html): A Collection of Tools and Helpers Extending the Tidyverse
* [tidyverse](https://cran.r-project.org/web/packages/tidyverse/index.html): Easily Install and Load the 'Tidyverse'
* [tm](https://cran.r-project.org/web/packages/tm/index.html): Text Mining Package
* [tmap](https://cran.r-project.org/web/packages/tmap/index.html): Thematic Maps
* [treemapify](https://cran.r-project.org/web/packages/treemapify/index.html): Draw Treemaps in 'ggplot2'
* [TSstudio](https://cran.r-project.org/web/packages/TSstudio/index.html): Functions for Time Series Analysis and Forecasting
* [TwoSampleMR](https://github.com/MRCIEU/TwoSampleMR): Mendelian randomization with GWAS summary data
* [UCSCXenaTools](https://cran.r-project.org/web/packages/UCSCXenaTools/index.html): Download and Explore Datasets from UCSC Xena Data Hubs
* [UpSetR](https://cran.r-project.org/web/packages/UpSetR/index.html): A More Scalable Alternative to Venn and Euler Diagrams for Visualizing Intersecting Sets
* [venn](https://cran.r-project.org/web/packages/venn/index.html): Draw Venn Diagrams
* [VennDiagram](https://cran.r-project.org/web/packages/VennDiagram/index.html): Generate High-Resolution Venn and Euler Plots
* [VIM](https://cran.r-project.org/web/packages/VIM/index.html): Visualization and Imputation of Missing Values
* [viridis](https://cran.r-project.org/web/packages/viridis/index.html): Default Color Maps from 'matplotlib'
* [visNetwork](https://cran.r-project.org/web/packages/visNetwork/index.html): Network Visualization using 'vis.js' Library
* [visreg](https://pbreheny.github.io/visreg/): An R package for the visualization of regression models
* [vroom](https://cran.r-project.org/web/packages/vroom/index.html): Read and Write Rectangular Text Data Quickly
* [wget](https://github.com/YuLab-SMU/wget): Setting Download Method to "wget -c"
* [wordcloud](https://cran.r-project.org/web/packages/wordcloud/index.html): Word Clouds
* [workflowr](https://cran.r-project.org/web/packages/workflowr/index.html): A Framework for Reproducible and Collaborative Data Science
* [xaringan](https://cran.r-project.org/web/packages/xaringan/index.html): Presentation Ninja
* []()

## TeX
* [texshade](https://ctan.org/pkg/texshade): Package for setting nucleotide and peptide alignments

## Perl
* [Data::Dumper](https://metacpan.org/pod/Data::Dumper): stringified perl data structures, suitable for both printing and eval
* [Data::Printer](https://metacpan.org/pod/Data::Printer): colored pretty-print of Perl data structures and objects
* [Getopt::Long](https://metacpan.org/pod/Getopt::Long): Extended processing of command line options
* [List::AllUtils](https://metacpan.org/pod/List::AllUtils): Combines List::Util, List::SomeUtils and List::UtilsBy in one bite-sized package
* [List::MoreUtils](https://metacpan.org/pod/List::MoreUtils): Provide the stuff missing in List::Util 
* [List::Util](https://metacpan.org/pod/List::Util): A selection of general-utility list subroutines
* [List::UtilsBy](https://metacpan.org/pod/List::UtilsBy): higher-order list utility functions 
* [Parallel::ForkManager](https://metacpan.org/pod/Parallel::ForkManager): A simple parallel processing fork manager
* [Path::Tiny](https://metacpan.org/pod/Path::Tiny): File path utility
* [Try::Tiny](https://metacpan.org/pod/Try::Tiny): Minimal try/catch with proper preservation of $@




## Tools
* [bioawk](https://github.com/lh3/bioawk): BWK awk modified for biological data
* [GraPhlAn](https://bitbucket.org/nsegata/graphlan/wiki/Home): a software tool for producing high-quality circular representations of taxonomic and phylogenetic trees
* [IQ-TREE](http://www.iqtree.org/): Efficient software for phylogenomic inference
* [Recharts](http://recharts.org/en-US/): A composable charting library built on React components
* [pyGenomeTracks](https://github.com/deeptools/pyGenomeTracks): Standalone program and library to plot beautiful genome browser tracks
* [miRWalk](http://mirwalk.umm.uni-heidelberg.de/)
* [miRSystem](http://mirsystem.cgm.ntu.edu.tw/): miRNA integration system for Target gene prediction
* [Heatmapper](http://www.heatmapper.ca/expression/): a freely available web server that allows users to interactively visualize their data in the form of heat maps through an easy-to-use graphical interface
* [jvenn](http://jvenn.toulouse.inra.fr/app/index.html): an interactive Venn diagram viewer
* [webtools-Venn](http://bioinformatics.psb.ugent.be/webtools/Venn/): draw venn diagram
* [EVenn](http://www.ehbio.com/test/venn/#/): Easy to create repeatable, editable, and statistically measurable Venn diagrams online.
* [NetworkAnalyst](https://www.networkanalyst.ca/): comprehensive gene expression analysis, meta-analysis & network biology
* [BioJupies](https://amp.pharm.mssm.edu/biojupies/): Automatically Generates RNA-seq Data Analysis Notebooks
* [bioDBnet](https://biodbnet-abcc.ncifcrf.gov/): biological DataBase network

## scRNA
* [scRNA-tools](https://www.scrna-tools.org/table)
* [scater](https://bioconductor.org/packages/release/bioc/html/scater.html): Single-Cell Analysis Toolkit for Gene Expression Data in R
* [Seurat](https://cran.r-project.org/web/packages/Seurat/index.html): Tools for Single Cell Genomics
* [monocle](https://www.bioconductor.org/packages/release/bioc/html/monocle.html): Clustering, differential expression, and trajectory analysis for single- cell RNA-Seq
* [cellassign](https://github.com/irrationone/cellassign): Automated, probabilistic assignment of cell types in scRNA-seq data
* [DRAGON](https://github.com/arc85/dragonsc): Deterministic annealing gaussian mixtures models for single cell RNAseq
* [singleseqgset](https://github.com/arc85/singleseqgset): Gene set enrichment analysis for single-cell RNAseq data
* [celltalker](https://github.com/arc85/celltalker): Inference of cell-cell communication from single-cell RNAseq data
* [DPT](https://bioinformaticshome.com/tools/rna-seq/descriptions/DPT.html): estimate the temporal order of differentiating cell in single-cell RNA-seq (scRNA-seq) data
* [diffusionMap](https://github.com/rcannood/diffusionMap)
* [destiny](http://www.bioconductor.org/packages/release/bioc/html/destiny.html): Creates diffusion maps
* [iCellR](https://cran.r-project.org/web/packages/iCellR/index.html): Analyzing High-Throughput Single Cell Sequencing Data
* [presto](https://github.com/immunogenomics/presto): Presto performs a fast Wilcoxon rank sum test and auROC analysis
* [harmony](https://github.com/immunogenomics/harmony): Fast, sensitive and accurate integration of single-cell data with Harmony
* [ESTIMATE](https://bioinformatics.mdanderson.org/public-software/estimate/): Estimation of STromal and Immune cells in MAlignant Tumor tissues using Expression data
* [xCell](https://github.com/dviraran/xCell): cell types enrichment analysis
* [SingleR](https://bioconductor.org/packages/release/bioc/html/SingleR.html): Reference-Based Single-Cell RNA-Seq Annotation
* [CIBERSORTx]( impute gene expression profiles and provide an estimation of the abundances of member cell types in a mixed cell population, using gene expression data)
* [CellPhoneDB](https://github.com/Teichlab/cellphonedb): CellPhoneDB can be used to search for a particular ligand/receptor, or interrogate your own single-cell transcriptomics data.
* [clustermole](https://cran.r-project.org/web/packages/clustermole/index.html): Unbiased Single-Cell Transcriptomic Data Cell Type Identification
* [CellChat](https://github.com/sqjin/CellChat/): Inference and analysis of cell-cell communication
* [iTALK](https://github.com/Coolgenome/iTALK): is an R toolkit for characterizing and illustrating intercellular communication
* [SciBet](http://scibet.cancer-pku.cn/): Single Cell Identificator Based on E-test (SciBet) is a computational tool for predicting cell identity for any randomly sequenced cell by single cell RNA sequencing technique
* [CHETAH](https://github.com/jdekanter/CHETAH): a selective, hierarchical cell type identification method for single-cell RNA sequencing
* [dyno](https://github.com/dynverse/dyno): Inferring trajectories with 60 TI methods
* [Scanpy](https://github.com/theislab/scanpy): Single-Cell Analysis in Python
* [singleseqgset](https://github.com/arc85/singleseqgset): package for gene set enrichment analysis for single-cell RNAseq data
* [clustree](https://cran.r-project.org/web/packages/clustree/index.html): Visualise Clusterings at Different Resolutions
* [batchelor](https://bioconductor.org/packages/release/bioc/html/batchelor.html): Single-Cell Batch Correction Methods
* [presto](https://github.com/immunogenomics/presto): Presto performs a fast Wilcoxon rank sum test and auROC analysis. Latest benchmark ran 1 million observations, 1K features, and 10 groups in 16 seconds (sparse input) and 85 seconds (dense input).
* [SoupX](https://cran.r-project.org/web/packages/SoupX/index.html): Single Cell mRNA Soup eXterminator
* [liger](https://github.com/MacoskoLab/liger): Linked Inference of Genomic Experimental Relationships, LIGER (`liger`) is a package for integrating and analyzing multiple single-cell datasets
* [RcisTarget](https://www.bioconductor.org/packages/release/bioc/html/RcisTarget.html): Identify transcription factor binding motifs enriched on a gene list
* [AUCell](http://bioconductor.org/packages/release/bioc/html/AUCell.html): Analysis of 'gene set' activity in single-cell RNA-seq data (e.g. identify cells with specific gene signatures)
* [Giotto](http://spatialgiotto.rc.fas.harvard.edu/): The Giotto package consists of two modules, Giotto Analyzer and Viewer, which provide tools to process, analyze and visualize single-cell spatial expression data. https://rubd.github.io/Giotto_site/index.html
* [BayesSpace](https://bioconductor.org/packages/release/bioc/html/BayesSpace.html): Clustering and Resolution Enhancement of Spatial Transcriptomes
* [spdep](https://cran.r-project.org/web/packages/spdep/index.html): Spatial Dependence: Weighting Schemes, Statistics

## [Cancer/Survival-related, 生存分析在线工具 2020 年 4 月吐血整理！](https://mp.weixin.qq.com/s?__biz=Mzg4NjA5Mzg2Mw==&mid=2247488689&idx=2&sn=92818d38ffba4f97fd295dcf51f05857&chksm=cf9fb5f9f8e83cefc35b6a980c125fe8be1b5860f66211594cb27d7419be02f5adcb95588c56&mpshare=1&scene=1&srcid=&sharer_sharetime=1587258755943&sharer_shareid=49bb68e4d4ad9f65af077f4e54025da0&key=7024fc3958d21a49f38d5962a504384cb7aa02cfec7d99b587fbbaad39d6f014de6891e0d0d0a1c51311259cc5116c5fda4311d0c4a8bf171a88b52cae8649f6495594a0becdfce753717e6f209ed145&ascene=1&uin=MjEyMzUzNDk2MQ%3D%3D&devicetype=Windows+XP&version=62060841&lang=zh_CN&exportkey=Af7QoGlOZcxdXwTntJ6mQfA%3D&pass_ticket=egSQZf3ZmjmNSBYxFfxeMhxks6Fcdfesee2EK45qb6wqbW1FRDSL1ND6vhbrpF2H)
* [Kaplan Meier-plotter](http://kmplot.com/analysis/index.php)
* [PROGgene](http://genomics.jefferson.edu/proggene/)
* [SurvExpress](http://bioinformatica.mty.itesm.mx:8080/Biomatec/SurvivaX.jsp)
* [KM-express](http://ec2-52-201-246-161.compute-1.amazonaws.com/kmexpress/index.php)
* [PrognoScan](http://dna00.bio.kyutech.ac.jp/PrognoScan/index.html)
* [lnCAR: lncRNA Explorer](https://lncar.renlab.org/)
* [UALCAN](http://ualcan.path.uab.edu/analysis.html)
* [OncoLnc](http://www.oncolnc.org/)
* [OncomiR](http://www.oncomir.org)
* [MethSurv](https://biit.cs.ut.ee/methsurv/)
* [BloodSpot](http://servers.binf.ku.dk/bloodspot/)
* [GEPIA 2](http://gepia2.cancer-pku.cn/#survival)
* [GenomicScape](http://www.genomicscape.com/microarray/survival.php)
* [ExSurv](https://exsurv.soic.iupui.edu/)
* [LOGpc](http://bioinfo.henu.edu.cn/DatabaseList.jsp)
* [CBioportal](https://www.cbioportal.org/)
* [UCSC Xena](https://xena.ucsc.edu/)
* [GDC portal](https://portal.gdc.cancer.gov/)
* [SurvMicro](http://bioinformatica.mty.itesm.mx:8080/Biomatec/Survmicro.jsp)
* [DriverDBv3](http://driverdb.tms.cmu.edu.tw/)
* [LncACTdb 2.0](http://www.bio-bigdata.net/LncACTdb/)
* [TCGAportal](http://tumorsurvival.org/index.html)

# Long read sequencing
* [LRS-DB](https://long-read-tools.org/): A catalogue of long read sequencing data analysis tools

# Scholar search
* [虫部落搜索](https://scholar.chongbuluo.com/): 集合了市面上最为常见的学术搜索引擎，让你一站式搜到底
* [Open-i](https://openi.nlm.nih.gov): 美国国立卫生研究院开发的科研图片数据库，生物和医学类首推的图片搜索网站
* [Viziometrics](http://viziometrics.org/search/): The First Visual Search Engine for Scientific Diagrams
* [CNKI 学术图片库](http://image.cnki.net/)
* [rcrossref](https://cran.r-project.org/web/packages/rcrossref/index.html): Client for Various 'CrossRef' 'APIs'
* [crminer](https://cran.r-project.org/web/packages/crminer/index.html): Fetch 'Scholary' Full Text from 'Crossref'

# Chrome
* [谷粒 - Chrome 插件英雄榜](https://github.com/zhaoolee/ChromeAppHeroes): 为优秀的 Chrome 插件写一本中文说明书，让 Chrome 插件英雄们造福人类

# Shell

* [starship](https://starship.rs/): The minimal, blazing-fast, and infinitely customizable prompt for any shell!



# Others

* [E Venn](http://www.ehbio.com/test/venn/#/)

# Reproducibility

* [构建可重复的单细胞数据分析流程](https://www.jianshu.com/p/675335f3fe4a)
* [Reproducibility in Science](http://ropensci.github.io/reproducibility-guide/): A Guide to enhancing reproducibility in scientific results and writing
* [Top ten reasons to not share your code (and why you should anyway)](https://www.software.ac.uk/blog/2016-10-07-top-ten-reasons-not-share-your-code-and-why-you-should-anyway)
* [Does your code stand up to scrutiny?](https://www.nature.com/articles/d41586-018-02741-4)
* [Excuse me, do you have a moment to talk about version control?](https://peerj.com/preprints/3159v2/)
* [[As a reviewer, can I justify requesting data and code be made available even if the journal does not?](https://stats.stackexchange.com/questions/14416/as-a-reviewer-can-i-justify-requesting-data-and-code-be-made-available-even-if)]