# Killer Tools

## Color in R

* [**R Color Palettes** [497 continuous and discrete palettes] | R CHARTS](https://r-charts.com/color-palettes/)
* [**R ËØ≠Ë®Ä ggplot2 ‰ΩúÂõæÂ•ΩÁúãÁöÑÈÖçËâ≤Â§áÈÄâ**](https://mp.weixin.qq.com/s/vVRFxvjJZubO5VykxYSxPA)
* [cartography](https://cran.r-project.org/web/packages/cartography/index.html): Thematic Cartography
* [colorBlindness](https://cran.r-project.org/web/packages/colorBlindness/index.html): Safe Color Set for Color Blindness
* [dichromat](https://cran.r-project.org/web/packages/dichromat/index.html): Color Schemes for Dichromats
* [ggsci](https://cran.r-project.org/web/packages/ggsci/index.html): Scientific Journal and Sci-Fi Themed Color Palettes for 'ggplot2'
* [ggthemes](https://cran.r-project.org/web/packages/ggthemes/index.html): Extra Themes, Scales and Geoms for 'ggplot2'
* [grDevices-package](https://rdrr.io/r/grDevices/grDevices-package.html): The R Graphics Devices and Support for Colours and Fonts
* [RColorBrewer](https://cran.r-project.org/web/packages/RColorBrewer/index.html): ColorBrewer Palettes
* [tidyquant](https://cran.r-project.org/web/packages/tidyquant/index.html): Tidy Quantitative Financial Analysis
* [tvthemes](https://cran.r-project.org/web/packages/tvthemes/index.html): TV Show Themes and Color Palettes for 'ggplot2' Graphics
* [vapoRwave: ](https://github.com/moldach/vapoRwave)üìºüëæüïπVaporwave themes and color palettes for ggplot2üíæüë®‚Äçüé§üì∫
* [viridis](https://cran.r-project.org/web/packages/viridis/index.html): Colorblind-Friendly Color Maps for R
* [wesanderson](https://cran.r-project.org/web/packages/wesanderson/index.html): A Wes Anderson Palette Generator
* [Colorblind checks for qualitative palettes](https://emitanaka.org/posts/2022-02-20-color-considerations/)


## R packages

* [**Awesome R**](https://github.com/qinwf/awesome-R): **A curated list of awesome R packages and tools.**
* [**15 Essential packages in R for Data Science**](https://finnstats.com/index.php/2021/04/07/essential-packages-in-r/)
* [**awesome-ggplot2**](https://github.com/erikgahner/awesome-ggplot2): **A curated list of awesome ggplot2 tutorials, packages etc.**
* [**CRAN Task Views**](https://cran.r-project.org/web/views/)
* [**addinslist**](https://github.com/daattali/addinslist): **Discover and install useful RStudio addins**
* [**RStudio Package Manager**](https://packagemanager.rstudio.com/client/#/)

### A

* [A3](https://cran.r-project.org/web/packages/A3/index.html): Accurate, Adaptable, and Accessible Error Metrics for Predictive Models
* [abbrevr](https://github.com/patrickbarks/abbrevr): Abbreviate journal titles using the ISO 4 standard
* [ActivePathways](https://cran.r-project.org/web/packages/ActivePathways/index.html): Integrative Pathway Enrichment Analysis of Multivariate Omics Data
* [adegenet](https://cran.r-project.org/web/packages/adegenet/index.html): Exploratory Analysis of Genetic and Genomic Data
* [adjustedCurves](https://cran.r-project.org/web/packages/adjustedCurves/index.html): Confounder-Adjusted Survival Curves and Cumulative Incidence Functions
* [animate](https://cran.rstudio.com/web/packages/animate/index.html): A Web-Based Graphics Device for Animated Visualisations
* [animation](https://cran.r-project.org/web/packages/animation/index.html): A Gallery of Animations in Statistics and Utilities to Create Animations
* [AnnoProbe](https://github.com/jmzeng1314/AnnoProbe): Ë°®ËææËäØÁâáÊï∞ÊçÆÂàÜÊûê‰º¥‰æ£
* [annotater](https://github.com/luisDVA/annotater): The goal of annotater is to annotate package load calls in character strings and R/Rmd files, so we can have an idea of the overall purpose of the libraries we‚Äôre loading.
* [AnnotationHub](https://bioconductor.org/packages/release/bioc/html/AnnotationHub.html): Client to access AnnotationHub resources
* [aPEAR](https://cran.r-project.org/web/packages/aPEAR/index.html): Advanced Pathway Enrichment Analysis Representation
* [apcluster](https://cran.r-project.org/web/packages/apcluster/index.html): Affinity Propagation Clusterin
* [aPCoA](https://cran.r-project.org/web/packages/aPCoA/index.html): Covariate Adjusted PCoA Plot
* [aplot](https://cran.r-project.org/web/packages/aplot/index.html): Decorate a 'ggplot' with Associated Information
* [archive](https://cran.r-project.org/web/packages/archive/index.html): Multi-Format Archive and Compression Support
* [arules](https://cran.r-project.org/web/packages/arules/index.html): Mining Association Rules and Frequent Itemsets
* [arulesViz](https://cran.r-project.org/web/packages/arulesViz/index.html): Visualizing Association Rules and Frequent Itemsets
* [autoGO](https://cran.r-project.org/web/packages/autoGO/index.html): Reproducible, Robust and High Quality Ontology Enrichment Visualizations
* [autoplotly](https://cran.r-project.org/web/packages/autoplotly/index.html): Automatic Generation of Interactive Visualizations for Statistical Results
* [autoReg](https://cran.r-project.org/web/packages/autoReg/index.html): Automatic Linear and Logistic Regression and Survival Analysis

### B

* [babelgene](https://cran.rstudio.com/web/packages/babelgene/index.html): Gene Orthologs for Model Organisms in a Tidy Data Format
* [barplot3d](https://cran.r-project.org/web/packages/barplot3d/index.html): Create 3D Barplots
* [bayesplot](https://cran.r-project.org/web/packages/bayesplot/index.html): Plotting for Bayesian Models
* [bayestestR](https://cran.r-project.org/web/packages/bayestestR/index.html): Understand and Describe Bayesian Models and Posterior Distributions
* [bbplot](https://github.com/bbc/bbplot): R package that helps create and export ggplot2 charts in the style used by the BBC News data team
* [beepr](https://cran.r-project.org/web/packages/beepr/index.html): Easily Play Notification Sounds on any Platform
* [bestNormalize](https://cran.r-project.org/web/packages/bestNormalize/index.html): Normalizing Transformation Functions
* [bestglm](https://cran.r-project.org/web/packages/bestglm/index.html): Best Subset GLM and Regression Utilities
* [bibliometrix](https://cran.r-project.org/web/packages/bibliometrix/index.html): Comprehensive Science Mapping Analysis
* [bigstatsr](https://cran.r-project.org/web/packages/bigstatsr/index.html): Statistical Tools for Filebacked Big Matrices
* [BiocFileCache](https://bioconductor.org/packages/release/bioc/html/BiocFileCache.html): Manage Files Across Sessions
* [BioInstaller](https://cran.r-project.org/web/packages/BioInstaller/index.html): Integrator of Bioinformatics Resources
* [BiocParallel](https://bioconductor.org/packages/release/bioc/html/BiocParallel.html): Bioconductor facilities for parallel evaluation
* [Biostrings](https://bioconductor.org/packages/release/bioc/html/Biostrings.html): Efficient manipulation of biological strings
* [biscale](https://cran.r-project.org/web/packages/biscale/index.html): Tools and Palettes for Bivariate Thematic Mapping
* [blastula](https://cran.r-project.org/web/packages/blastula/index.html): Easily Send HTML Email Messages
* [blogdown](https://cran.r-project.org/web/packages/blogdown/index.html): Create Blogs and Websites with R Markdown
* [bluster](https://www.bioconductor.org/packages/release/bioc/html/bluster.html): Clustering Algorithms for Bioconductor
* [bnlearn](https://cran.r-project.org/web/packages/bnlearn/index.html): Bayesian Network Structure Learning, Parameter Learning and Inference
* [bookdown](https://cran.rstudio.com/web/packages/bookdown/index.html): Authoring Books and Technical Documents with R Markdown
* [bookdownplus](https://github.com/pzhaonet/bookdownplus): The easiest way to use R package bookdown for writing varied types of books and documents
* [boot](https://cran.r-project.org/web/packages/boot/index.html): Bootstrap Functions (Originally by Angelo Canty for S)
* [Boov](https://cran.r-project.org/web/packages/Boov/index.html): Boolean Operations on Volumes
* [Boruta](https://cran.r-project.org/web/packages/Boruta/index.html): Wrapper Algorithm for All Relevant Feature Selection
* [brickr](https://cran.r-project.org/web/packages/brickr/index.html): Emulate LEGO Bricks in 2D and 3D
* [broom](https://cran.r-project.org/web/packages/broom/index.html): Convert Statistical Objects into Tidy Tibbles
* [broom.mixed](https://cran.r-project.org/web/packages/broom.mixed/index.html): Tidying Methods for Mixed Models
* [brranching](https://cran.r-project.org/web/packages/brranching/index.html): Fetch 'Phylogenies' from Many Sources
* [bruceR](https://cran.r-project.org/web/packages/bruceR/index.html): Broadly Useful Convenient and Efficient R Functions
* [bslib](https://cran.r-project.org/web/packages/bslib/index.html): Custom 'Bootstrap' 'Sass' Themes for 'shiny' and 'rmarkdown'
* [bubbleHeatmap](https://cran.r-project.org/web/packages/bubbleHeatmap/index.html): Produces 'bubbleHeatmap' Plots for Visualising Metabolomics Data
* [bulletchartr](https://github.com/ACDIVOCATech/bulletchartr): This package builds bullet charts suitable for tracking KPIs, and used by ACDI/VOCA to track progress against indicators
* [bundle](https://cran.r-project.org/web/packages/bundle/index.html): Serialize Model Objects with a Consistent Interface

### C

* [Cairo](https://cran.r-project.org/web/packages/Cairo/index.html): R Graphics Device using Cairo Graphics Library for Creating High-Quality Bitmap (PNG, JPEG, TIFF), Vector (PDF, SVG, PostScript) and Display (X11 and Win32) Output
* [calendR](https://cran.r-project.org/web/packages/calendR/index.html): Ready to Print Monthly and Yearly Calendars Made with 'ggplot2'
* [camcorder](https://github.com/thebioengineer/camcorder): Record plots generated during an R session and replay as a gif!
* [canvasXpress](https://cran.r-project.org/web/packages/canvasXpress/index.html): Visualization Package for CanvasXpress in R
* [car](https://cran.r-project.org/web/packages/car/index.html): Companion to Applied Regression
* [caret](https://cran.r-project.org/web/packages/caret/index.html): Classification and Regression Training; Misc functions for training and plotting classification and regression models.
* [Cartogram](https://r-graph-gallery.com/cartogram.html): build cartograms in R
* [cartography](https://cran.r-project.org/web/packages/cartography/index.html): Thematic Cartography
* [caTools](https://cran.r-project.org/web/packages/caTools/index.html): Tools: Moving Window Statistics, GIF, Base64, ROC AUC, etc
* [CBCgrps](https://cran.r-project.org/web/packages/CBCgrps/index.html): Compare Baseline Characteristics Between Groups
* [CBNplot](https://github.com/noriakis/CBNplot): Bayesian network plot for the enrichment analysis results
* [ccgraph](https://github.com/gaospecial/ccgraph): Âú® R ÁéØÂ¢É‰∏ãÁªòÂà∂ ‚ÄúÂúÜÁéØÂ•óÂúÜÁéØ‚Äù ÁöÑ ‚ÄúÂõæ‚Äù
* [cellbaseR](https://www.bioconductor.org/packages/release/bioc/html/cellbaseR.html): Querying annotation data from the high performance Cellbase web
* [celltypist](https://github.com/Teichlab/celltypist): A tool for semi-automatic cell type annotation
* [cellxgenedp](https://bioconductor.org/packages/release/bioc/html/cellxgenedp.html): Discover and Access Single Cell Data Sets in the cellxgene Data Portal
* [cgdsr](https://cran.r-project.org/web/packages/cgdsr/index.html): R-Based API for Accessing the MSKCC Cancer Genomics Data Server (CGDS)
* [CGPfunctions](https://cran.r-project.org/web/packages/CGPfunctions/index.html): Powell Miscellaneous Functions for Teaching and Learning Statistics
* [checkmate](https://cran.r-project.org/web/packages/checkmate/index.html): Fast and Versatile Argument Checks
* [ChIPseeker](https://www.bioconductor.org/packages/release/bioc/html/ChIPseeker.html): ChIPseeker for ChIP peak Annotation, Comparison, and Visualization
* [ChromHeatMap](http://bioconductor.org/packages/release/bioc/html/ChromHeatMap.html): Heat map plotting by genome coordinate
* [chromoMap](https://cran.r-project.org/web/packages/chromoMap/index.html): Interactive Visualization and Mapping of Chromosomes
* [circlize](https://github.com/jokergoo/circlize): circular visualization in R
* [cisDynet](https://github.com/tzhu-bio/cisDynet): An integrated platform for modeling gene-regulatory dynamics and networks
* [cli](https://cran.r-project.org/web/packages/cli/index.html): Helpers for Developing Command Line Interfaces
* [clipr](https://cran.r-project.org/web/packages/clipr/index.html): Read and Write from the System Clipboard
* [clock](https://cran.r-project.org/web/packages/clock/index.html): Date-Time Types and Tools
* [cluster](https://cran.r-project.org/web/packages/cluster/index.html): "Finding Groups in Data": Cluster Analysis Extended Rousseeuw et al.
* [ClusterGVis](https://github.com/junjunlab/ClusterGVis): One-step to Cluster and Visualize Gene Expression Matrix
* [clusterProfiler](https://bioconductor.org/packages/release/bioc/html/clusterProfiler.html): statistical analysis and visualization of functional profiles for genes and gene clusters
* [CMplot](https://cran.r-project.org/web/packages/CMplot/index.html): Circle Manhattan Plot
* [cmprsk](https://cran.r-project.org/web/packages/cmprsk/index.html): Subdistribution Analysis of Competing Risks
* [coefplot](https://cran.r-project.org/web/packages/coefplot/index.html): Plots Coefficients from Fitted Models
* [cola](https://www.bioconductor.org/packages/release/bioc/html/cola.html): A Framework for Consensus Partitioning
* [coloc](https://cran.r-project.org/web/packages/coloc/index.html): Colocalisation Tests of Two Genetic Traits
* [colorblindcheck](https://cran.r-project.org/web/packages/colorblindcheck/index.html): Check Color Palettes for Problems with Color Vision Deficiency
* [colorfindr](https://github.com/zumbov2/colorfindr): Extracts colors from various image types, plots treemaps and 3D scatterplots of color compositions, create color palettes
* [colorfulVennPlot](https://cran.r-project.org/web/packages/colorfulVennPlot/index.html): Plot and add custom coloring to Venn diagrams for 2-dimensional, 3-dimensional and 4-dimensional data
* [colorist](https://cran.r-project.org/web/packages/colorist/index.html): Coloring Wildlife Distributions in Space-Time
* [colorplaner](http://cran.nexr.com/web/packages/colorplaner/index.html): A 'ggplot2' Extension to Visualize Two Variables per Color Aesthetic Through Color Space Projections
* [colorspace](https://cran.r-project.org/web/packages/colorspace/index.html): A Toolbox for Manipulating and Assessing Colors and Palettes
* [colourpicker](https://cran.r-project.org/web/packages/colourpicker/index.html): A Colour Picker Tool for Shiny and for Selecting Colours in Plots
* [cols4all](https://github.com/mtennekes/cols4all): Colors for all (R package)
* [combinat](https://cran.r-project.org/web/packages/combinat/index.html): combinatorics utilities
* [compareGroups](https://cran.r-project.org/web/packages/compareGroups/index.html): Descriptive Analysis by Groups
* [ComplexHeatmap](http://bioconductor.org/packages/release/bioc/html/ComplexHeatmap.html): Make Complex Heatmaps
* [ComplexUpset](https://cran.r-project.org/web/packages/ComplexUpset/index.html): Create Complex UpSet Plots Using 'ggplot2' Components
* [concaveman](https://cran.r-project.org/web/packages/concaveman/index.html): A Very Fast 2D Concave Hull Algorithm
* [conflicted](https://cran.r-project.org/web/packages/conflicted/index.html): An Alternative Conflict Resolution Strategy
* [ConsensusClusterPlus](https://bioconductor.org/packages/release/bioc/html/ConsensusClusterPlus.html): algorithm for determining cluster count and membership by stability evidence in unsupervised analysis
* [contsurvplot](https://cran.r-project.org/web/packages/contsurvplot/index.html): Visualize the Effect of a Continuous Variable on a Time-to-Event Outcome
* [correlation](https://cran.r-project.org/web/packages/correlation/index.html): Methods for Correlation Analysis
* [corrgram](https://cran.r-project.org/web/packages/corrgram/index.html): Plot a Correlogram
* [corrmorant](https://github.com/r-link/corrmorant): R package for flexible correlation matrices based on ggplot2
* [corrplot](https://cran.r-project.org/web/packages/corrplot/index.html): Visualization of a Correlation Matrix
* [countdown: ](https://github.com/gadenbuie/countdown)‚è≤ countdown timer for R Markdown slides and HTML docs
* [cowplot](https://cran.r-project.org/web/packages/cowplot/index.html): Streamlined Plot Theme and Plot Annotations for 'ggplot2'
* [CRANsearcher](https://cran.r-project.org/web/packages/CRANsearcher/index.html): RStudio Addin for Searching Packages in CRAN Database Based on Keywords
* [crayon](https://cran.r-project.org/web/packages/crayon/index.html): Colored Terminal Output
* [cropcircles](https://github.com/doehm/cropcircles): Crops an image to a circle in R
* [CrossICC](https://www.bioconductor.org/packages/release/bioc/html/CrossICC.html): An Interactive Consensus Clustering Framework for Multi-platform Data Analysis
* [crosslink](https://github.com/zzwch/crosslink): Network Visualization Tailed for Grouped Nodes
* [crosstable](https://cran.r-project.org/web/packages/crosstable/index.html): Crosstables for Descriptive Analyses
* [ctv](https://cran.r-project.org/web/packages/ctv/index.html): CRAN Task Views
* [curatedTCGAData](https://bioconductor.org/packages/release/data/experiment/html/curatedTCGAData.html): Curated Data From The Cancer Genome Atlas (TCGA) as MultiAssayExperiment Objects
* [customLayout](https://cran.r-project.org/web/packages/customLayout/index.html): Arrange Elements on the R's Drawing Area or Inside the PowerPoint's Slide
* [cutoff](https://cran.r-project.org/web/packages/cutoff/index.html): Seek the Significant Cutoff Value

### D

* [DALEX](https://cran.r-project.org/web/packages/DALEX/index.html): moDel Agnostic Language for Exploration and eXplanation
* [data.table](https://cran.r-project.org/web/packages/data.table/index.html): Extension of 'data.frame'
* [datadictionary](https://cran.r-project.org/web/packages/datadictionary/index.html): Create a Data Dictionary
* [DataExplorer](https://cran.r-project.org/web/packages/DataExplorer/index.html): Automate Data Exploration and Treatment
* [dataMojo](https://cran.rstudio.com/web/packages/dataMojo/index.html): Reshape Data Table
* [datapasta](https://github.com/MilesMcBain/datapasta): is about reducing resistance associated with copying and pasting data to and from R
* [datarium](https://cran.r-project.org/web/packages/datarium/index.html): Data Bank for Statistical Analysis and Visualization
* [datasauRus](https://cran.r-project.org/web/packages/datasauRus/index.html): Datasets from the Datasaurus Dozen
* [ddqc_R](https://github.com/ayshwaryas/ddqc_R): Biology-centered data-driven quality control for single cell/nucleus RNA sequencing
* [deconstructSigs](https://cran.r-project.org/web/packages/deconstructSigs/index.html): Identifies Signatures Present in a Tumor Sample
* [DEGAS](https://github.com/tsteelejohnson91/DEGAS): DEGAS is an R package that can be used to prioritize cells in relation to disease.
* [DemographicTable](https://cran.r-project.org/web/packages/DemographicTable/index.html): Creating Demographic Table
* [dendextend](https://cran.r-project.org/web/packages/dendextend/index.html): Extending 'dendrogram' Functionality in R
* [dendsort](https://cran.rstudio.com/web/packages/dendsort/index.html): Modular Leaf Ordering Methods for Dendrogram Nodes
* [descriptr](https://cran.r-project.org/web/packages/descriptr/index.html): Generate Descriptive Statistics
* [DescrTab2](https://cran.r-project.org/web/packages/DescrTab2/index.html): Publication Quality Descriptive Statistics Tables
* [DescTools](https://cran.r-project.org/web/packages/DescTools/index.html): Tools for Descriptive Statistics
* [dlookr](https://cran.r-project.org/web/packages/dlookr/index.html): Tools for Data Diagnosis, Exploration, Transformation
* [DNAshapeR](https://www.bioconductor.org/packages/release/bioc/html/DNAshapeR.html): High-throughput prediction of DNA shape features
* [DiagrammeR](https://cran.r-project.org/web/packages/DiagrammeR/index.html): Graph/Network Visualization
* [diffdf](https://cran.r-project.org/web/packages/diffdf/index.html): Dataframe Difference Tool
* [distill](https://cran.r-project.org/web/packages/distill/index.html): 'R Markdown' Format for Scientific and Technical Writing
* [doMC](https://cran.r-project.org/web/packages/doMC/index.html): Foreach Parallel Adaptor for 'parallel'
* [doParallel](https://cran.r-project.org/web/packages/doParallel/index.html): Foreach Parallel Adaptor for the 'parallel' Package
* [dorothea](https://bioconductor.org/packages/release/data/experiment/html/dorothea.html): Collection Of Human And Mouse TF Regulons
* [doSNOW](https://cran.r-project.org/web/packages/doSNOW/index.html): Foreach Parallel Adaptor for the 'snow' Package
* [drake](https://cran.r-project.org/web/packages/drake/index.html): A Pipeline Toolkit for Reproducible Computation at Scale
* [drawProteins](https://bioconductor.org/packages/release/bioc/html/drawProteins.html): Package to Draw Protein Schematics from Uniprot API output
* [DrugSim2DR](https://cran.r-project.org/web/packages/DrugSim2DR/index.html): Predict Drug Functional Similarity to Drug Repurposing
* [DR.SC](https://cran.r-project.org/web/packages/DR.SC/index.html): Joint Dimension Reduction and Spatial Clustering
* [dslabs](https://cran.r-project.org/web/packages/dslabs/index.html): Data Science Labs
* [DT](https://cran.r-project.org/web/packages/DT/index.html): A Wrapper of the JavaScript Library 'DataTables'
* [dtrackr](https://cran.r-project.org/web/packages/dtrackr/index.html): Track your Data Pipelines
* [dynamicTreeCut](https://cran.r-project.org/web/packages/dynamicTreeCut/index.html): Methods for Detection of Clusters in Hierarchical Clustering Dendrograms
* [DynNom](https://cran.r-project.org/web/packages/DynNom/index.html): Visualising Statistical Models using Dynamic Nomograms

### E

* [e1071](https://cran.r-project.org/web/packages/e1071/index.html): Functions for latent class analysis, short time Fourier transform, fuzzy clustering, support vector machines, shortest path computation, bagged clustering, naive Bayes classifier, ...
* [easier](https://www.bioconductor.org/packages/release/bioc/html/easier.html): Estimate Systems Immune Response from RNA-seq data
* [easierData](https://bioconductor.org/packages/release/data/experiment/html/easierData.html): easier internal data and exemplary dataset from IMvigor210CoreBiologies package
* [easyalluvial](https://cran.r-project.org/web/packages/easyalluvial/index.html): Generate Alluvial Plots with a Single Line of Code
* [easylabel](https://cran.r-project.org/web/packages/easylabel/index.html): Interactive Scatter Plot and Volcano Plot Labels
* [easyPubMed](https://cran.r-project.org/web/packages/easyPubMed/index.html): Search and Retrieve Scientific Publication Records from PubMed
* [EasyStat](https://github.com/taowenmicro/EasyStat): Â∑ÆÂºÇÂàÜÊûêÂÆåÊï¥Ëß£ÂÜ≥ÊñπÊ°à
* [EBImage](https://bioconductor.org/packages/release/bioc/html/EBImage.html): Image processing and analysis toolbox for R
* [echarts4r](https://cran.r-project.org/web/packages/echarts4r/index.html): Create Interactive Graphs with 'Echarts JavaScript' Version 5
* [effectsize](https://cran.r-project.org/web/packages/effectsize/index.html): Indices of Effect Size and Standardized Parameters
* [egg](https://cran.r-project.org/web/packages/egg/index.html): Extensions for 'ggplot2': Custom Geom, Custom Themes, Plot Alignment, Labelled Panels, Symmetric Scales, and Fixed Panel Size
* [embed](https://cran.r-project.org/web/packages/embed/index.html): Extra Recipes for Encoding Categorical Predictors
* [emmeans](https://cran.r-project.org/web/packages/emmeans/index.html): Estimated Marginal Means, aka Least-Squares Means
* [emojifont](https://cran.r-project.org/web/packages/emojifont/index.html): Emoji and Font Awesome in Graphics
* [englishfootball](https://github.com/jfjelstul/englishfootball): A Comprehensive Database on the Premier League and the English Football League (1888-2022)
* [enrichplot](https://bioconductor.org/packages/release/bioc/html/enrichplot.html): Visualization of Functional Enrichment Result
* [EnhancedVolcano](https://bioconductor.org/packages/release/bioc/html/EnhancedVolcano.html): Publication-ready volcano plots with enhanced colouring and labeling
* [EnrichedHeatmap](https://bioconductor.org/packages/release/bioc/html/EnrichedHeatmap.html): Making Enriched Heatmaps
* [eoffice](https://cran.r-project.org/web/packages/eoffice/index.html): Export or Graph and Tables to 'Microsoft' Office and Import Figures and Tables
* [epistack](https://www.bioconductor.org/packages/release/bioc/html/epistack.html): Heatmaps of Stack Profiles from Epigenetic Signals
* [esquisse](https://cran.r-project.org/web/packages/esquisse/index.html): Explore and Visualize Your Data Interactively
* [eulerr](https://cran.r-project.org/web/packages/eulerr/index.html): Area-Proportional Euler and Venn Diagrams with Ellipses
* [europepmc](https://cran.r-project.org/web/packages/europepmc/index.html): R Interface to the Europe PubMed Central RESTful Web Service
* [exams](https://cran.r-project.org/web/packages/exams/index.html): Automatic Generation of Exams in R
* [exomePeak2](https://bioconductor.org/packages/release/bioc/html/exomePeak2.html): Bias-aware Peak Calling and Quantification for MeRIP-Seq
* [explore](https://cran.r-project.org/web/packages/explore/index.html): Simplifies Exploratory Data Analysis
* [exploreR](https://cran.r-project.org/web/packages/exploreR/index.html): Tools for Quickly Exploring Data
* [export](https://github.com/tomwenseleers/export): easily export active R graphs and statistical output in publication quality to Microsoft Office (Word, PowerPoint and Excel), HTML and Latex
* [expss](https://cran.r-project.org/web/packages/expss/index.html): Tables, Labels and Some Useful Functions from Spreadsheets and 'SPSS' Statistics
* [extrafont](https://cran.r-project.org/web/packages/extrafont/index.html): Tools for using fonts
* [ExPosition](https://cran.r-project.org/web/packages/ExPosition/index.html): Exploratory Analysis with the Singular Value Decomposition
* [ezcox](https://cran.r-project.org/web/packages/ezcox/index.html): Easily Process a Batch of Cox Models

### F

* [factoextra](https://cran.r-project.org/web/packages/factoextra/index.html): Extract and Visualize the Results of Multivariate Data Analyses
* [FactoInvestigate](https://cran.r-project.org/web/packages/FactoInvestigate/index.html): Automatic Description of Factorial Analysis
* [FactoMineR](https://cran.r-project.org/web/packages/FactoMineR/index.html): Multivariate Exploratory Data Analysis and Data Mining
* [familiar](https://cran.r-project.org/web/packages/familiar/index.html): End-to-End Automated Machine Learning and Model Evaluation
* [fanyi](https://cran.r-project.org/web/packages/fanyi/index.html): Translate Words or Sentences via Online Translators
* [fbroc](https://cran.r-project.org/web/packages/fbroc/index.html): Fast Algorithms to Bootstrap Receiver Operating Characteristics Curves
* [fgsea](https://bioconductor.org/packages/release/bioc/html/fgsea.html): Fast Gene Set Enrichment Analysis
* [fields](https://mran.microsoft.com/snapshot/2017-02-04/web/packages/fields/index.html): Tools for Spatial Data
* [figuRes2](https://cran.r-project.org/web/packages/figuRes2/index.html): Support for a Variety of Figure Production Tasks
* [FindIT2](https://bioconductor.org/packages/devel/bioc/html/FindIT2.html): find influential TF and Target based on multi-omics data
* [filehash](https://cran.r-project.org/web/packages/filehash/index.html): Simple Key-Value Database
* [finalfit](https://cran.r-project.org/web/packages/finalfit/index.html): Quickly Create Elegant Regression Results Tables and Plots when Modelling
* [fishplot](https://github.com/chrisamiller/fishplot): Create timecourse "fish plots" that show changes in the clonal architecture of tumors
* [Fletcher2013a](https://bioconductor.org/packages/release/data/experiment/html/Fletcher2013a.html): Gene expression data from breast cancer cells under FGFR2 signalling perturbation
* [flextable](https://cran.r-project.org/web/packages/flextable/index.html): Functions for Tabular Reporting
* [fmsb](https://cran.r-project.org/web/packages/fmsb/index.html): Functions for Medical Statistics Book with some Demographic Data
* [forcats](https://cran.r-project.org/web/packages/forcats/index.html): Tools for Working with Categorical Variables (Factors)
* [foreach](https://cran.r-project.org/web/packages/foreach/index.html): Provides Foreach Looping Construct
* [forecast](https://cran.r-project.org/web/packages/forecast/index.html): Forecasting Functions for Time Series and Linear Models
* [forestmodel](https://cran.r-project.org/web/packages/forestmodel/index.html): Forest Plots from Regression Models
* [forestplot](https://cran.r-project.org/web/packages/forestplot/index.html): Advanced Forest Plot Using 'grid' Graphics
* [forestploter](https://cran.r-project.org/web/packages/forestploter/index.html): Create Flexible Forest Plot
* [formatdown](https://cran.r-project.org/web/packages/formatdown/index.html): Formatting Tools for 'rmarkdown' Documents
* [formattable](https://cran.r-project.org/web/packages/formattable/index.html): Create 'Formattable' Data Structures
* [forplo](https://cran.r-project.org/web/packages/forplo/index.html): Flexible Forest Plots
* [fpc](https://cran.r-project.org/web/packages/fpc/index.html): Flexible Procedures for Clustering
* [fpp3](https://cran.r-project.org/web/packages/fpp3/index.html): Data for "Forecasting: Principles and Practice" (3rd Edition)
* [framecleaner](https://cran.r-project.org/web/packages/framecleaner/index.html): Clean Data Frames
* [funkyheatmap](https://cran.r-project.org/web/packages/funkyheatmap/index.html): Generating Funky Heatmaps for Data Frames
* [FunnelPlotR](https://cran.r-project.org/web/packages/FunnelPlotR/index.html): Funnel Plots for Comparing Institutional Performance
* [furrr](https://cran.r-project.org/web/packages/furrr/index.html): Apply Mapping Functions in Parallel using Futures
* [future](https://cran.r-project.org/web/packages/future/index.html): Unified Parallel and Distributed Processing in R for Everyone
* [future.apply](https://cran.r-project.org/web/packages/future.apply/index.html): Apply Function to Elements in Parallel using Futures

### G

* [ganttrify](https://github.com/giocomai/ganttrify): Create beautiful Gantt charts with ggplot2
* [gapminder](https://cran.r-project.org/web/packages/gapminder/index.html): Data from Gapminder
* [gdata](https://cran.r-project.org/web/packages/gdata/index.html): Various R Programming Tools for Data Manipulation
* [GDCRNATools](https://bioconductor.org/packages/release/bioc/html/GDCRNATools.html): an R/Bioconductor package for integrative analysis of lncRNA, mRNA, and miRNA data in GDC
* [geneHapR](https://cran.r-project.org/web/packages/geneHapR/index.html): Gene Haplotype Statistics, Phenotype Association and Visualization
* [genekitr](https://cran.r-project.org/web/packages/genekitr/index.html): Gene Analysis Toolkit
* [genieBPC](https://cran.r-project.org/web/packages/genieBPC/index.html): Project GENIE BioPharma Collaborative Data Processing Pipeline
* [genomicInstability](https://www.bioconductor.org/packages/release/bioc/html/genomicInstability.html): Genomic Instability estimation for scRNA-Seq
* [GenomicRanges](https://bioconductor.org/packages/release/bioc/html/GenomicRanges.html): Representation and manipulation of genomic intervals
* [geojsonsf](https://cran.r-project.org/web/packages/geojsonsf/index.html): GeoJSON to Simple Feature Converter
* [geomnet](http://cran.nexr.com/web/packages/geomnet/index.html): Network Visualization in the 'ggplot2' Framework
* [geomtextpath](https://cran.r-project.org/web/packages/geomtextpath/index.html): Curved Text in 'ggplot2'
* [geoR](https://cran.r-project.org/web/packages/geoR/index.html): Analysis of Geostatistical Data
* [geoviz](https://cran.r-project.org/web/packages/geoviz/index.html): Elevation and GPS Data Visualisation
* [GetoptLong](https://cran.r-project.org/web/packages/GetoptLong/index.html): Parsing Command-Line Arguments and Variable Interpolation
* [gg.gap](https://cran.r-project.org/web/packages/gg.gap/index.html): Define Segments in y-Axis for 'ggplot2'
* [ggalluvial](https://cran.r-project.org/web/packages/ggalluvial/index.html): Alluvial Plots in 'ggplot2'
* [GGally](https://cran.r-project.org/web/packages/GGally/index.html): Extension to 'ggplot2'
* [ggalt](https://cran.r-project.org/web/packages/ggalt/index.html): Extra Coordinate Systems, 'Geoms', Statistical Transformations, Scales and Fonts for 'ggplot2'
* [gganatogram](https://github.com/jespermaag/gganatogram): Create anatogram images for different organisms using ggplot2
* [gganimate](https://cran.r-project.org/web/packages/gganimate/index.html): A Grammar of Animated Graphics
* [ggannotate](https://github.com/MattCowgill/ggannotate): Interactively annotate ggplots
* [ggarchery](https://cran.r-project.org/web/packages/ggarchery/index.html): Flexible Segment Geoms with Arrows for 'ggplot2'
* [ggbeeswarm](https://github.com/eclarke/ggbeeswarm): Beeswarm-style plots with ggplot2
* [ggbio](https://bioconductor.org/packages/release/bioc/html/ggbio.html): Visualization tools for genomic data
* [ggblanket](https://cran.r-project.org/web/packages/ggblanket/index.html): Wrappers to Simplify 'ggplot2' Visualisation
* [ggblanket](https://cran.r-project.org/web/packages/ggblanket/index.html): Simplify 'ggplot2' Visualisation
* [ggblur](https://github.com/coolbutuseless/ggblur): Blurry Point Geom for ggplot2
* [ggborderline](https://cran.r-project.org/web/packages/ggborderline/index.html): Line Plots that Pop
* [ggbreak](https://cran.r-project.org/web/packages/ggbreak/index.html): Set Axis Break for 'ggplot2'
* [ggbrick](https://github.com/doehm/ggbrick): A brick like geom for ggplot2
* [ggBubbles](https://cran.r-project.org/web/packages/ggBubbles/index.html): Mini Bubble Plots for Comparison of Discrete Data with 'ggplot2'
* [ggbump](https://cran.r-project.org/web/packages/ggbump/index.html): Bump Chart and Sigmoid Curves
* [ggchicklet](https://github.com/hrbrmstr/ggchicklet): Create Chicklet (Rounded Segmented Column) Charts
* [ggchromatic](https://github.com/teunbrand/ggchromatic): Colour and fill scales for 'ggplot2' using colour spaces.
* [ggClusterNet](https://github.com/taowenmicro/ggClusterNet): Microbial ecological network visualization clustering
* [ggcor](https://github.com/houyunhuang/ggcor): Extended tools for correlation analysis and visualization
* [ggcorrplot](https://cran.r-project.org/web/packages/ggcorrplot/index.html): Visualization of a Correlation Matrix using 'ggplot2'
* [ggcorrplot2](https://github.com/caijun/ggcorrplot2): Implementation of corrplot using ggplot2
* [ggCyberPunk](https://github.com/delabj/ggCyberPunk): A "Geom" for creating charts with a cyberpunk aesthetic
* [ggcyto](https://www.bioconductor.org/packages/release/bioc/html/ggcyto.html): Visualize Cytometry data with ggplot
* [ggDCA](https://cran.r-project.org/web/packages/ggDCA/index.html): Calculate and Plot Decision Curve
* [ggdendro](https://cran.r-project.org/web/packages/ggdendro/index.html): Create Dendrograms and Tree Diagrams Using 'ggplot2'
* [ggdist](https://cran.r-project.org/web/packages/ggdist/index.html): Visualizations of Distributions and Uncertainty
* [ggDoubleHeat](https://cran.rstudio.com/web/packages/ggDoubleHeat/index.html): A Heatmap-Like Visualization Tool
* [ggeasy](https://cran.r-project.org/web/packages/ggeasy/index.html): Easy Access to 'ggplot2' Commands
* [ggeconodist](https://github.com/hrbrmstr/ggeconodist): Create Diminutive Distribution Charts
* [ggeffects](https://cran.r-project.org/web/packages/ggeffects/index.html): Create Tidy Data Frames of Marginal Effects for 'ggplot' from Model Outputs
* [ggExtra](https://cran.r-project.org/web/packages/ggExtra/index.html): Add Marginal Histograms to 'ggplot2', and More 'ggplot2' Enhancements
* [ggfan](https://cran.r-project.org/web/packages/ggfan/index.html): Summarise a Distribution Through Coloured Intervals
* [ggfittext](https://cran.r-project.org/web/packages/ggfittext/index.html): Fit Text Inside a Box in 'ggplot2'
* [ggfx](https://cran.r-project.org/web/packages/ggfx/index.html): Pixel Filters for 'ggplot2' and 'grid'
* [ggforce](https://cran.r-project.org/web/packages/ggforce/index.html): Accelerating 'ggplot2'
* [ggforestplot](https://github.com/NightingaleHealth/ggforestplot): An R package to create forestplots of measures of effects with their confidence intervals.
* [ggfortify](https://cran.r-project.org/web/packages/ggfortify/index.html): Data Visualization Tools for Statistical Analysis Results
* [ggfocus](https://cran.r-project.org/web/packages/ggfocus/index.html): Scales that Focus Specific Levels in your ggplot()
* [gggenes](https://cran.r-project.org/web/packages/gggenes/index.html): Draw Gene Arrow Maps in 'ggplot2'
* [gggenomes](https://github.com/thackl/gggenomes): A grammar of graphics for comparative genomics
* [gggibbous](https://cran.r-project.org/web/packages/gggibbous/index.html): Moon Charts, a Pie Chart Alternative
* [ggh4x](https://cran.rstudio.com/web/packages/ggh4x/index.html): Hacks for 'ggplot2'
* [gghalves](https://cran.r-project.org/web/packages/gghalves/index.html): Compose Half-Half Plots Using Your Favourite Geoms
* [ggheatmap](https://cran.r-project.org/web/packages/ggheatmap/index.html): Plot Heatmap
* [ggheatmap](https://github.com/XiaoLuo-boy/ggheatmap): ggplot2 version of heatmap
* [gghighlight](https://cran.r-project.org/web/packages/gghighlight/index.html): Highlight Lines and Points in 'ggplot2'
* [ggHoriPlot](https://cran.r-project.org/web/packages/ggHoriPlot/index.html): Horizon Plots for 'ggplot2'
* [ggiraph](https://cran.r-project.org/web/packages/ggiraph/index.html): Make 'ggplot2' Graphics Interactive
* [ggimage](https://cran.r-project.org/web/packages/ggimage/index.html): Use Image in 'ggplot2'
* [ggimg](https://cran.r-project.org/web/packages/ggimg/index.html): Graphics Layers for Plotting Image Data with 'ggplot2'
* [gglmannotate](https://github.com/wilkox/gglmannotate): üìàAnnotate a ggplot with a description of a linear model
* [ggmagnify](https://github.com/hughjonesd/ggmagnify): Create a magnified inset of part of a ggplot object
* [ggmap](https://cran.r-project.org/web/packages/ggmap/index.html): Spatial Visualization with ggplot2
* [ggmcmc](https://cran.r-project.org/web/packages/ggmcmc/index.html): Tools for Analyzing MCMC Simulations from Bayesian Inference
* [ggmosaic](https://cran.r-project.org/web/packages/ggmosaic/index.html): Mosaic Plots in the 'ggplot2' Framework
* [ggmsa](https://cran.r-project.org/web/packages/ggmsa/index.html): Plot Multiple Sequence Alignment using 'ggplot2'
* [ggmuller](https://github.com/robjohnnoble/ggmuller): Create Muller Plots of Evolutionary Dynamics
* [ggnetwork](https://cran.r-project.org/web/packages/ggnetwork/index.html): Geometries to Plot Networks with 'ggplot2'
* [ggnewscale](https://cran.r-project.org/web/packages/ggnewscale/index.html): Multiple Fill and Colour Scales in 'ggplot2'
* [ggOceanMaps](https://cran.r-project.org/web/packages/ggOceanMaps/index.html): Plot Data on Oceanographic Maps using 'ggplot2'
* [ggord](https://github.com/fawda123/ggord): a take on ordination plots using ggplot2
* [ggpackets](https://cran.r-project.org/web/packages/ggpackets/index.html): Package Plot Layers for Easier Portability and Modularization
* [ggparliament](https://github.com/RobWHickman/ggparliament): Simple parliament plots using ggplot2
* [ggpathway](https://github.com/cxli233/ggpathway): : A tutorial for pathway visualization using tidyverse, igraph, and ggraph
* [ggpattern](https://github.com/coolbutuseless/ggpattern): ggplot geoms with pattern fills
* [ggpcp](https://cran.r-project.org/web/packages/ggpcp/index.html): Parallel Coordinate Plots in the 'ggplot2' Framework
* [ggpie](https://cran.r-project.org/web/packages/ggpie/index.html): Pie, Donut and Rose Pie Plots
* [ggplate](https://cran.r-project.org/web/packages/ggplate/index.html): Create Layout Plots of Biological Culture Plates and Microplates
* [ggplot.multistats](https://cran.r-project.org/web/packages/ggplot.multistats/index.html): Multiple Summary Statistics for Binned Stats/Geometries
* [ggplotgui](https://cran.r-project.org/web/packages/ggplotgui/index.html): Create Ggplots via a Graphical User Interface
* [ggplotify](https://cran.r-project.org/web/packages/ggplotify/index.html): Convert Plot to 'grob' or 'ggplot' Object
* [ggpmisc](https://cran.r-project.org/web/packages/ggpmisc/index.html): Miscellaneous Extensions to 'ggplot2'
* [ggpointdensity](https://cran.r-project.org/web/packages/ggpointdensity/index.html): A Cross Between a 2D Density Plot and a Scatter Plot
* [ggpol](https://cran.r-project.org/web/packages/ggpol/index.html): Visualizing Social Science Data with 'ggplot2'
* [ggpp](https://cran.r-project.org/web/packages/ggpp/index.html): Grammar Extensions to 'ggplot2'
* [ggprism](https://github.com/csdaw/ggprism): The `ggprism` package provides various themes, palettes, and other useful functions to customise ggplots and give them the *‚ÄòGraphPad Prism‚Äô* look.
* [ggpubr](https://cran.r-project.org/web/packages/ggpubr/index.html): 'ggplot2' Based Publication Ready Plots
* [ggpval](https://cran.r-project.org/web/packages/ggpval/index.html): Annotate Statistical Tests for 'ggplot2'
* [ggradar](https://github.com/ricardo-bion/ggradar): radar charts with ggplot2
* [ggrain](https://cran.r-project.org/web/packages/ggrain/index.html): A Rainclouds Geom for 'ggplot2'
* [ggraph](https://cran.r-project.org/web/packages/ggraph/index.html): An Implementation of Grammar of Graphics for Graphs and Networks
* [ggrastr](https://cran.rstudio.com/web/packages/ggrastr/index.html): Rasterize Layers for 'ggplot2'
* [ggreverse](https://github.com/coolbutuseless/ggreverse): Reverse a ggplot object back into code
* [ggrcs](https://github.com/liuqiang070488/ggrcs): A function to draw histograms and restricted cubic splines (RCS).
* [ggrepel](https://cran.r-project.org/web/packages/ggrepel/index.html): Automatically Position Non-Overlapping Text Labels with 'ggplot2'
* [ggridges](https://cran.r-project.org/web/packages/ggridges/index.html): Ridgeline Plots in 'ggplot2'
* [ggrisk](https://cran.r-project.org/web/packages/ggrisk/index.html): Risk Score Plot for Cox Regression
* [ggsankey](https://github.com/davidsjoberg/ggsankey): The goal of ggsankey is to make beautiful sankey, alluvial and sankey bump plots in ggplot2
* [ggsc](https://github.com/YuLab-SMU/ggsc): 
* [ggsector](https://cran.r-project.org/web/packages/ggsector/index.html): Draw Sectors
* [ggsci](https://cran.r-project.org/web/packages/ggsci/index.html): Scientific Journal and Sci-Fi Themed Color Palettes for 'ggplot2'
* [ggsector](https://cran.r-project.org/web/packages/ggsector/index.html): Draw Sectors
* [ggseqlogo](https://cran.r-project.org/web/packages/ggseqlogo/index.html): A 'ggplot2' Extension for Drawing Publication-Ready Sequence Logos
* [ggshadow](https://cran.rstudio.com/web/packages/ggshadow/index.html): Shadow and Glow Geoms for 'ggplot2'
* [ggside](https://cran.r-project.org/web/packages/ggside/index.html): Side Grammar Graphics
* [ggsignif](https://cran.r-project.org/web/packages/ggsignif/index.html): Significance Brackets for 'ggplot2'
* [ggsimplex](https://github.com/marvinschmitt/ggsimplex): provide a set of visualization utilities revolving around the simplex as its underlying canvas
* [ggsn](https://cran.r-project.org/web/packages/ggsn/index.html): North Symbols and Scale Bars for Maps Created with 'ggplot2' or 'ggmap'
* [ggspatial](https://cran.r-project.org/web/packages/ggspatial/index.html): Spatial Data Framework for ggplot2
* [ggstance](https://cran.r-project.org/web/packages/ggstance/index.html): Horizontal 'ggplot2' Components
* [ggstar](https://cran.rstudio.com/web/packages/ggstar/index.html): Multiple Geometric Shape Point Layer for 'ggplot2'
* [ggstats](https://cran.r-project.org/web/packages/ggstats/index.html): Extension to 'ggplot2' for Plotting Stats
* [ggstatsplot](https://cran.r-project.org/web/packages/ggstatsplot/index.html): 'ggplot2' Based Plots with Statistical Details
* [ggstream](https://github.com/davidsjoberg/ggstream): create a simple but powerful implementation of streamplot/streamgraph in `ggplot2`
* [ggsurvfit](https://cran.r-project.org/web/packages/ggsurvfit/index.html): Flexible Time-to-Event Figures
* [ggsymbol](https://github.com/xiangpin/ggsymbol): symbols layer based on ggplot2 and ggstar
* [ggtern](https://cran.r-project.org/web/packages/ggtern/index.html): An Extension to 'ggplot2', for the Creation of Ternary Diagrams
* [ggtext](https://cran.r-project.org/web/packages/ggtext/index.html): Improved Text Rendering Support for 'ggplot2'
* [ggThemeAssist](https://cran.r-project.org/web/packages/ggThemeAssist/index.html): Add-in to Customize 'ggplot2' Themes
* [ggthemes](https://cran.r-project.org/web/packages/ggthemes/index.html): Extra Themes, Scales and Geoms for 'ggplot2'
* [ggthemr](https://github.com/Mikata-Project/ggthemr): Themes for ggplot2.
* [ggTimeSeries](https://cran.r-project.org/web/packages/ggTimeSeries/index.html): Time Series Visualisations Using the Grammar of Graphics
* [ggtranscript](https://github.com/dzhang32/ggtranscript): Visualizing transcript structure and annotation using ggplot2
* [ggtree](https://www.bioconductor.org/packages/release/bioc/html/ggtree.html): an R package for visualization of tree and annotation data
* [ggtreeExtra](https://bioconductor.org/packages/release/bioc/html/ggtreeExtra.html): An R Package To Add Geometric Layers On Circular Or Other Layout Tree Of "ggtree"
* [ggtrendline](https://cran.r-project.org/web/packages/ggtrendline/index.html): Add Trendline and Confidence Interval to 'ggplot'
* [ggupset](https://github.com/const-ae/ggupset): Combination matrix axis for 'ggplot2' to create 'UpSet' plots
* [ggvenn](https://cran.r-project.org/web/packages/ggvenn/index.html): Draw Venn Diagram by 'ggplot2'
* [ggVennDiagram](https://cran.r-project.org/web/packages/ggVennDiagram/index.html): A 'ggplot2' Implement of Venn Diagram
* [ggvis](https://cran.r-project.org/web/packages/ggvis/index.html): Interactive Grammar of Graphics
* [ggwordcloud](https://cran.r-project.org/web/packages/ggwordcloud/index.html): A Word Cloud Geom for 'ggplot2'
* [BioSenior/ggVolcano](https://github.com/BioSenior/ggVolcano): The goal of `ggVolcano` is to help users make a beautiful volcano map more easily, including general volcano plot(`ggvolcano`), gradient color volcano plot(`gradual_volcano`) and GO term volcano plot(`term_volcano`).
* [ggvwline](https://github.com/houyunhuang/ggvwline): Extension to ggplot2 for drawing variable-width lines
* [ggwaffle](https://github.com/liamgilbey/ggwaffle): Creating waffle charts in a ggplot friendly way
* [ggx](https://github.com/brandmaier/ggx): A Natural Language Interface to ‚Äòggplot2‚Äô
* [ghactions](https://www.maxheld.de/ghactions/): GitHub actions for R
* [ghibli](https://cran.r-project.org/web/packages/ghibli/index.html): Studio Ghibli Colour Palettes
* [gifski](https://cran.r-project.org/web/packages/gifski/index.html): Highest Quality GIF Encoder
* [glmnet](https://cran.r-project.org/web/packages/glmnet/index.html): Lasso and Elastic-Net Regularized Generalized Linear Models
* [glmSparseNet](https://www.bioconductor.org/packages/release/bioc/html/glmSparseNet.html): Network Centrality Metrics for Elastic-Net Regularized Models
* [glue](https://cran.r-project.org/web/packages/glue/index.html): Interpreted String Literals
* [GOplot](http://wencke.github.io/): Visualization of Functional Analysis Data
* [GOSemSim](https://bioconductor.org/packages/release/bioc/html/GOSemSim.html): GO-terms Semantic Similarity Measures
* [gplots](https://cran.r-project.org/web/packages/gplots/index.html): Various R Programming Tools for Plotting Data
* [gprofiler2](https://cran.r-project.org/web/packages/gprofiler2/index.html): Interface to the 'g:Profiler' Toolset
* [gptchatteR](https://github.com/isinaltinkaya/gptchatteR): An experimental and unofficial wrapper for interacting with ChatGPT in R.
* [grafify](https://github.com/ashenoy-cmbi/grafify): Easy grafs, ANOVAs and posthoc comparisons.
* [graphlayouts](https://cran.r-project.org/web/packages/graphlayouts/index.html): Additional Layout Algorithms for Network Visualizations
* [grDevices](https://www.rdocumentation.org/packages/grDevices/versions/3.6.2): The R Graphics Devices and Support for Colours and Fonts
* [GSEABase](https://bioconductor.org/packages/release/bioc/html/GSEABase.html): Gene set enrichment data structures and methods
* [GseaVis](https://github.com/junjunlab/GseaVis): An implement R package to visualize GSEA results
* [gson](https://cran.r-project.org/web/packages/gson/index.html): Base Class and Methods for 'gson' Format
* [GSVA](https://www.bioconductor.org/packages/release/bioc/html/GSVA.html): Gene Set Variation Analysis for microarray and RNA-seq data
* [gt](https://cran.r-project.org/web/packages/gt/index.html): Easily Create Presentation-Ready Display Tables
* [gtreg](https://cran.r-project.org/web/packages/gtreg/index.html): Regulatory Tables for Clinical Research
* [gtsummary](https://cran.r-project.org/web/packages/gtsummary/index.html): Presentation-Ready Data Summary and Analytic Result Tables
* [gtools](https://cran.r-project.org/web/packages/gtools/index.html): Various R Programming Tools
* [Guitar](https://www.bioconductor.org/packages/release/bioc/html/Guitar.html): The package is designed for visualization of RNA-related genomic features with respect to the landmarks of RNA transcripts, i.e., transcription starting site, start codon, stop codon and transcription ending site.
* [Gviz](https://bioconductor.org/packages/release/bioc/html/Gviz.html): Plotting data and annotation information along genomic coordinates

### H

* [h2o](https://cran.r-project.org/web/packages/h2o/index.html): R Interface for the 'H2O' Scalable Machine Learning Platform
* [hchinamap](https://cran.r-project.org/web/packages/hchinamap/index.html): Mapping China and Its Provinces
* [hdrcde](https://cran.r-project.org/web/packages/hdrcde/index.html): Highest Density Regions and Conditional Density Estimation
* [heatmaply](https://cran.r-project.org/web/packages/heatmaply/index.html): Interactive Cluster Heat Maps Using 'plotly' and 'ggplot2'
* [here](https://cran.r-project.org/web/packages/here/index.html): A Simpler Way to Find Your Files
* [HGNChelper](https://cran.r-project.org/web/packages/HGNChelper/index.html): Identify and Correct Invalid HGNC Human Gene Symbols and MGI Mouse Gene Symbols
* [highcharter](https://cran.r-project.org/web/packages/highcharter/index.html): A Wrapper for the 'Highcharts' Library
* [HilbertCurve](https://www.bioconductor.org/packages/release/bioc/html/HilbertCurve.html): Making 2D Hilbert Curve
* [Hmisc](https://cran.r-project.org/web/packages/Hmisc/index.html): Harrell Miscellaneous
* [homologene](https://cran.r-project.org/web/packages/homologene/index.html): Quick Access to Homologene and Gene Annotation Updates
* [hrbrthemes](https://cran.r-project.org/web/packages/hrbrthemes/index.html): Additional Themes, Theme Components and Utilities for 'ggplot2'
* [httr](https://cran.r-project.org/web/packages/httr/index.html): Tools for Working with URLs and HTTP
* [huxtable](https://cran.r-project.org/web/packages/huxtable/index.html): Easily Create and Style Tables for LaTeX, HTML and Other Formats

### I

* [igraph](https://cran.r-project.org/web/packages/igraph/index.html): Network Analysis and Visualization
* [iheatmapr](https://cran.r-project.org/web/packages/iheatmapr/index.html): Interactive, Complex Heatmaps
* [IMvigor210CoreBiologies](http://research-pub.gene.com/IMvigor210CoreBiologies/): TGF-b attenuates tumor response to PD-L1 blockade by contributing to exclusion of T cells
* [infer](https://cran.r-project.org/web/packages/infer/index.html): Tidy Statistical Inference
* [inferregex](https://github.com/daranzolin/inferregex): Infer the regular expression (regex) of a string :1234:
* [intansv](https://bioconductor.org/packages/release/bioc/html/intansv.html): Integrative analysis of structural variations
* [interactions](https://cran.r-project.org/web/packages/interactions/index.html): Comprehensive, User-Friendly Toolkit for Probing Interactions
* [introdataviz](https://github.com/PsyTeachR/introdataviz):  two functions (`geom_split_violin()` and `geom_flat_violin()`) used to make split-violin and raincloud plots
* [IPEDS](https://cran.r-project.org/web/packages/IPEDS/index.html): Data from the Integrated Post-Secondary Education Data System
* [ipred](https://cran.r-project.org/web/packages/ipred/index.html): Improved Predictors
* [iterators](https://cran.r-project.org/web/packages/iterators/index.html): Provides Iterator Construct
* [itol.toolkit](https://cran.r-project.org/web/packages/itol.toolkit/index.html): Helper Functions for 'Interactive Tree Of Life'

### J

* [janitor](https://cran.r-project.org/web/packages/janitor/index.html): Simple Tools for Examining and Cleaning Dirty Data
* [janus](https://cran.r-project.org/web/packages/janus/index.html): Optimized Recommending System Based on 'tensorflow'
* [jcvi](https://github.com/tanghaibao/jcvi): Python library to facilitate genome assembly, annotation, and comparative genomics
* [jiebaR](https://cran.r-project.org/web/packages/jiebaR/index.html): Chinese Text Segmentation
* [jiebaRD](https://cran.r-project.org/web/packages/jiebaRD/index.html): Chinese Text Segmentation Data for jiebaR Package
* [jjAnno](https://github.com/junjunlab/jjAnno): An annotation package for ggplot plot
* [jjPlot](https://github.com/junjunlab/jjPlot): some geom/stat functions to produce funny graphs

### K

* [kableExtra](https://cran.r-project.org/web/packages/kableExtra/index.html): Construct Complex Table with 'kable' and Pipe Syntax
* [karyoploteR](https://bioconductor.org/packages/release/bioc/html/karyoploteR.html): Plot customizable linear genomes displaying arbitrary data
* [KEGGREST](https://bioconductor.org/packages/release/bioc/html/KEGGREST.html): Client-side REST access to the Kyoto Encyclopedia of Genes and Genomes (KEGG)
* [ keras](https://cran.r-project.org/web/packages/keras/index.html): R Interface to 'Keras'
* [kingfisher-download](https://github.com/wwood/kingfisher-download): Easier download/extract of FASTA/Q read data and metadata from the ENA, NCBI, AWS or GCP.
* [KMunicate](https://cran.r-project.org/web/packages/KMunicate/index.html): KMunicate-Style Kaplan‚ÄìMeier Plots

### L

* [LaCroixColoR](https://github.com/johannesbjork/LaCroixColoR): LaCroix Color Palettes for R
* [latex2exp](https://cran.r-project.org/web/packages/latex2exp/index.html): Use LaTeX Expressions in Plots
* [leaflet](https://cran.r-project.org/web/packages/leaflet/index.html): Create Interactive Web Maps with the JavaScript 'Leaflet' Library
* [leafletCN](https://cran.r-project.org/web/packages/leafletCN/index.html): An R Gallery for China and Other Geojson Choropleth Map in Leaflet
* [learnr](https://cran.r-project.org/web/packages/learnr/index.html): Interactive Tutorials for R
* [legendMap](https://github.com/3wen/legendMap): Provides functions to add a North arrow and/or a scale bar to maps done using ggplot2
* [lfmm](https://cran.r-project.org/web/packages/lfmm/index.html): Latent Factor Mixed Models
* [lime](https://cran.r-project.org/web/packages/lime/index.html): Local Interpretable Model-Agnostic Explanations
* [lindia](https://cran.r-project.org/web/packages/lindia/index.html): Automated Linear Regression Diagnostic
* [LinkageMapView](https://cran.r-project.org/web/packages/LinkageMapView/index.html): Plot Linkage Group Maps with Quantitative Trait Loci
* [linkET](https://github.com/Hy4m/linkET): visualize simply and directly a matrix heatmap based on ‚Äòggplot2‚Äô

### M

* [magick](https://cran.r-project.org/web/packages/magick/index.html): Advanced Graphics and Image-Processing in R
* [manhattanly](https://cran.r-project.org/web/packages/manhattanly/index.html): Interactive Q-Q and Manhattan Plots Using 'plotly.js'
* [Map2NCBI](https://cran.r-project.org/web/packages/Map2NCBI/index.html): Mapping Markers to the Nearest Genomic Feature
* [mapdeck](https://cran.r-project.org/web/packages/mapdeck/index.html): Interactive Maps Using 'Mapbox GL JS' and 'Deck.gl'
* [mapproj](https://cran.r-project.org/web/packages/mapproj/index.html): Map Projections
* [maps](https://cran.r-project.org/web/packages/maps/index.html): Draw Geographical Maps
* [mapsf](https://cran.r-project.org/web/packages/mapsf/index.html): Thematic Cartography
* [maptools](https://cran.r-project.org/web/packages/maptools/index.html): Tools for Handling Spatial Objects
* [MASS](https://cran.r-project.org/web/packages/MASS/index.html): Support Functions and Datasets for Venables and Ripley's MASS
* [mathml](https://cran.r-project.org/web/packages/mathml/index.html): Translate R Expressions to 'MathML' and 'LaTeX'/'MathJax'
* [mdthemes](https://cran.r-project.org/web/packages/mdthemes/index.html): Markdown Themes for 'ggplot2'
* [MetBrewer](https://github.com/BlakeRMills/MetBrewer): Color palette package in R inspired by works at the Metropolitan Museum of Art in New York
* [metevalue](https://cran.r-project.org/web/packages/metevalue/index.html): E-Value in the Omics Data Association Studies
* [metanetwork](https://cran.r-project.org/web/packages/metanetwork/index.html): Handling and Representing Trophic Networks in Space and Time
* [metaviz](https://cran.r-project.org/web/packages/metaviz/index.html): Forest Plots, Funnel Plots, and Visual Funnel Plot Inference for Meta-Analysis
* [MetBrewer](https://cran.r-project.org/web/packages/MetBrewer/index.html): Color Palettes Inspired by Works at the Metropolitan Museum of Art
* [mcb](https://cran.r-project.org/web/packages/mcb/index.html): Model Confidence Bounds
* [mclust](https://cran.r-project.org/web/packages/mclust/index.html): Gaussian Mixture Modelling for Model-Based Clustering, Classification, and Density Estimation
* [mde](https://cran.r-project.org/web/packages/mde/index.html): Missing Data Explorer
* [MEGENA](https://cran.r-project.org/web/packages/MEGENA/index.html): Multiscale Clustering of Geometrical Network
* [MetaboSignal](https://bioconductor.org/packages/release/bioc/html/MetaboSignal.html): a network-based approach to overlay and explore metabolic and signaling KEGG pathways
* [MEGENA](https://cran.r-project.org/web/packages/MEGENA/index.html): Multiscale Clustering of Geometrical Network
* [memes](https://www.bioconductor.org/packages/release/bioc/html/memes.html): motif matching, comparison, and de novo discovery using the MEME Suite
* [MeSHDbi](https://www.bioconductor.org/packages/release/bioc/html/MeSHDbi.html): DBI to construct MeSH-related package from sqlite file
* [meshes](https://bioconductor.org/packages/release/bioc/html/meshes.html): MeSH Enrichment and Semantic analyses
* [metafor](https://cran.r-project.org/web/packages/metafor/index.html): Meta-Analysis Package for R
* [MetBrewer](https://github.com/BlakeRMills/MetBrewer/tree/main): Color palette package in R inspired by works at the Metropolitan Museum of Art in New York
* [Mfuzz](https://www.bioconductor.org/packages/release/bioc/html/Mfuzz.html): Soft clustering of time series gene expression data
* [mgcv](https://cran.r-project.org/web/packages/mgcv/index.html): Mixed GAM Computation Vehicle with Automatic Smoothness Estimation
* [mi](https://cran.r-project.org/web/packages/mi/index.html): Missing Data Imputation and Model Checking
* [mice](https://cran.r-project.org/web/packages/mice/index.html): Multivariate Imputation by Chained Equations
* [microbenchmark](https://cran.r-project.org/web/packages/microbenchmark/index.html): Accurate Timing Functions
* [mindr](https://cran.r-project.org/web/packages/mindr/index.html): Convert Files Between Markdown or R Markdown Files and Mind Maps
* [mixOmics](http://www.bioconductor.org/packages/release/bioc/html/mixOmics.html): Omics Data Integration Project
* [MixSim](https://cran.r-project.org/web/packages/MixSim/index.html): Simulating Data to Study Performance of Clustering Algorithms
* [mixtools](https://cran.r-project.org/web/packages/mixtools/index.html): Tools for Analyzing Finite Mixture Models
* [mlim](https://cran.r-project.org/web/packages/mlim/index.html): Multiple Imputation with Automated Machine Learning
* [mlr](https://cran.r-project.org/web/packages/mlr/index.html): Machine Learning in R
* [mlr3](https://cran.r-project.org/web/packages/mlr3/index.html): Machine Learning in R - Next Generation
* [mlr3verse](https://cran.rstudio.com/web/packages/mlr3verse/index.html): Easily Install and Load the 'mlr3' Package Family
* [modelbased](https://cran.r-project.org/web/packages/modelbased/index.html): Estimation of Model-Based Predictions, Contrasts and Means
* [modelROC](https://cran.r-project.org/web/packages/modelROC/index.html): Model Based ROC Analysis
* [modelStudio](https://cran.r-project.org/web/packages/modelStudio/index.html): Interactive Studio for Explanatory Model Analysis
* [modeltime](https://cran.r-project.org/web/packages/modeltime/index.html): The Tidymodels Extension for Time Series Modeling
* [moderndive](https://cran.r-project.org/web/packages/moderndive/index.html): Tidyverse-Friendly Introductory Linear Regression
* [MOFA2](https://www.bioconductor.org/packages/release/bioc/html/MOFA2.html): Multi-Omics Factor Analysis v2
* [MoMAColors](https://github.com/BlakeRMills/MoMAColors): Color Schemes based around artwork and the Museum of Modern Art.
* [moonBook](https://cran.r-project.org/web/packages/moonBook/index.html): Functions and Datasets for the Book by Keon-Woong Moon
* [motifStack](https://bioconductor.org/packages/release/bioc/html/motifStack.html): Plot stacked logos for single or multiple DNA, RNA and amino acid sequence
* [msa](https://bioconductor.org/packages/release/bioc/html/msa.html): Multiple Sequence Alignment
* [msigdbr](https://cran.r-project.org/web/packages/msigdbr/index.html): MSigDB Gene Sets for Multiple Organisms in a Tidy Data Format
* [mstate](https://cran.r-project.org/web/packages/mstate/index.html): Data Preparation, Estimation and Prediction in Multi-State Models
* [multcomp](https://cran.r-project.org/web/packages/multcomp/index.html): Simultaneous Inference in General Parametric Models
* [multcompView](https://cran.r-project.org/web/packages/multcompView/index.html): Visualizations of Paired Comparisons
* [multiMiR](https://bioconductor.org/packages/release/bioc/html/multiMiR.html): Integration of multiple microRNA-target databases with their disease and drug associations
* [multiROC](https://cran.r-project.org/web/packages/multiROC/index.html): Calculating and Visualizing ROC and PR Curves Across Multi-Class Classifications
* [musicatk](https://bioconductor.org/packages/release/bioc/html/musicatk.html): Mutational Signature Comprehensive Analysis Toolkit

### N

* [namedropR](https://cran.rstudio.com/web/packages/namedropR/index.html): Create Visual Citations for Presentation Slides
* [naniar](https://cran.r-project.org/web/packages/naniar/index.html): Data Structures, Summaries, and Visualisations for Missing Data
* [NbClust](https://cran.r-project.org/web/packages/NbClust/index.html): Determining the Best Number of Clusters in a Data Set
* [network](https://cran.r-project.org/web/packages/network/index.html): Classes for Relational Data
* [networkD3](https://cran.r-project.org/web/packages/networkD3/index.html): D3 JavaScript Network Graphs from R
* [nextGenShinyApps](https://cran.r-project.org/web/packages/nextGenShinyApps/index.html): Advanced Tools for Building the Next Generation of 'Shiny' Applications and Dashboards
* [ngsReports](https://bioconductor.org/packages/release/bioc/html/ngsReports.html): Load FastqQC reports and other NGS related files
* [nlme](https://cran.r-project.org/web/packages/nlme/index.html): Linear and Nonlinear Mixed Effects Models
* [NMF](https://cran.r-project.org/web/packages/NMF/index.html): Algorithms and Framework for Nonnegative Matrix Factorization (NMF)
* [normSeq](https://arn.ugr.es/normSeq): RNA-seq data normalization and data visualization
* [nparcomp](https://cran.r-project.org/web/packages/nparcomp/index.html): Multiple Comparisons and Simultaneous Confidence Intervals
* [nplyr](https://cran.r-project.org/web/packages/nplyr/index.html): A Grammar of Nested Data Manipulation

### O

* [oaqc](https://cran.r-project.org/web/packages/oaqc/index.html): Computation of the Orbit-Aware Quad Census
* [officedown](https://cran.r-project.org/web/packages/officedown/index.html): Enhanced 'R Markdown' Format for 'Word' and 'PowerPoint'
* [officer](https://cran.r-project.org/web/packages/officer/index.html): Manipulation of Microsoft Word and PowerPoint Documents
* [OmicCircos](https://bioconductor.org/packages/release/bioc/html/OmicCircos.html): High-quality circular visualization of omics data
* [ontologyIndex](https://cran.r-project.org/web/packages/ontologyIndex/index.html): Functions for Reading Ontologies into R
* [openair](https://cran.r-project.org/web/packages/openair/index.html): Tools for the Analysis of Air Pollution Data
* [openxlsx](https://cran.r-project.org/web/packages/openxlsx/index.html): Read, Write and Edit xlsx Files
* [options](https://cran.r-project.org/web/packages/options/): Simple, Consistent Package Options
* [optparse](https://cran.r-project.org/web/packages/optparse/index.html): Command Line Option Parser
* [overviewR](https://cran.r-project.org/web/packages/overviewR/index.html): Easily Extracting Information About Your Data

### P

* [packcircles](https://cran.r-project.org/web/packages/packcircles/index.html): Circle Packing
* [pacman](https://cran.r-project.org/web/packages/pacman/index.html): Package Management Tool
* [pak](https://cran.r-project.org/web/packages/pak/index.html): Another Approach to Package Installation
* [pals](https://cran.r-project.org/web/packages/pals/index.html): Color Palettes, Colormaps, and Tools to Evaluate Them
* [paletteer](https://cran.r-project.org/web/packages/paletteer/index.html): Comprehensive Collection of Color Palettes
* [parabar](https://cran.r-project.org/web/packages/parabar/index.html): Progress Bar for Parallel Tasks
* [parameters](https://cran.r-project.org/web/packages/parameters/index.html): Processing of Model Parameters
* [parttime](https://cran.r-project.org/web/packages/parttime/index.html): Partial Datetime Handling
* [parttree](https://github.com/grantmcdermott/parttree): Simple package for plotting decision tree partitions in R
* [patchwork](https://cran.r-project.org/web/packages/patchwork/index.html): The Composer of ggplots
* [pathview](https://www.bioconductor.org/packages/release/bioc/html/pathview.html): a tool set for pathway based data integration and visualization
* [patternplot](https://cran.r-project.org/web/packages/patternplot/index.html): Versatile Pie Charts, Ring Charts, Bar Charts and Box Plots using Patterns, Colors and Images
* [pbapply](https://cran.rstudio.com/web/packages/pbapply/index.html): Adding Progress Bar to '*apply' Functions
* [PCAtools](https://bioconductor.org/packages/release/bioc/html/PCAtools.html): Everything Principal Components Analysis
* [pdftables](https://cran.r-project.org/web/packages/pdftables/index.html): Programmatic Conversion of PDF Tables
* [pdftools](https://cran.r-project.org/web/packages/pdftools/index.html): Text Extraction, Rendering and Converting of PDF Documents
* [Peptides](https://cran.r-project.org/web/packages/Peptides/index.html): Calculate Indices and Theoretical Physicochemical Properties of Protein Sequences
* [performance](https://cran.r-project.org/web/packages/performance/index.html): Assessment of Regression Models Performance
* [phyloseq](https://www.bioconductor.org/packages/release/bioc/html/phyloseq.html): Handling and analysis of high-throughput microbiome census data
* [phylter](https://cran.r-project.org/web/packages/phylter/index.html): Detect and Remove Outliers in Phylogenomics Datasets
* [phytools](https://cran.r-project.org/web/packages/phytools/index.html): Phylogenetic Tools for Comparative Biology (and Other Things)
* [Pi](https://www.bioconductor.org/packages/release/bioc/html/Pi.html): Leveraging Genetic Evidence to Prioritise Drug Targets at the Gene and Pathway Level
* [PieGlyph](https://cran.r-project.org/web/packages/PieGlyph/index.html): Axis Invariant Scatter Pie Plots
* [pivottabler](https://cran.r-project.org/web/packages/pivottabler/index.html): Create Pivot Tables
* [pkgdown](https://cran.r-project.org/web/packages/pkgdown/index.html): Make Static HTML Documentation for a Package
* [pkgndep](https://cran.r-project.org/web/packages/pkgndep/index.html): Analyze Dependency Heaviness of R Packages
* [pkgverse](https://cran.r-project.org/web/packages/pkgverse/index.html): Build a Meta-Package Universe
* [plot2](https://github.com/grantmcdermott/plot2): Lightweight extension of base R's plot system
* [plot3D](https://cran.r-project.org/web/packages/plot3D/index.html): Plotting Multi-Dimensional Data
* [plotrix](https://cran.r-project.org/web/packages/plotrix/index.html): Various Plotting Functions
* [PMCMRplus](https://cran.r-project.org/web/packages/PMCMRplus/index.html): Calculate Pairwise Multiple Comparisons of Mean Rank Sums Extended
* [png](https://cran.r-project.org/web/packages/png/index.html): Read and write PNG images
* [PNWColors](https://cran.r-project.org/web/packages/PNWColors/index.html): Color Palettes Inspired by Nature in the US Pacific Northwest
* [polyclid](https://polyclid.r-euclid.com/index.html): Polygon and polyline for euclid ‚Ä¢ polyclid
* [poolHelper](https://cran.r-project.org/web/packages/poolHelper/index.html): Simulates Pooled Sequencing Genetic Data
* [poppr](https://cran.r-project.org/web/packages/poppr/index.html): Genetic Analysis of Populations with Mixed Reproduction
* [popsicleR](https://github.com/bicciatolab/PoPsicleR): a flexible R package aimed at providing most of the necessary quality controls and preliminary information for complete and detailed analysis of single cell RNA-seq data.
* [precrec](https://cran.r-project.org/web/packages/precrec/index.html): Calculate Accurate Precision-Recall and ROC (Receiver Operator Characteristics) Curves
* [presenter](https://cran.r-project.org/web/packages/presenter/index.html): Present Data with Style
* [prettydoc](https://cran.r-project.org/web/packages/prettydoc/index.html): Creating Pretty Documents from R Markdown
* [prettyunits](https://cran.r-project.org/web/packages/prettyunits/index.html): Pretty, Human Readable Formatting of Quantities
* [prismatic](https://cran.r-project.org/web/packages/prismatic/index.html): Color Manipulation Tools
* [pROC](https://cran.r-project.org/web/packages/pROC/index.html): Display and Analyze ROC Curves
* [progeny](https://bioconductor.org/packages/release/bioc/html/progeny.html): Pathway RespOnsive GENes for activity inference from gene expression
* [ProjecTILs](https://github.com/carmonalab/ProjecTILs): Interpretation of T cell states using reference single-cell atlases
* [promor](https://cran.r-project.org/web/packages/promor/index.html): Proteomics Data Analysis and Modeling Tools
* [psych](https://cran.r-project.org/web/packages/psych/index.html): Procedures for Psychological, Psychometric, and Personality Research

### Q

* [qgraph](https://cran.r-project.org/web/packages/qgraph/index.html): Graph Plotting Methods, Psychometric Data Visualization and Graphical Model Estimation
* [qqboxplot](https://cran.rstudio.com/web/packages/qqboxplot/index.html): Implementation of the Q-Q Boxplot
* [qqman](https://cran.r-project.org/web/packages/qqman/index.html): Q-Q and Manhattan Plots for GWAS Data
* [qqplotr](https://cran.r-project.org/web/packages/qqplotr/index.html): Quantile-Quantile Plot Extensions for 'ggplot2'
* [qtwAcademic](https://cran.rstudio.com/web/packages/qtwAcademic/index.html): 'Quarto' Website Templates for Academics
* [quanteda](https://cran.r-project.org/web/packages/quanteda/index.html): Quantitative Analysis of Textual Data
* [quantmod](https://cran.r-project.org/web/packages/quantmod/index.html): Quantitative Financial Modelling Framework
* [quantreg](https://cran.r-project.org/web/packages/quantreg/index.html): Quantile Regression
* [qvalue](http://www.bioconductor.org/packages/release/bioc/html/qvalue.html): Q-value estimation for false discovery rate control

### R

* [r2resize](https://cran.r-project.org/web/packages/r2resize/index.html): In-Text Resizing for Containers, Images and Data Tables in 'Shiny', 'Markdown' and 'Quarto' Documents
* [r2rtf](https://cran.r-project.org/web/packages/r2rtf/index.html): Easily Create Production-Ready Rich Text Format (RTF) Table and Figure
* [R.utils](https://cran.r-project.org/web/packages/R.utils/index.html): Various Programming Utilities
* [radarchart](https://cran.r-project.org/web/packages/radarchart/index.html): Radar Chart from 'Chart.js'
* [ragg](https://cran.r-project.org/web/packages/ragg/index.html): Graphic Devices Based on AGG
* [RainCloudPlots](https://github.com/RainCloudPlots/RainCloudPlots): Code and tutorials to visualise your data that is both beautiful *and* statistically valid
* [raincloudplots](https://github.com/jorvlan/raincloudplots): The easy way to create raincloud plots
* [rainette](https://cran.r-project.org/web/packages/rainette/index.html): The Reinert Method for Textual Data Clustering
* [randomcoloR](https://cran.r-project.org/web/packages/randomcoloR/index.html): Generate Attractive Random Colors
* [randomForest](https://cran.r-project.org/web/packages/randomForest/index.html): Breiman and Cutler's Random Forests for Classification and Regression
* [randtoolbox](https://mirrors.sjtug.sjtu.edu.cn/cran/web/packages/randtoolbox/index.html): Toolbox for Pseudo and Quasi Random Number Generation and Random Generator Tests
* [raster](https://cran.r-project.org/web/packages/raster/index.html): Geographic Data Analysis and Modeling
* [rasterVis](https://cran.r-project.org/web/packages/rasterVis/index.html): Visualization Methods for Raster Data
* [rattle](https://cran.r-project.org/web/packages/rattle/index.html): Graphical User Interface for Data Science in R
* [rayshader](https://cran.r-project.org/web/packages/rayshader/index.html): Create Maps and Visualize Data in 2D and 3D
* [rbenchmark](https://cran.r-project.org/web/packages/rbenchmark/index.html): Benchmarking routine for R
* [rcartocolor](https://cran.r-project.org/web/packages/rcartocolor/index.html): 'CARTOColors' Palettes
* [rCharts](https://github.com/ramnathv/rCharts): Interactive JS Charts from R
* [RCircos](https://cran.r-project.org/web/packages/RCircos/index.html): Circos 2D Track Plot
* [RcisTarget](https://www.bioconductor.org/packages/release/bioc/html/RcisTarget.html): RcisTarget Identify transcription factor binding motifs enriched on a list of genes or genomic regions
* [rco](https://github.com/jcrodriguez1989/rco): The R Code Optimizer
* [RColorBrewer](https://cran.r-project.org/web/packages/RColorBrewer/index.html): ColorBrewer Palettes
* [rcrossref](https://cran.r-project.org/web/packages/rcrossref/index.html): Client for Various 'CrossRef' 'APIs'
* [rcssci](https://cran.r-project.org/web/packages/rcssci/index.html): Visualization of Restricted Cubic Splines
* [reactable](https://cran.r-project.org/web/packages/reactable/index.html): Interactive Data Tables Based on 'React Table'
* [reactablefmtr](https://kcuilla.github.io/reactablefmtr/index.html): The {reactablefmtr} package streamlines and enhances the styling and formatting of tables built with the {reactable} **R** package. The {reactablefmtr} package provides many conditional formatters that are highly customizable and easy to use.
* [recipes](https://cran.r-project.org/web/packages/recipes/index.html): Preprocessing Tools to Create Design Matrices
* [refdb](https://cran.r-project.org/web/packages/refdb/index.html): A DNA Reference Library Manager
* [regexplain](https://github.com/gadenbuie/regexplain): üîç An RStudio addin slash regex utility belt
* [regplot](https://cran.r-project.org/web/packages/regplot/index.html): Enhanced Regression Nomogram Plot
* [remorse](https://github.com/matt-dray/remorse): R package: text to Morse Code to audio
* [remotes](https://cran.r-project.org/web/packages/remotes/index.html): R Package Installation from Remote Repositories, Including 'GitHub'
* [renderthis](https://cran.r-project.org/web/packages/renderthis/index.html): Render Slides to Different Formats
* [report](https://cran.r-project.org/web/packages/report/index.html): Automated Reporting of Results and Statistical Models
* [reportROC](https://cran.r-project.org/web/packages/reportROC/index.html): An Easy Way to Report ROC Analysis
* [reprex](https://cran.r-project.org/web/packages/reprex/index.html): Prepare Reproducible Example Code via the Clipboard
* [Rfast](https://cran.r-project.org/web/packages/Rfast/index.html): A Collection of Efficient and Extremely Fast R Functions
* [rfPermute](https://cran.r-project.org/web/packages/rfPermute/index.html): Estimate Permutation p-Values for Random Forest Importance Metrics
* [rgdal](https://cran.r-project.org/web/packages/rgdal/index.html): Bindings for the 'Geospatial' Data Abstraction Library
* [rgl](https://cran.r-project.org/web/packages/rgl/index.html): 3D Visualization Using OpenGL
* [rhub](https://github.com/r-hub/rhub): a multi-platform build and check service for R packages
* [RImagePalette](https://cran.r-project.org/web/packages/RImagePalette/index.html): Extract the Colors from Images
* [rio](https://cran.r-project.org/web/packages/rio/index.html): A Swiss-Army Knife for Data I/O
* [RIdeogram](https://cran.r-project.org/web/packages/RIdeogram/index.html): Drawing SVG Graphics to Visualize and Map Genome-Wide Data on Idiograms
* [RImagePalette](https://github.com/joelcarlson/RImagePalette): Create a representative palette from an image.
* [riskRegression](https://cran.r-project.org/web/packages/riskRegression/index.html): Risk Regression Models and Prediction Scores for Survival Analysis with Competing Risks
* [riverplot](https://cran.r-project.org/web/packages/riverplot/index.html): Sankey or Ribbon Plots
* [rjtools](https://cran.r-project.org/web/packages/rjtools/index.html): Preparing, Checking, and Submitting Articles to the 'R Journal'
* [rlist](https://cran.r-project.org/web/packages/rlist/index.html): A Toolbox for Non-Tabular Data Manipulation
* [rmdformats](https://cran.r-project.org/web/packages/rmdformats/index.html): HTML Output Formats and Templates for 'rmarkdown' Documents
* [rms](https://cran.r-project.org/web/packages/rms/index.html): Regression Modeling Strategies
* [RNASeqR](https://bioconductor.org/packages/release/bioc/html/RNASeqR.html): an R package for automated two-group RNA-Seq analysis workflow
* [RNAseqStat](https://gitmemory.cn/index.php/repo/jisnoalia/RNAseqStat): ‰∏ÄË°å‰ª£Á†ÅÂÆåÊàêËΩ¨ÂΩïÁªÑ‰∏ãÊ∏∏ÂàÜÊûê
* [rnaturalearth](https://cran.r-project.org/web/packages/rnaturalearth/index.html): World Map Data from Natural Earth
* [rnaturalearthdata](https://cran.r-project.org/web/packages/rnaturalearthdata/index.html): World Vector Map Data from Natural Earth Used in 'rnaturalearth'
* [ROCit](https://cran.r-project.org/web/packages/ROCit/index.html): Performance Assessment of Binary Classifier with Visualization
* [ROCR](https://cran.r-project.org/web/packages/ROCR/index.html): Visualizing the Performance of Scoring Classifiers
* [rOpenSci](https://ropensci.org/): open tools for open science
* [ropls](https://bioconductor.org/packages/release/bioc/html/ropls.html): PCA, PLS(-DA) and OPLS(-DA) for multivariate analysis and feature selection of omics data
* [RobustRankAggreg](https://cran.r-project.org/web/packages/RobustRankAggreg/index.html): Methods for robust rank aggregation
* [roughsf](https://cran.r-project.org/web/packages/roughsf/index.html): Visualize Spatial Data using 'roughjs'
* [rpart](https://cran.r-project.org/web/packages/rpart/index.html): Recursive Partitioning and Regression Trees
* [rpivotTable](https://cran.r-project.org/web/packages/rpivotTable/index.html): Build Powerful Pivot Tables and Dynamically Slice & Dice your Data
* [Rsamtools](https://bioconductor.org/packages/release/bioc/html/Rsamtools.html): Binary alignment (BAM), FASTA, variant call (BCF), and tabix file import
* [RSelenium](https://cran.r-project.org/web/packages/RSelenium/index.html): R Bindings for 'Selenium WebDriver'
* [rstan](https://cran.r-project.org/web/packages/rstan/index.html): R Interface to Stan
* [rstatix](https://cran.r-project.org/web/packages/rstatix/index.html): Pipe-Friendly Framework for Basic Statistical Tests
* [rsvg](https://cran.r-project.org/web/packages/rsvg/index.html): Render SVG Images into PDF, PNG, (Encapsulated) PostScript, or Bitmap Arrays
* [RTCGA](https://www.bioconductor.org/packages/release/bioc/html/RTCGA.html): The Cancer Genome Atlas Data Integration
* [rticles](https://cran.r-project.org/web/packages/rticles/index.html): Article Formats for R Markdown
* [rtracklayer](https://bioconductor.org/packages/release/bioc/html/rtracklayer.html): R interface to genome annotation files and the UCSC genome browser
* [Rtsne](https://cran.r-project.org/web/packages/Rtsne/index.html): T-Distributed Stochastic Neighbor Embedding using a Barnes-Hut Implementation
* [runway](https://github.com/ML4LHS/runway): Visualizing Prediction Model Performance
* [RVA](https://cran.r-project.org/web/packages/RVA/index.html): RNAseq Visualization Automation
* [rvcheck](https://cran.rstudio.com/web/packages/rvcheck/index.html): R/Package Version Check, Check latest release version of R and R package (both in 'CRAN', 'Bioconductor' or 'Github')
* [rvest](https://cran.r-project.org/web/packages/rvest/index.html): Easily Harvest (Scrape) Web Pages
* [Rwordseg](https://cran.r-project.org/web/packages/Rwordseg/index.html): Chinese Word Segmentation

### S

* [sankeywheel](https://cran.r-project.org/web/packages/sankeywheel/index.html): Create Dependency Wheels and Sankey Diagrams
* [scAB](https://github.com/jinworks/scAB): scAB detects multiresolution cell states with clinical significance
* [scales](https://cran.r-project.org/web/packages/scales/index.html): Scale Functions for Visualization
* [scatterpie](https://cran.r-project.org/web/packages/scatterpie/index.html): Scatter Pie Plot
* [scatterplot3d](https://cran.r-project.org/web/packages/scatterplot3d/index.html): 3D Scatter Plot
* [scatterPlotMatrix](https://cran.r-project.org/web/packages/scatterPlotMatrix/index.html): 'Htmlwidget' for a Scatter Plot Matrix & Multiple Scatter Plot
* [scCancer](https://github.com/wguo-research/scCancer): A package for automated processing of single cell RNA-seq data in cancer
* [schex](https://www.bioconductor.org/packages/release/bioc/html/schex.html): Hexbin plots for single cell omics data
* [scholar](https://cran.r-project.org/web/packages/scholar/index.html): Analyse Citation Data from Google Scholar
* [scico](https://cran.r-project.org/web/packages/scico/index.html): Colour Palettes Based on the Scientific Colour-Maps
* [Scillus](https://github.com/xmc811/Scillus): R Package for Single-Cell Dataset Processing and Visualization
* [Scissor](https://github.com/sunduanchen/Scissor): Single-Cell Identification of Subpopulations with bulk Sample phenOtype coRrelation
* [scistreer](https://cran.r-project.org/web/packages/scistreer/index.html): Maximum-Likelihood Perfect Phylogeny Inference at Scale
* [scitb](https://cran.r-project.org/web/packages/scitb/index.html): Provides Some Useful Functions for Making Statistical Tables
* [SCORPIUS](https://cran.rstudio.com/web/packages/SCORPIUS/index.html): Inferring Developmental Chronologies from Single-Cell RNA Sequencing Data
* [scPrognosis](https://github.com/XiaomeiLi1/scPrognosis): Single cell transcriptomes improve cancer prognosis
* [SCpubr](https://cran.r-project.org/web/packages/SCpubr/index.html): Generate Publication Ready Visualizations of Single Cell Transcriptomics Data
* [scRNAstat](https://cran.r-project.org/web/packages/scRNAstat/index.html): A Pipeline to Process Single Cell RNAseq Data
* [scRNAtoolVis](https://github.com/junjunlab/scRNAtoolVis): Useful functions to make your scRNA-seq plot more cool!
* [see](https://cran.r-project.org/web/packages/see/index.html): Visualisation Toolbox for 'easystats' and Extra Geoms, Themes and Color Palettes for 'ggplot2'
* [seqinr](https://cran.r-project.org/web/packages/seqinr/index.html): Biological Sequences Retrieval and Analysis
* [seriation](https://cran.r-project.org/web/packages/seriation/index.html): Infrastructure for Ordering Objects Using Seriation
* [sf](https://cran.r-project.org/web/packages/sf/index.html): Simple Features for R
* [shadowtext](https://cran.r-project.org/web/packages/shadowtext/index.html): Shadow Text Grob and Layer
* [shiny](https://cran.r-project.org/web/packages/shiny/index.html): Web Application Framework for R
* [shiny.fluent](https://cran.r-project.org/web/packages/shiny.fluent/index.html): Microsoft Fluent UI for Shiny Apps
* [ShinyCell](https://github.com/SGDDNB/ShinyCell): Shiny Interactive Web Apps for Single-Cell Data
* [shinyPredict](https://cran.r-project.org/web/packages/shinyPredict/index.html): Predictions using Shiny
* [shinythemes](https://cran.r-project.org/web/packages/shinythemes/index.html): Themes for Shiny
* [ShortRead](https://www.bioconductor.org/packages/release/bioc/html/ShortRead.html): FASTQ input and manipulation
* [showtext](https://cran.r-project.org/web/packages/showtext/index.html): Using Fonts More Easily in R Graphs
* [sigminer](https://cran.r-project.org/web/packages/sigminer/index.html): Extract, Analyze and Visualize Mutational Signatures for Genomic Variations
* [simplifyEnrichment](https://bioconductor.org/packages/release/bioc/html/simplifyEnrichment.html): Simplify Functional Enrichment Results
* [simputation](https://cran.r-project.org/web/packages/simputation/index.html): Simple Imputation
* [SingCellaR](https://supatt-lab.github.io/SingCellaR.Doc/index.html): an integrative analysis tool for analysing single-cell RNA sequencing data
* [sjlabelled](https://cran.r-project.org/web/packages/sjlabelled/index.html): Labelled Data Utility Functions
* [sjmisc](https://cran.r-project.org/web/packages/sjmisc/index.html): Data and Variable Transformation Functions
* [sjPlot](https://cran.r-project.org/web/packages/sjPlot/index.html): Data Visualization for Statistics in Social Science
* [skimr](https://cran.r-project.org/web/packages/skimr/index.html): Compact and Flexible Summaries of Data
* [SmartEDA](https://cran.r-project.org/web/packages/SmartEDA/index.html): Summarize and Explore the Data
* [smplot](https://github.com/smin95/smplot): R package for statistical data visualization that complements ggplot2
* [SNFtool](https://cran.r-project.org/web/packages/SNFtool/index.html): Similarity Network Fusion
* [SnowballC](https://cran.r-project.org/web/packages/SnowballC/index.html): Snowball Stemmers Based on the C 'libstemmer' UTF-8 Library
* [sos](https://cran.r-project.org/web/packages/sos/index.html): Search Contributed R Packages, Sort by Package
* [sp](https://cran.r-project.org/web/packages/sp/index.html): Classes and Methods for Spatial Data
* [spatstat](https://cran.r-project.org/web/packages/spatstat/index.html): Spatial Point Pattern Analysis, Model-Fitting, Simulation, Tests
* [spiralize](https://github.com/jokergoo/spiralize): Visualize data on spirals
* [splines2](https://cran.r-project.org/web/packages/splines2/index.html): Regression Spline Functions and Classes
* [stargazer](https://cran.r-project.org/web/packages/stargazer/index.html): Well-Formatted Regression and Summary Statistics Tables
* [statmod](https://cran.r-project.org/web/packages/statmod/index.html): Statistical Modeling
* [stm](https://cran.r-project.org/web/packages/stm/index.html): Estimation of the Structural Topic Model
* [STRINGdb](https://www.bioconductor.org/packages/release/bioc/html/STRINGdb.html): Search Tool for the Retrieval of Interacting proteins database
* [sparklyr](https://spark.rstudio.com/): R interface for Apache Spark
* [stringdist](https://cran.r-project.org/web/packages/stringdist/index.html): Approximate String Matching, Fuzzy Text Search, and String Distance Functions
* [stringr.plus](https://github.com/johncassil/stringr.plus): provides some additional functions not found in stringr for working with strings
* [styler](https://styler.r-lib.org/): Non-Invasive Pretty Printing of R Code
* [summarytools](https://cran.r-project.org/web/packages/summarytools/index.html): Tools to Quickly and Neatly Summarize Data
* [superb](https://cran.r-project.org/web/packages/superb/index.html): Summary Plots with Adjusted Error Bars
* [Surrogate](https://cran.r-project.org/web/packages/Surrogate/index.html): Evaluation of Surrogate Endpoints in Clinical Trials
* [survival](https://cran.r-project.org/web/packages/survival/index.html): Survival Analysis
* [survivalAnalysis](https://cran.r-project.org/web/packages/survivalAnalysis/index.html): High-Level Interface for Survival Analysis and Associated Plots
* [survminer](https://cran.r-project.org/web/packages/survminer/index.html): Drawing Survival Curves using 'ggplot2'
* [survout](https://cran.r-project.org/web/packages/survout/index.html): Excel Conversion of R Surival Analysis Output
* [survRM2](https://cran.r-project.org/web/packages/survRM2/index.html): Comparing Restricted Mean Survival Time
* [Sushi](https://bioconductor.org/packages/release/bioc/html/Sushi.html): Tools for visualizing genomics data
* [susieR](https://cran.r-project.org/web/packages/susieR/index.html): Sum of Single Effects Linear Regression
* [sva](https://bioconductor.org/packages/release/bioc/html/sva.html): Surrogate Variable Analysis
* [survex](https://cran.r-project.org/web/packages/survex/index.html): Explainable Machine Learning in Survival Analysis
* [sysfonts](https://cran.r-project.org/web/packages/sysfonts/index.html): Loading Fonts into R
* [systemfonts](https://cran.r-project.org/web/packages/systemfonts/index.html): System Native Font Finding
* [systemPipeR](https://bioconductor.org/packages/release/bioc/html/systemPipeR.html): NGS workflow and report generation environment
* [systemPipeTools](https://bioconductor.org/packages/release/bioc/html/systemPipeTools.html): Tools for data visualization

### T

* [table1](https://cran.r-project.org/web/packages/table1/index.html): Tables of Descriptive Statistics in HTML
* [tableone](https://cran.r-project.org/web/packages/tableone/index.html): Create 'Table 1' to Describe Baseline Characteristics with or without Propensity Score Weights
* [tablexlsx](https://cran.r-project.org/web/packages/tablexlsx/index.html): Export Data Frames to Excel Workbook
* [tackPlotR](https://github.com/junjunlab/tackPlotR): A R package to visualize bigwigs.
* [tagger](https://github.com/eliocamp/tagger): Adds tags to 'ggpot2' facets
* [TCGAbiolinks](https://bioconductor.org/packages/release/bioc/html/TCGAbiolinks.html): An R/Bioconductor package for integrative analysis with GDC data
* [TCGAmutations](https://github.com/PoisonAlien/TCGAmutations): R data package for pre-compiled somatic mutations from TCGA cohorts (from Broad Firehose and TCGA MC3 Project)
* [TCGAplot](https://github.com/tjhwangxiong/TCGAplot): A number of functions were generated to perform pan-cancer DEG analysis, correlation analysis between gene expression and TMB, MSI, TIME, and promoter methylation. Methods for visualization were provided in order to easily perform integrative pan-cancer multi-omics analysis.
* [TCGAutils](https://bioconductor.org/packages/release/bioc/html/TCGAutils.html): A suite of helper functions for checking and manipulating TCGA data including data obtained from the curatedTCGAData experiment package. These functions aim to simplify and make working with TCGA data more manageable.
* [TCseq](https://bioconductor.org/packages/release/bioc/html/TCseq.html): Time course sequencing data analysis
* [tensorflow](https://cran.r-project.org/web/packages/tensorflow/index.html): R Interface to 'TensorFlow'
* [terra](https://cran.r-project.org/web/packages/terra/index.html): Spatial Data Analysis
* [tesseract](https://cran.r-project.org/web/packages/tesseract/index.html): Open Source OCR Engine
* [tesseractgt](https://github.com/arcruz0/tesseractgt): An R package to generate ground truth data for Tesseract
* [textir](https://cran.r-project.org/web/packages/textir/index.html): Inverse Regression for Text Analysis
* [textplot](https://cran.r-project.org/web/packages/textplot/index.html): Text Plots
* [thematic](https://cran.r-project.org/web/packages/thematic/index.html): Unified and Automatic 'Theming' of 'ggplot2', 'lattice', and 'base' R Graphics
* [tictoc](https://cran.r-project.org/web/packages/tictoc/index.html): Functions for Timing R Scripts, as Well as Implementations of "Stack" and "List" Structures
* [tidybayes](https://cran.r-project.org/web/packages/tidybayes/index.html): Tidy Data and 'Geoms' for Bayesian Models
* [tidybulk](https://www.bioconductor.org/packages/release/bioc/html/tidybulk.html): Brings transcriptomics to the tidyverse
* [tidyCDISC](https://cran.r-project.org/web/packages/tidyCDISC/index.html): Quick Table Generation & Exploratory Analyses on ADaM-Ish Datasets
* [tidycmprsk](https://cran.r-project.org/web/packages/tidycmprsk/index.html): Competing Risks Estimation
* [tidydr](https://cran.r-project.org/web/packages/tidydr/index.html): Unify Dimensionality Reduction Results
* [tidyfit](https://cran.r-project.org/web/packages/tidyfit/index.html): Regularized Linear Modeling with Tidy Data
* [tidyfst](https://cran.r-project.org/web/packages/tidyfst/index.html): Tidy Verbs for Fast Data Manipulation
* [tidygraph](https://cran.r-project.org/web/packages/tidygraph/index.html): A Tidy API for Graph Manipulation
* [TidyMass](https://www.tidymass.org/): a comprehensive computational framework that can process the whole workflow of data processing and analysis for LC-MS-based untargeted metabolomics using [tidyverse](https://www.tidyverse.org/) principles
* [tidymodels](https://cran.r-project.org/web/packages/tidymodels/index.html): Easily Install and Load the 'Tidymodels' Packages
* [tidyquant](https://cran.r-project.org/web/packages/tidyquant/index.html): Tidy Quantitative Financial Analysis
* [tidytext](https://cran.r-project.org/web/packages/tidytext/index.html): Text Mining using 'dplyr', 'ggplot2', and Other Tidy Tools
* [tidytidbits](https://cran.r-project.org/web/packages/tidytidbits/index.html): A Collection of Tools and Helpers Extending the Tidyverse
* [tidyverse](https://cran.r-project.org/web/packages/tidyverse/index.html): Easily Install and Load the 'Tidyverse'
* [timeROC](https://cran.r-project.org/web/packages/timeROC/index.html): Time-Dependent ROC Curve and AUC for Censored Survival Data
* [timetk](https://cran.r-project.org/web/packages/timetk/index.html): A Tool Kit for Working with Time Series in R
* [tip](https://cran.r-project.org/web/packages/tip/index.html): ayesian Clustering Using the Table Invitation Prior (TIP)
* [tm](https://cran.r-project.org/web/packages/tm/index.html): Text Mining Package
* [tmap](https://cran.r-project.org/web/packages/tmap/index.html): Thematic Maps
* [toastui](https://cran.r-project.org/web/packages/toastui/index.html): Interactive Tables, Calendars and Charts for the Web
* [transPlotR](https://github.com/junjunlab/transPlotR): An elegant package to visualize gene structures
* [TreeAndLeaf](https://bioconductor.org/packages/release/bioc/html/TreeAndLeaf.html): Displaying binary trees with focus on dendrogram leaves
* [treemap](https://cran.r-project.org/web/packages/treemap/index.html): Treemap Visualization
* [treemapify](https://cran.r-project.org/web/packages/treemapify/index.html): Draw Treemaps in 'ggplot2'
* [trend](https://cran.r-project.org/web/packages/trend/index.html): Non-Parametric Trend Tests and Change-Point Detection
* [TSstudio](https://cran.r-project.org/web/packages/TSstudio/index.html): Functions for Time Series Analysis and Forecasting
* [tufte](https://cran.r-project.org/web/packages/tufte/index.html): Tufte's Styles for R Markdown Documents
* [tvthemes](https://cran.r-project.org/web/packages/tvthemes/index.html): TV Show Themes and Color Palettes for 'ggplot2' Graphics
* [TwoSampleMR](https://github.com/MRCIEU/TwoSampleMR): Mendelian randomization with GWAS summary data

### U

* [UCSCXenaShiny](https://cran.r-project.org/web/packages/UCSCXenaShiny/index.html): Interactive Analysis of UCSC Xena Data
* [UCSCXenaTools](https://cran.r-project.org/web/packages/UCSCXenaTools/index.html): Download and Explore Datasets from UCSC Xena Data Hubs
* [UpSetR](https://cran.r-project.org/web/packages/UpSetR/index.html): A More Scalable Alternative to Venn and Euler Diagrams for Visualizing Intersecting Sets

### V

* [VariantAnnotation](https://bioconductor.org/packages/release/bioc/html/VariantAnnotation.html): Annotation of Genetic Variants
* [vcfR](https://cran.r-project.org/web/packages/vcfR/index.html): Manipulate and Visualize VCF Data
* [vegan](https://cran.r-project.org/web/packages/vegan/index.html): Community Ecology Package
* [venn](https://cran.r-project.org/web/packages/venn/index.html): Draw Venn Diagrams
* [VennDiagram](https://cran.r-project.org/web/packages/VennDiagram/index.html): Generate High-Resolution Venn and Euler Plots
* [viewenhance](https://github.com/kieranjmartin/viewenhance): An add in for RStudio which allows the user to filter and select columns interactively
* [ViewPipeSteps](https://github.com/daranzolin/ViewPipeSteps): Create tabs of View() output for each chained pipe
* [vidger](https://www.bioconductor.org/packages/release/bioc/html/vidger.html): Create rapid visualizations of RNAseq data in R
* [VIM](https://cran.r-project.org/web/packages/VIM/index.html): Visualization and Imputation of Missing Values
* [vioplot](https://cran.r-project.org/web/packages/vioplot/index.html): Violin Plot
* [viper](https://bioconductor.org/packages/release/bioc/html/viper.html): Virtual Inference of Protein-activity by Enriched Regulon analysis
* [viridis](https://cran.r-project.org/web/packages/viridis/index.html): Default Color Maps from 'matplotlib'
* [viscomp](https://cran.r-project.org/web/packages/viscomp/index.html): Visualize Multi-Component Interventions in Network Meta-Analysis
* [visdat](https://cran.r-project.org/web/packages/visdat/index.html): Preliminary Visualisation of Data
* [visNetwork](https://cran.r-project.org/web/packages/visNetwork/index.html): Network Visualization using 'vis.js' Library
* [visreg](https://pbreheny.github.io/visreg/): An R package for the visualization of regression models
* [vissE](https://bioconductor.org/packages/release/bioc/html/vissE.html): Visualising Set Enrichment Analysis Results
* [vitae](https://cran.r-project.org/web/packages/vitae/index.html): Curriculum Vitae for R Markdown
* [VoronoiPlus](https://github.com/AllanCameron/VoronoiPlus): Create weighted Voronoi maps and treemaps
* [voronoiTreemap](https://cran.rstudio.com/web/packages/voronoiTreemap/index.html): Voronoi Treemaps with Added Interactivity by Shiny
* [vroom](https://cran.r-project.org/web/packages/vroom/index.html): Read and Write Rectangular Text Data Quickly
* [VSOLassoBag](https://cran.r-project.org/web/packages/VSOLassoBag/index.html): Variable Selection Oriented LASSO Bagging Algorithm
* [VSURF](https://cran.r-project.org/web/packages/VSURF/index.html): Variable Selection Using Random Forests
* [vtreat](https://cran.r-project.org/web/packages/vtreat/index.html): A Statistically Sound 'data.frame' Processor/Conditioner

### W

* [waldo](https://cran.r-project.org/web/packages/waldo/index.html): Find Differences Between R Objects
* [webpea](https://github.com/nucleic-acid/webpea): Tool to save R plots in WebP format
* [wesanderson](https://cran.r-project.org/web/packages/wesanderson/index.html): A Wes Anderson Palette Generator
* [wget](https://github.com/YuLab-SMU/wget): Setting Download Method to "wget -c"
* [wiggleplotr](https://bioconductor.org/packages/release/bioc/html/wiggleplotr.html): Make read coverage plots from BigWig files
* [wikiprofiler](https://cran.r-project.org/web/packages/wikiprofiler/index.html): 'WikiPathway' Based Data Integration and Visualization
* [wordcloud](https://cran.r-project.org/web/packages/wordcloud/index.html): Word Clouds
* [wordcloud2](https://cran.r-project.org/web/packages/wordcloud2/index.html): Create Word Cloud by 'htmlwidget'
* [workflowr](https://cran.r-project.org/web/packages/workflowr/index.html): A Framework for Reproducible and Collaborative Data Science
* [wzRfun](https://github.com/walmes/wzRfun): Auxiliary functions for courses, analysis and data visualization.]

### X

* [xaringan](https://cran.r-project.org/web/packages/xaringan/index.html): Presentation Ninja
* [xgboost](https://cran.r-project.org/web/packages/xgboost/index.html): Extreme Gradient Boosting
* [xlsx](https://cran.r-project.org/web/packages/xlsx/index.html): Read, Write, Format Excel 2007 and Excel 97/2000/XP/2003 Files
* [xkcd](https://cran.r-project.org/web/packages/xkcd/index.html): Plotting ggplot2 Graphics in an XKCD Style
* [XML](https://cran.r-project.org/web/packages/XML/index.html): Tools for Parsing and Generating XML Within R and S-Plus
* [xSelectCells](https://github.com/vikkki/xSelectCells): A light weight shiny app tool to enable manually selection of cells from seurat object.
* [xts](https://cran.r-project.org/web/packages/xts/index.html): eXtensible Time Series

### Y

* [YAPSA](https://www.bioconductor.org/packages/release/bioc/html/YAPSA.html): Yet Another Package for Signature Analysis
* [ymlthis](https://cran.r-project.org/web/packages/ymlthis/index.html): Write 'YAML' for 'R Markdown', 'bookdown', 'blogdown', and More

---

| R ÂåÖÂêçÁß∞        | ‰∏ªË¶ÅÂäüËÉΩ                       |
| --------------- | ------------------------------ |
| devtools        | R ÂåÖÂÆâË£Ö                       |
| BiocManager     | R ÂåÖÂÆâË£Ö                       |
| tidyverse       | Êï∞ÊçÆÊï¥ÁêÜÂèäÂèØËßÜÂåñÂ∑•ÂÖ∑ÂåÖÈõÜÂêà     |
| future          | ÊîØÊåÅÂπ∂Ë°åËÆ°ÁÆóÁöÑÂåÖ               |
| future.apply    | ÊîØÊåÅÂπ∂Ë°åËÆ°ÁÆóÁöÑÂåÖ               |
| Seurat          | ÂçïÁªÜËÉûÂàÜÊûêÂü∫Á°ÄÂ∑•ÂÖ∑ÂåÖ           |
| SeuratData      | ÂçïÁªÜËÉûÊï∞ÊçÆ                     |
| SeuratWrappers  | Êèê‰æõÂÖ∂‰ªñ R ÂåÖ‰∏é Seurat ÁöÑÊé•Âè£  |
| SingleR         | ÁªÜËÉûÈâ¥ÂÆö                       |
| batchelor       | ÊâπÊ¨°Ê†°Ê≠£                       |
| harmony         | ÊâπÊ¨°Ê†°Ê≠£                       |
| liger           | ÊâπÊ¨°Ê†°Ê≠£ÔºåÊï∞ÊçÆÊï¥Âêà             |
| limma           | Â∑ÆÂºÇÂàÜÊûê                       |
| edgeR           | Â∑ÆÂºÇÂàÜÊûê                       |
| DESeq2          | Â∑ÆÂºÇÂàÜÊûê                       |
| MAST            | Â∑ÆÂºÇÂàÜÊûê                       |
| GSVA            | Âü∫Âõ†ÈõÜÂèòÂºÇÂàÜÊûê                 |
| msigdbr         | Êèê‰æõÈùû‰∫∫Áâ©ÁßçÁöÑ MSigDB Âü∫Âõ†ÈõÜ   |
| clusterProfiler | Âü∫Âõ†ÈõÜÂØåÈõÜÂàÜÊûê                 |
| scRepertoire    | ÂÖçÁñ´ÁªÑÂ∫ìÂàÜÊûê                   |
| survival        | ÁîüÂ≠òÂàÜÊûê                       |
| survminer       | ÁîüÂ≠òÂàÜÊûê                       |
| CellChat        | ÁªÜËÉûÈÄöËÆØ                       |
| monocle         | ÊãüÊó∂ËΩ®ËøπÂàÜÊûê                   |
| monocle3        | ÊãüÊó∂ËΩ®ËøπÂàÜÊûê                   |
| velocyto.R      | RNA ÈÄüÁéáÂàÜÊûêÔºåÁ±ª‰ººÊãüÊó∂ËΩ®ËøπÂàÜÊûê |
| infercnv        | ÂçïÁªÜËÉû CNV ÂàÜÊûê                |
| nichenetr       | ÁªÜËÉûÈÄöËÆØ                       |
| garnett         | ÁªÜËÉûÂàÜÁ±ª                       |
| DoubletFinder   | ËØÜÂà´ doublets                  |
| WGCNA           | ÂÖ±Ë°®ËææÊùÉÈáçÁΩëÁªúÂàÜÊûê             |
| Signac          | scATAC ÂàÜÊûê                    |
| biomaRt         | Âü∫Âõ† ID ÂèäÂêåÊ∫êÂü∫Âõ†ËΩ¨Êç¢         |
| CytoTRACE       | ËΩ®ËøπÂàÜÊûê                       |
| slingshot       | ËΩ®ËøπÂàÜÊûê                       |
| copykat         | ÂçïÁªÜËÉû CNV ÂàÜÊûê                |
| xCell           | bulkRNA ÂéªÂç∑ÁßØ                 |
| MuSiC           | bulkRNA ÂéªÂç∑ÁßØ                 |
| SCDC            | bulkRNA ÂéªÂç∑ÁßØ                 |
| SPOTlight       | Á©∫Èó¥ËΩ¨ÂΩïÁªÑÂàÜÊûê                 |
| ggpubr          | ÁîªÂõæ                           |

## TeX
* [texshade](https://ctan.org/pkg/texshade): Package for setting nucleotide and peptide alignments

## Perl
* [Data::Dumper](https://metacpan.org/pod/Data::Dumper): stringified perl data structures, suitable for both printing and eval
* [Data::Printer](https://metacpan.org/pod/Data::Printer): colored pretty-print of Perl data structures and objects
* [Getopt::Long](https://metacpan.org/pod/Getopt::Long): Extended processing of command line options
* [List::AllUtils](https://metacpan.org/pod/List::AllUtils): Combines List::Util, List::SomeUtils and List::UtilsBy in one bite-sized package
* [List::MoreUtils](https://metacpan.org/pod/List::MoreUtils): Provide the stuff missing in List::Util 
* [List::Util](https://metacpan.org/pod/List::Util): A selection of general-utility list subroutines
* [List::UtilsBy](https://metacpan.org/pod/List::UtilsBy): higher-order list utility functions 
* [Parallel::ForkManager](https://metacpan.org/pod/Parallel::ForkManager): A simple parallel processing fork manager
* [Path::Tiny](https://metacpan.org/pod/Path::Tiny): File path utility
* [Try::Tiny](https://metacpan.org/pod/Try::Tiny): Minimal try/catch with proper preservation of $@




## Tools
* [bioawk](https://github.com/lh3/bioawk): BWK awk modified for biological data
* [GraPhlAn](https://bitbucket.org/nsegata/graphlan/wiki/Home): a software tool for producing high-quality circular representations of taxonomic and phylogenetic trees
* [IQ-TREE](http://www.iqtree.org/): Efficient software for phylogenomic inference
* [Recharts](http://recharts.org/en-US/): A composable charting library built on React components
* [pyGenomeTracks](https://github.com/deeptools/pyGenomeTracks): Standalone program and library to plot beautiful genome browser tracks
* [miRWalk](http://mirwalk.umm.uni-heidelberg.de/)
* [miRSystem](http://mirsystem.cgm.ntu.edu.tw/): miRNA integration system for Target gene prediction
* [Heatmapper](http://www.heatmapper.ca/expression/): a freely available web server that allows users to interactively visualize their data in the form of heat maps through an easy-to-use graphical interface
* [jvenn](http://jvenn.toulouse.inra.fr/app/index.html): an interactive Venn diagram viewer
* [webtools-Venn](http://bioinformatics.psb.ugent.be/webtools/Venn/): draw venn diagram
* [EVenn](http://www.ehbio.com/test/venn/#/): Easy to create repeatable, editable, and statistically measurable Venn diagrams online.
* [NetworkAnalyst](https://www.networkanalyst.ca/): comprehensive gene expression analysis, meta-analysis & network biology
* [BioJupies](https://amp.pharm.mssm.edu/biojupies/): Automatically Generates RNA-seq Data Analysis Notebooks
* [bioDBnet](https://biodbnet-abcc.ncifcrf.gov/): biological DataBase network
* [iTOL](https://itol.embl.de/): Interactive Tree Of Life
* [ETE](http://etetoolkit.org/treeview/): Phylogenetic tree (newick) viewer, Âú®Á∫øÂèØËßÜÂåñËøõÂåñÊ†ëÂíåÂ§öÂ∫èÂàóÊØîÂØπ
* [easyGEO](https://easygeo.cn/#/): ‰∏ÄÁ´ôÂºè GEO Êï∞ÊçÆÂàÜÊûêÂπ≥Âè∞
* [Quickomics](http://quickomics.bxgenomics.com/): Ë°®ËææÊï∞ÊçÆÂàÜÊûêÂ∑•ÂÖ∑
* [EvolView](https://www.evolgenius.info/evolview/#login): a webserver for visualization, annotation, and management of phylogenetic trees
* [kingfisher-download](https://github.com/wwood/kingfisher-download): Easier download/extract of FASTA/Q read data from the ENA, NCBI, AWS or GCP.
* [PathwayMapper](https://www.pathwaymapper.org/): a collaborative visual web editor for cancer pathways and genomic data
* [kingfisher-download](https://github.com/wwood/kingfisher-download): Easier download/extract of FASTA/Q read data and metadata from the ENA, NCBI, AWS or GCP.
* [parallel-fastq-dump](https://github.com/rvalieris/parallel-fastq-dump): parallel fastq-dump wrapper
* [Refgenie](http://refgenie.databio.org/en/latest/): Refgenie manages storage, access, and transfer of reference genome resources. It provides command-line and Python interfaces to *download* pre-built reference genome "assets", like indexes used by bioinformatics tools. It can also *build* assets for custom genome assemblies. Refgenie provides programmatic access to a standard genome folder structure, so software can swap from one genome to another.
* [ImageGP](http://www.ehbio.com/ImageGP/): An easy-to-use data visualization web server for scientific researchers
* [TISIDB](http://cis.hku.hk/TISIDB/): an integrated repository portal for tumor-immune system interactions
* [Êü•ÊñáÁåÆ - paperneed](https://www.paperneed.cn/articles/search)
* [Ê¶ÇÊôÆÂ≠¶ÊúØÂØºËà™](https://gapscholar.com/)
* [‰∏ÄÁ´ôÂºè GEO Êï∞ÊçÆÂàÜÊûêÂπ≥Âè∞ÔºöÂÖ®ÁïåÈù¢ÂåñÊìç‰ΩúÔºåËÆ© GEO Êï∞ÊçÆÂàÜÊûêÊõ¥ÂÆπÊòì - easyGEO](https://easygeo.cn/)
* [ÁßëÂõæ SHOW | Âø´ÈÄü‰ªé PDF ‰∏≠ÊèêÂèñÂõæÁâá](https://ktshow.gaptools.cn/#/)
* [ÊòìËΩØËëó](https://erz.gaptools.cn/)
* [ÂåªÁîüËøõÈò∂ÂøÖÁúãÁöÑÂçÅÂ§ßÁßëÁ†îÁΩëÁ´ô](https://mp.weixin.qq.com/s/9a4kPrYxUF-zwfcaH2pAvA)
* [SMART - Servier Medical ART](https://smart.servier.com/)
* [circMine](http://www.biomedical-web.com/circmine/home)
* [JinmiaoChenLab/DISCOtoolkit](https://github.com/JinmiaoChenLab/DISCOtoolkit): DISCOtoolkit is an R package that allows users to access data and use the tools provided by the [DISCO database](https://www.immunesinglecell.org/).
* [Â§öÁªÑÂ≠¶Êï¥ÂêàÂàÜÊûêÁöÑ‰∏Ä‰∫õ R ÂåÖ](https://mp.weixin.qq.com/s/RQwUJavkUlfQFiTwpvzd0g)

## scRNA
* [**awesome-single-cell**](https://github.com/seandavi/awesome-single-cell)**: Community-curated list of software packages and data resources for single-cell, including RNA-seq, ATAC-seq, etc.**
* [**awesome-multi-omics**](https://github.com/mikelove/awesome-multi-omics)**: List of software packages for multi-omics analysis**
* [**scRNA-tools**](https://www.scrna-tools.org/table): The scRNA-tools database is a catalogue of tools for analysing single-cell RNA sequencing data.
* [scrnapip](https://github.com/OpenGene/scrnapip): A Systematic and Dynamic Pipeline for Single-Cell RNA Sequencing Analysis
* [scater](https://bioconductor.org/packages/release/bioc/html/scater.html): Single-Cell Analysis Toolkit for Gene Expression Data in R
* [Seurat](https://cran.r-project.org/web/packages/Seurat/index.html): Tools for Single Cell Genomics
* [monocle](https://www.bioconductor.org/packages/release/bioc/html/monocle.html): Clustering, differential expression, and trajectory analysis for single- cell RNA-Seq
* [cellassign](https://github.com/irrationone/cellassign): Automated, probabilistic assignment of cell types in scRNA-seq data
* [DRAGON](https://github.com/arc85/dragonsc): Deterministic annealing gaussian mixtures models for single cell RNAseq
* [singleseqgset](https://github.com/arc85/singleseqgset): Gene set enrichment analysis for single-cell RNAseq data
* [celltalker](https://github.com/arc85/celltalker): Inference of cell-cell communication from single-cell RNAseq data
* [DPT](https://bioinformaticshome.com/tools/rna-seq/descriptions/DPT.html): estimate the temporal order of differentiating cell in single-cell RNA-seq (scRNA-seq) data
* [diffusionMap](https://github.com/rcannood/diffusionMap)
* [destiny](http://www.bioconductor.org/packages/release/bioc/html/destiny.html): Creates diffusion maps
* [iCellR](https://cran.r-project.org/web/packages/iCellR/index.html): Analyzing High-Throughput Single Cell Sequencing Data
* [presto](https://github.com/immunogenomics/presto): Presto performs a fast Wilcoxon rank sum test and auROC analysis
* [harmony](https://github.com/immunogenomics/harmony): Fast, sensitive and accurate integration of single-cell data with Harmony
* [ESTIMATE](https://bioinformatics.mdanderson.org/public-software/estimate/): Estimation of STromal and Immune cells in MAlignant Tumor tissues using Expression data
* [xCell](https://github.com/dviraran/xCell): cell types enrichment analysis
* [SingleR](https://bioconductor.org/packages/release/bioc/html/SingleR.html): Reference-Based Single-Cell RNA-Seq Annotation
* [CIBERSORTx]( impute gene expression profiles and provide an estimation of the abundances of member cell types in a mixed cell population, using gene expression data)
* [CellPhoneDB](https://github.com/Teichlab/cellphonedb): CellPhoneDB can be used to search for a particular ligand/receptor, or interrogate your own single-cell transcriptomics data.
* [clustermole](https://cran.r-project.org/web/packages/clustermole/index.html): Unbiased Single-Cell Transcriptomic Data Cell Type Identification
* [CellChat](https://github.com/sqjin/CellChat/): Inference and analysis of cell-cell communication
* [iTALK](https://github.com/Coolgenome/iTALK): is an R toolkit for characterizing and illustrating intercellular communication
* [SciBet](http://scibet.cancer-pku.cn/): Single Cell Identificator Based on E-test (SciBet) is a computational tool for predicting cell identity for any randomly sequenced cell by single cell RNA sequencing technique
* [CHETAH](https://github.com/jdekanter/CHETAH): a selective, hierarchical cell type identification method for single-cell RNA sequencing
* [dyno](https://github.com/dynverse/dyno): Inferring trajectories with 60 TI methods
* [Scanpy](https://github.com/theislab/scanpy): Single-Cell Analysis in Python
* [singleseqgset](https://github.com/arc85/singleseqgset): package for gene set enrichment analysis for single-cell RNAseq data
* [clustree](https://cran.r-project.org/web/packages/clustree/index.html): Visualise Clusterings at Different Resolutions
* [batchelor](https://bioconductor.org/packages/release/bioc/html/batchelor.html): Single-Cell Batch Correction Methods
* [presto](https://github.com/immunogenomics/presto): Presto performs a fast Wilcoxon rank sum test and auROC analysis. Latest benchmark ran 1 million observations, 1K features, and 10 groups in 16 seconds (sparse input) and 85 seconds (dense input).
* [SoupX](https://cran.r-project.org/web/packages/SoupX/index.html): Single Cell mRNA Soup eXterminator
* [liger](https://github.com/MacoskoLab/liger): Linked Inference of Genomic Experimental Relationships, LIGER (`liger`) is a package for integrating and analyzing multiple single-cell datasets
* [RcisTarget](https://www.bioconductor.org/packages/release/bioc/html/RcisTarget.html): Identify transcription factor binding motifs enriched on a gene list
* [AUCell](http://bioconductor.org/packages/release/bioc/html/AUCell.html): Analysis of 'gene set' activity in single-cell RNA-seq data (e.g. identify cells with specific gene signatures)
* [Giotto](http://spatialgiotto.rc.fas.harvard.edu/): The Giotto package consists of two modules, Giotto Analyzer and Viewer, which provide tools to process, analyze and visualize single-cell spatial expression data. https://rubd.github.io/Giotto_site/index.html
* [BayesSpace](https://bioconductor.org/packages/release/bioc/html/BayesSpace.html): Clustering and Resolution Enhancement of Spatial Transcriptomes
* [spdep](https://cran.r-project.org/web/packages/spdep/index.html): Spatial Dependence: Weighting Schemes, Statistics
* [nichenetr](https://github.com/saeyslab/nichenetr): the R implementation of the NicheNet method
* [SPOTlight](https://github.com/MarcElosua/SPOTlight): The goal of **SPOTlight** is to provide a tool that enables the deconvolution of cell types and cell type proportions present within each capture locations comprising mixtures of cells, originally developed for 10X‚Äôs Visium - spatial trancsiptomics- technology, it can be used for all technologies returning mixtures of cells.
* [iCellR](https://cran.r-project.org/web/packages/iCellR/index.html): Analyzing High-Throughput Single Cell Sequencing Data
* [ncborcherding/escape](https://github.com/ncborcherding/escape): Easy single cell analysis platform for enrichment
* [xCell](https://github.com/dviraran/xCell): cell types enrichment analysis
* [stLearn](https://stlearn.readthedocs.io/en/latest/):  a downstream analysis toolkit for Spatial Transcriptomics data
* [SPATA](https://themilolab.github.io/SPATA/index.html): A Toolbox for Spatial Gene Expression Analysis
* [celldex](https://bioconductor.org/packages/release/data/experiment/html/celldex.html): Reference Index for Cell Types
* [scCancer](https://github.com/wguo-research/scCancer): A package for automated processing of single cell RNA-seq data in cancer
* [UCell](https://github.com/carmonalab/UCell): An R package for single-cell gene signature scoring
* [metacell](https://github.com/tanaylab/metacell): Metacell - Single-cell mRNA Analysis
* [RCTD](https://github.com/dmcable/RCTD): Robust Cell Type Decomposition
* [RCAv2](https://github.com/prabhakarlab/RCAv2): R package for robust clustering of single cell RNA sequencing data
* [seurat-wrappers](https://github.com/satijalab/seurat-wrappers): Community-provided extensions to Seurat
* [seurat-disk](https://github.com/mojaveazure/seurat-disk): Interfaces for HDF5-based Single Cell File Formats
* [scVelo](https://scvelo.readthedocs.io/): RNA velocity generalized through dynamical modeling
* [DeepSEM](https://github.com/HantaoShu/DeepSEM): a deep-learning-based approach with novel neural network architecture that can infer gene regulatory network, embed scRNA-seq data, and simulate realistic scRNA-seq data by interpreting different modules
* [velocyto.R](https://github.com/velocyto-team/velocyto.R): RNA velocity estimation in R
* [SPOTlight](https://github.com/MarcElosua/SPOTlight): Spatial Transcriptomics Capture Location Deconvolution
* [stereoscope](https://github.com/almaan/stereoscope): Spatial mapping of cell types by integration of transcriptomics data
* [ArchR]: Analysis of Regulatory Chromatin in R (www.ArchRProject.com)
* [SnapATAC](https://github.com/r3fang/SnapATAC): Analysis Pipeline for Single Cell ATAC-seq
* [signac](https://github.com/timoast/signac): R toolkit for the analysis of single-cell chromatin data
* [dynplot](https://github.com/dynverse/dynplot): Visualising single-cell trajectories, including comparisons between two models üìà
* [dorothea](https://bioconductor.org/packages/release/data/experiment/html/dorothea.html): Collection Of Human And Mouse TF Regulons
* [metacell](https://github.com/tanaylab/metacell): Metacell - Single-cell mRNA Analysis
* [ShinyCell](https://github.com/SGDDNB/ShinyCell#installation): Shiny Interactive Web Apps for Single-Cell Data
* [SCEVAN](https://github.com/AntonioDeFalco/SCEVAN): R package that automatically classifies the cells in the scRNA data by segregating non-malignant cells of tumor microenviroment from the malignant cells. It also infers the copy number profile of malignant cells, identifies subclonal structures and analyses the specific and shared alterations of each subpopulation.
* [projectLSI](https://github.com/sajuukLyu/projectLSI): Project bulk/single-cell RNA-seq data to given LSI space
* [Teichlab/celltypist](https://github.com/Teichlab/celltypist): A tool for semi-automatic cell type classification, harmonization and integration
* [CuratedAtlasQueryR](https://stemangiola.github.io/CuratedAtlasQueryR/): Queries the Human Cell Atlas

### Êï∞ÊçÆÊï¥Âêà

* [Seurat](https://satijalab.org/seurat/): Tools for Single Cell Genomics
* [harmony](https://portals.broadinstitute.org/harmony/index.html): Fast, Sensitive, and Accurate Integration of Single Cell Data
* [batchelor](https://www.bioconductor.org/packages/release/bioc/html/batchelor.html): Single-Cell Batch Correction Methods
* [rliger](https://cran.r-project.org/web/packages/rliger/index.html): Linked Inference of Genomic Experimental Relationships
* [simspec](https://github.com/quadbiolab/simspec): Calculation of Reference/Cluster Similarity Spectrum (RSS/CSS)
* [tanaylab/metacells: Metacells - Single-cell RNA Sequencing Analysis](https://github.com/tanaylab/metacells)
* [navinlabcode/CellTrek](https://github.com/navinlabcode/CellTrek): a computational framework that can directly map single cells back to their spatial coordinates in tissue sections based on scRNA-seq and ST data

### Doublets Èâ¥ÂÆö

* [scrublet](https://github.com/swolock/scrublet): Detect doublets in single-cell RNA-seq data
* [DoubletFinder](https://github.com/chris-mcginnis-ucsf/DoubletFinder): R package for detecting doublets in single-cell RNA sequencing data

### Ê≥®ÈáäÁªÜËÉûÁ±ªÂûã

* [SingleR](https://bioconductor.org/packages/release/bioc/html/SingleR.html): Reference-Based Single-Cell RNA-Seq Annotation
* [scAnnotate](https://cran.r-project.org/web/packages/scAnnotate/index.html): An Automated Cell Type Annotation Tool for Single-Cell RNA-Sequencing Data
* [ToppCell](https://toppcell.cchmc.org/biosystems/go/index3/): A Hierarchical Modular Single Cell Gene Expression Analysis System
* [CellKb Immune - Immune cell type markers](https://www.cellkb.com/immune): Database of mouse hematopoietic cell types
* [sc-type](https://sctype.app/): **ScType** is a computational platform, which enables data-driven, fully-automated and ultra-fast cell-type identification based solely on given scRNA-seq data, combined with our comprehensive cell marker database as background information.
* [ACT](http://xteam.xbio.top/ACT/)Ôºö Annotation of Cell TypesÔºå[ACT](http://biocc.hrbmu.edu.cn/ACT/)
* [Azimuth](https://azimuth.hubmapconsortium.org/): a web application that uses an annotated reference dataset to **automate the processing, analysis, and interpretation of a new single-cell RNA-seq or ATAC-seq experiment**.

### ÁªÜËÉûÈÄöËÆØ

* [CellphoneDB](https://github.com/ventolab/CellphoneDB): CellPhoneDB can be used to search for a particular ligand/receptor, or interrogate your own HUMAN single-cell transcriptomics data.

### ‰ª£Ë∞¢ÂàÜÊûê

* [changwn/scFEA: single cell Flux Estimation Analysis (scFEA) Try the below web server!](https://github.com/changwn/scFEA)
* [Yosef Lab | compass](https://yoseflab.github.io/software/compass/): Metabolic modeling using single-cell transcriptomes

### Ë∞ÉÊéßÁΩëÁªú

* [netbiolab/scHumanNet](https://github.com/netbiolab/scHumanNet): Analysis cell-type-specific functional gene network, with SCINET and HumanNetv3
* [dpeerlab/SEACells](https://github.com/dpeerlab/SEACells): SEACells algorithm for Inference of transcriptional and epigenomic cellular states from single-cell genomics data

### GPT

* [GitHub - bowang-lab/scGPT](https://github.com/bowang-lab/scGPT?tab=readme-ov-file): Towards Building a Foundation Model for Single-Cell Multi-omics Using Generative AI

## [Cancer/Survival-related, ÁîüÂ≠òÂàÜÊûêÂú®Á∫øÂ∑•ÂÖ∑ 2020 Âπ¥ 4 ÊúàÂêêË°ÄÊï¥ÁêÜÔºÅ](https://mp.weixin.qq.com/s?__biz=Mzg4NjA5Mzg2Mw==&mid=2247488689&idx=2&sn=92818d38ffba4f97fd295dcf51f05857&chksm=cf9fb5f9f8e83cefc35b6a980c125fe8be1b5860f66211594cb27d7419be02f5adcb95588c56&mpshare=1&scene=1&srcid=&sharer_sharetime=1587258755943&sharer_shareid=49bb68e4d4ad9f65af077f4e54025da0&key=7024fc3958d21a49f38d5962a504384cb7aa02cfec7d99b587fbbaad39d6f014de6891e0d0d0a1c51311259cc5116c5fda4311d0c4a8bf171a88b52cae8649f6495594a0becdfce753717e6f209ed145&ascene=1&uin=MjEyMzUzNDk2MQ%3D%3D&devicetype=Windows+XP&version=62060841&lang=zh_CN&exportkey=Af7QoGlOZcxdXwTntJ6mQfA%3D&pass_ticket=egSQZf3ZmjmNSBYxFfxeMhxks6Fcdfesee2EK45qb6wqbW1FRDSL1ND6vhbrpF2H)

* [Kaplan Meier-plotter](http://kmplot.com/analysis/index.php)
* [PROGgene](http://genomics.jefferson.edu/proggene/)
* [SurvExpress](http://bioinformatica.mty.itesm.mx:8080/Biomatec/SurvivaX.jsp)
* [KM-express](http://ec2-52-201-246-161.compute-1.amazonaws.com/kmexpress/index.php)
* [PrognoScan](http://dna00.bio.kyutech.ac.jp/PrognoScan/index.html)
* [lnCAR: lncRNA Explorer](https://lncar.renlab.org/)
* [UALCAN](http://ualcan.path.uab.edu/analysis.html)
* [OncoLnc](http://www.oncolnc.org/)
* [OncomiR](http://www.oncomir.org)
* [MethSurv](https://biit.cs.ut.ee/methsurv/)
* [BloodSpot](http://servers.binf.ku.dk/bloodspot/)
* [GEPIA 2](http://gepia2.cancer-pku.cn/#survival)
* [GenomicScape](http://www.genomicscape.com/microarray/survival.php)
* [ExSurv](https://exsurv.soic.iupui.edu/)
* [LOGpc](http://bioinfo.henu.edu.cn/DatabaseList.jsp)
* [CBioportal](https://www.cbioportal.org/)
* [UCSC Xena](https://xena.ucsc.edu/)
* [GDC portal](https://portal.gdc.cancer.gov/)
* [SurvMicro](http://bioinformatica.mty.itesm.mx:8080/Biomatec/Survmicro.jsp)
* [DriverDBv3](http://driverdb.tms.cmu.edu.tw/)
* [LncACTdb 2.0](http://www.bio-bigdata.net/LncACTdb/)
* [TCGAportal](http://tumorsurvival.org/index.html)
* [Cancer RNA-Seq Nexus](http://syslab4.nchu.edu.tw/)
* [iCoMut](http://firebrowse.org/iCoMut/): ÁôåÁóáÁöÑsomatic Á™ÅÂèò‰ø°ÊÅØÂÖ®ÊôØÂõæ
* [UALCAN](http://ualcan.path.uab.edu/index.html)
* [CAMOIP](https://www.camoip.net/)
* [TIMEDB](https://timedb.deepomics.org/): *Tumor Immune MicroEnvironment cell composition DataBase*
* http://bio-bigdata.hrbmu.edu.cn/ImmCluster
* [THER](https://smuonco.shinyapps.io/THER/)Ôºö‰∏ÄÊ¨æÊé¢Á©∂Ê≥õËÇøÁò§Áº∫Ê∞ßÁöÑ‰∫§‰∫íÂºèÁΩëÈ°µÂ∑•ÂÖ∑

## Áü•ÂêçÁöÑËÇøÁò§Á†îÁ©∂Êú∫ÊûÑÈÉΩÊúâÁùÄËá™Â∑±ÁöÑ TCGA Êï∞ÊçÆÂ∫ìÊé¢Á¥¢Â∑•ÂÖ∑

* Broad Institute FireBrowse portal, The Broad Institute
* cBioPortal for Cancer Genomics, Memorial Sloan-Kettering Cancer Center
* TCGA Batch Effects, MD Anderson Cancer Center
* Regulome Explorer, Institute for Systems Biology
* Next-Generation Clustered Heat Maps, MD Anderson Cancer Center
* [Network of Cancer Genes Home](http://ncg.kcl.ac.uk/index.php)
* [Tumor suppressor gene database (TSGene) Home](https://bioinfo.uth.edu/TSGene/)

## Shiny-related

* [nanxstats/awesome-shiny-extensions](https://github.com/nanxstats/awesome-shiny-extensions): üêù Awesome R packages that offer extended UI or server components for the R web framework Shiny
* [Âü∫‰∫é Seurat ÁöÑÂçïÁªÜËÉûÂØπË±°‰ΩøÁî® ShinyCell Ëá™Âä®ÂåñÁîüÊàêÁΩëÈ°µ](https://mp.weixin.qq.com/s/RQC7twfVynw49KVXulsxKg)
* [Murine SC Atlas of the Lung Responses to Radiation Injury](https://lustra.shinyapps.io/Murine_RIPF_Atlas/)
* [SGDDNB/ShinyCell: Shiny Interactive Web Apps for Single-Cell Data](https://github.com/SGDDNB/ShinyCell)
* [C3BI-pasteur-fr/UTechSCB-SCHNAPPs: Shiny app for the analysis of single cell data](https://github.com/C3BI-pasteur-fr/UTechSCB-SCHNAPPs)
* [romanhaa/cerebroApp: R package containing the Cerebro application.](https://github.com/romanhaa/cerebroApp)
* [nlawlor/V-SVA: V-SVA: An R Shiny application for detecting and annotating hidden sources of variation in single cell RNA-seq data](https://github.com/nlawlor/V-SVA)
* [atakanekiz/CIPR-Shiny: Repository for Shiny implementation of CIPR](https://github.com/atakanekiz/CIPR-Shiny)
* [immcore/iS-CellR: iS-CellR: interactive graphical tool for analysis of single-cell RNAseq data](https://github.com/immcore/iS-CellR)
* [vallotlab/ChromSCape: ChromSCape](https://github.com/vallotlab/ChromSCape)
* [Kasper Lab: Mouse skin scRNA-seq atlas](http://kasperlab.org/mouseskin)

ÂèØ‰ª•ËÆ©ÁΩëÈ°µÁúãËµ∑Êù•Êõ¥‰∏ì‰∏öÁöÑ R ÂåÖÔºö

* shinythemes
* dashboardthemes
* shinydashboard
* shinydashboardPlus
* shinyjs
* shinyWidgets
* shinycssloaders
* shinyalert
* shinydisconnect
* shinybusy
* DT
* waiter
* slickR
* plotly
* heatmaply
* htmlwidgets

# Long read sequencing
* [LRS-DB](https://long-read-tools.org/): A catalogue of long read sequencing data analysis tools

# Scholar search
* [Ëô´ÈÉ®ËêΩÊêúÁ¥¢](https://scholar.chongbuluo.com/): ÈõÜÂêà‰∫ÜÂ∏ÇÈù¢‰∏äÊúÄ‰∏∫Â∏∏ËßÅÁöÑÂ≠¶ÊúØÊêúÁ¥¢ÂºïÊìéÔºåËÆ©‰Ω†‰∏ÄÁ´ôÂºèÊêúÂà∞Â∫ï
* [Open-i](https://openi.nlm.nih.gov): ÁæéÂõΩÂõΩÁ´ãÂç´ÁîüÁ†îÁ©∂Èô¢ÂºÄÂèëÁöÑÁßëÁ†îÂõæÁâáÊï∞ÊçÆÂ∫ìÔºåÁîüÁâ©ÂíåÂåªÂ≠¶Á±ªÈ¶ñÊé®ÁöÑÂõæÁâáÊêúÁ¥¢ÁΩëÁ´ô
* [Viziometrics](http://viziometrics.org/search/): The First Visual Search Engine for Scientific Diagrams
* [CNKI Â≠¶ÊúØÂõæÁâáÂ∫ì](http://image.cnki.net/)
* [rcrossref](https://cran.r-project.org/web/packages/rcrossref/index.html): Client for Various 'CrossRef' 'APIs'
* [crminer](https://cran.r-project.org/web/packages/crminer/index.html): Fetch 'Scholary' Full Text from 'Crossref'

# Chrome
* [Ë∞∑Á≤í - Chrome Êèí‰ª∂Ëã±ÈõÑÊ¶ú](https://github.com/zhaoolee/ChromeAppHeroes): ‰∏∫‰ºòÁßÄÁöÑ Chrome Êèí‰ª∂ÂÜô‰∏ÄÊú¨‰∏≠ÊñáËØ¥Êòé‰π¶ÔºåËÆ© Chrome Êèí‰ª∂Ëã±ÈõÑ‰ª¨ÈÄ†Á¶è‰∫∫Á±ª

# Shell

* [starship](https://starship.rs/): The minimal, blazing-fast, and infinitely customizable prompt for any shell!
* [ÊäÄÊúØ | Êñ∞Âºè Linux ÂëΩ‰ª§Ë°åÂ∑•ÂÖ∑Â§ßÂÖ®](https://linux.cn/article-14488-1.html)
* [ibraheemdev/modern-unix](https://github.com/ibraheemdev/modern-unix): A collection of modern/faster/saner alternatives to common unix commands.
* [romkatv/powerlevel10k](https://github.com/romkatv/powerlevel10k): A Zsh theme
* [Zsh](https://www.zsh.org/)
* [ohmyzsh/ohmyzsh](https://github.com/ohmyzsh/ohmyzsh): üôÉ A delightful community-driven (with 2,000+ contributors) framework for managing your zsh configuration. Includes 300+ optional plugins (rails, git, macOS, hub, docker, homebrew, node, php, python, etc), 140+ themes to spice up your morning, and an auto-update tool so that makes it easy to keep up with the latest updates from the community.



# Others

* [E Venn](http://www.ehbio.com/test/venn/#/)
* [Visinets](https://mp.weixin.qq.com/s?__biz=MzIwMzA1MzU5Mg==&mid=2652152715&idx=2&sn=24ee771dd63400cf5d84a94f01313fa6&chksm=8d354cfcba42c5ea90e5ec36c7fb75fc1d41e9b471fb93c3f05432acd9fc553aec6ac5268336&mpshare=1&scene=1&srcid=0213XFRKAbcWYa9igdvlKnCM&sharer_sharetime=1613191433944&sharer_shareid=49bb68e4d4ad9f65af077f4e54025da0&key=22735c9487e13afecff1211788e4d10128063c1442d91be53d61ed44a7fbd1a74abf3c3bdfe67590d74965a684fb45241c20309b52a70d34fcb5be5266b0cfa907062a9dd0528aa1922b293d2771056f40d061593fe631f38e1bb453218a0fa3dcc473a3a10746f1eb7d239d072e7ad34c101c3d6a28a16852b1691e003a9f6f&ascene=1&uin=MjEyMzUzNDk2MQ%3D%3D&devicetype=Windows+7&version=62090529&lang=en&exportkey=AcSdXDNN0VcaW0EE2WlGimU%3D&pass_ticket=bdtbHUindyL5pXtSUK8%2Ft0VMzBFMRgoZkxBxHnWZRa6qQ9jOt%2FDXFdHtbLQtDK1b&wx_header=0)Ôºö‰∏Ä‰∏™ÂèØ‰ª•ËÆ©‰Ω†ÁöÑ‰ø°Âè∑ÈÄöË∑ØÂõæÂä®Ëµ∑Êù•ÁöÑÁΩëÁ´ô
* [AutoGluon: AutoML for Text, Image, and Tabular Data](https://github.com/awslabs/autogluon)
* [HemI 2.0 - Heatmap Illustrator](https://hemi.biocuckoo.org/)
* [g:Profiler ‚Äì a web server for functional enrichment analysis and conversions of gene lists](https://biit.cs.ut.ee/gprofiler/gost)
* [0 ‰ª£Á†Å„ÄÅÂÖçË¥πÁöÑÂèØËßÜÂåñÂ∑•ÂÖ∑Êé®ËçêÔΩûÔΩûÔΩû](https://mp.weixin.qq.com/s/7wBc_kM8G75ehRAv_XTtow)
* [KOBAS](http://kobas.cbi.pku.edu.cn/): intelligent prioritization and exploratory visualization of biological functions for gene enrichment analysis

---

- MACS2 ËøõË°å Peak calleing
- csaw ËøõË°åÂ∑ÆÂºÇ Peak ÂàÜÊûê
- MEME suite ËøõË°å motif Ê£ÄÊµãÂíåÂØåÈõÜ
- ChIPseeker ËøõË°åÊ≥®ÈáäÂíåÂèØËßÜÂåñ
- HMMRATAC ËøõË°åÊ†∏Â∞è‰ΩìÊ£ÄÊµã
- HINT-ATAC ËøõË°åË∂≥ËøπÂàÜÊûê

# Âú®Á∫øÁªòÂõæ

* [ChartCube - Âú®Á∫øÂõæË°®Âà∂‰ΩúÂ∑•ÂÖ∑](https://chartcube.alipay.com/upload)
* [ÂõæËØ¥](https://tushuo.baidu.com/)
* [RAWGraphs](https://www.rawgraphs.io/)
* [IGV](https://igv.org/app/)
* [https://bioscitools.github.io](https://bioscitools.github.io/)
* [openbiox - Home Page](https://openbiox.org/#/)
* [Hiplot | Emerging biomedical data visualization toolkit](https://hiplot-academic.com/)
* [Bio-Nav](https://bio-nav.hiplot.com.cn/home)
* [HIPLOT PRO-‰∏äÊµ∑ËÖæ‰∫ë‰∫ëÁªòÂõæÁ≥ªÁªü](https://hiplot.com.cn/home/index.html)
* [Evenn](http://www.ehbio.com/test/venn/#/)

# Reproducibility

* [ÊûÑÂª∫ÂèØÈáçÂ§çÁöÑÂçïÁªÜËÉûÊï∞ÊçÆÂàÜÊûêÊµÅÁ®ã](https://www.jianshu.com/p/675335f3fe4a)
* [Reproducibility in Science](http://ropensci.github.io/reproducibility-guide/): A Guide to enhancing reproducibility in scientific results and writing
* [Top ten reasons to not share your code (and why you should anyway)](https://www.software.ac.uk/blog/2016-10-07-top-ten-reasons-not-share-your-code-and-why-you-should-anyway)
* [Does your code stand up to scrutiny?](https://www.nature.com/articles/d41586-018-02741-4)
* [Excuse me, do you have a moment to talk about version control?](https://peerj.com/preprints/3159v2/)
* [[As a reviewer, can I justify requesting data and code be made available even if the journal does not?](https://stats.stackexchange.com/questions/14416/as-a-reviewer-can-i-justify-requesting-data-and-code-be-made-available-even-if)]
* [Table Convert Online - table to markdown,csv,json,latex table,excel,sql,xml](https://tableconvert.com/)

# Single-cell databases

| Êï∞ÊçÆÂ∫ìÂêçÁß∞                         | ÁΩëÂùÄ                                           |
| ---------------------------------- | ---------------------------------------------- |
| CellMarker                         | http://biocc.hrbmu.edu.cn/CellMarker/index.jsp |
| Mouse Cell Atlas(MCAÔºâ             | http://bis.zju.edu.cn/MCA/index.html           |
| PanglaoDB                          | https://panglaodb.se/index.html                |
| Single Cell Expression Atlas       | https://www.ebi.ac.uk/gxa/sc/home              |
| Single Cell PORTAL                 | https://portals.broadinstitute.org/single_cell |
| SCPortalen                         | http://single-cell.clst.riken.jp/              |
| CancerSEA                          | http://biocc.hrbmu.edu.cn/CancerSEA/home.jsp   |
| SpatialDB                          | https://www.spatialomics.org/SpatialDB         |
| Human Cell LandscapeÔºàHCLÔºâ        | http://bis.zju.edu.cn/HCL/                     |
| IMMUcan SingleCell RNAseq Database | https://immucanscdb.vital-it.ch/               |
| Human Cell Atlas                   | https://www.humancellatlas.org/                |
| CZ CELLxGENE                       | https://cellxgene.cziscience.com/              |
| DISCO                              | https://www.immunesinglecell.org/              |

# ST databases

* [Spatial Transcript Omics DataBase - STOMICS DataBase - Á©∫Èó¥ËΩ¨ÂΩïÁªÑÂ≠¶Êï∞ÊçÆÂ∫ì](https://db.cngb.org/stomics/)
* [SpatialDB](http://spatialomics.org/SpatialDB/index.php)
* [Home|SPASCER: spatial transcriptomics annotation at single cell resolution](https://ccsm.uth.edu/SPASCER/)
* [SpatialOmics](https://gene.ai.tencent.com/SpatialOmics/)
* [pysodb ‚Äî pysodb 1.0 documentation](https://pysodb.readthedocs.io/en/latest/)

# Pipeline

* [Snakemake](https://snakemake.readthedocs.io/en/stable/index.html): The Snakemake workflow management system is a tool to create **reproducible and scalable** data analyses.
* [Galaxy | Europe](https://singlecell.usegalaxy.eu/): Single Cell Omics Galaxy Instance

# Genome visualization

* [346 ‰∏™Âü∫Âõ†ÁªÑÂèØËßÜÂåñÂ∑•ÂÖ∑‰∏ÄÁΩëÊâìËøõÔºÅ](https://mp.weixin.qq.com/s/kL21Oo4v6si6nde1EOOWYg)
* [cmdcolin/awesome-genome-visualization: A list of interesting genome browser or genome-browser-like implementations](https://github.com/cmdcolin/awesome-genome-visualization)
* [awesome-genome-visualization](https://cmdcolin.github.io/awesome-genome-visualization/?latest=true)



# Bioinformatics

* [Bioinformatics](https://replikation.github.io/bioinformatics_side/): Bioinformatics Collection, a collection of lectures, programs, tools, tutorials etc.
* [wwood/kingfisher-download](https://github.com/wwood/kingfisher-download): Easier download/extract of FASTA/Q read data and metadata from the ENA, NCBI, AWS or GCP.

# Databases

* [The Human Protein Atlas](https://www.proteinatlas.org/)
* [TIMER2.0](http://timer.cistrome.org/): a comprehensive resource for systematical analysis of immune infiltrates across diverse cancer types
* [GEPIA 2](http://gepia2.cancer-pku.cn/#index)
* [BioGRID | Database of Protein, Chemical, and Genetic Interactions](https://thebiogrid.org/)
* [GeneCards - Human Genes | Gene Database | Gene Search](https://www.genecards.org/)
* [MalaCards - human disease database](https://www.malacards.org/)
* [GeneCaRNA - The Human ncRNA Database](https://www.genecards.org/genecarna)
* [PathCards: Pathways integration](https://pathcards.genecards.org/)
* [TissGDB](https://bioinfo.uth.edu/TissGDB/): Tissue-specific Gene DataBase in cancer
* [MIMIC](https://mimic.mit.edu/): Medical Information Mart for Intensive Care
* [STOMICS DataBase](https://db.cngb.org/stomics/): Spatial Transcript Omics DataBase (STOmics DB) is a comprehensive repository of literature and data sets related to spatial transcriptomics topics, and provides convenient tools for data viewing, analysis, and visualization.
* [TIGER: Tumor Immunotherapy Gene Expression Resource](http://tiger.canceromics.org/#/)
* [DISCO](https://www.immunesinglecell.org/): Deeply Integrated human Single-Cell Omics data
* [Mapping the Human Body at the Cellular Level](https://data.humancellatlas.org/)
* [MCA | Mouse Cell Atlas](http://bis.zju.edu.cn/MCA/)
* [Cell-omics Data Coordinate Platform - CDCP - ÂçïÁªÜËÉûÁªÑÂ≠¶Âπ≥Âè∞](https://db.cngb.org/cdcp/)
* [TEDD](https://tedd.obg.cuhk.edu.hk/frontend/dist/index.html#/): The Temporal Expression during Development Database
* [Home | TFSyntax](https://tfsyntax.zhaopage.com/): a database of transcription factors binding syntax in mammalian genomes.
* [Cell-omics Data Coordinate Platform - CDCP - ÂçïÁªÜËÉûÁªÑÂ≠¶Âπ≥Âè∞](https://db.cngb.org/cdcp/)
* [Scalable Brain Atlas - NeuroMaps Atlas](https://scalablebrainatlas.incf.org/macaque/DB09)
* [HCL - Human Cell Landscape](https://db.cngb.org/HCL/index.html)
* [NCI Human Tumor Atlas Network](https://data.humantumoratlas.org/)

# Algorithm

* [The Algorithms](https://the-algorithms.com/)