# Killer Tools

## R

* [**Awesome R**](https://github.com/qinwf/awesome-R): **A curated list of awesome R packages and tools.**
* [**15 Essential packages in R for Data Science**](https://finnstats.com/index.php/2021/04/07/essential-packages-in-r/)
* [**awesome-ggplot2**](https://github.com/erikgahner/awesome-ggplot2): **A curated list of awesome ggplot2 tutorials, packages etc.**
* [**CRAN Task Views**](https://cran.r-project.org/web/views/)
* [**addinslist**](https://github.com/daattali/addinslist): **Discover and install useful RStudio addins**
* [**RStudio Package Manager**](https://packagemanager.rstudio.com/client/#/)
* [A3](https://cran.r-project.org/web/packages/A3/index.html): Accurate, Adaptable, and Accessible Error Metrics for Predictive Models
* [ActivePathways](https://cran.r-project.org/web/packages/ActivePathways/index.html): Integrative Pathway Enrichment Analysis of Multivariate Omics Data
* [adegenet](https://cran.r-project.org/web/packages/adegenet/index.html): Exploratory Analysis of Genetic and Genomic Data
* [animation](https://cran.r-project.org/web/packages/animation/index.html): A Gallery of Animations in Statistics and Utilities to Create Animations
* [AnnoProbe](https://github.com/jmzeng1314/AnnoProbe): Ë°®ËææËäØÁâáÊï∞ÊçÆÂàÜÊûê‰º¥‰æ£
* [annotater](https://github.com/luisDVA/annotater): The goal of annotater is to annotate package load calls in character strings and R/Rmd files, so we can have an idea of the overall purpose of the libraries we‚Äôre loading.
* [AnnotationHub](https://bioconductor.org/packages/release/bioc/html/AnnotationHub.html): Client to access AnnotationHub resources
* [apcluster](https://cran.r-project.org/web/packages/apcluster/index.html): Affinity Propagation Clustering
* [aplot](https://cran.r-project.org/web/packages/aplot/index.html): Decorate a 'ggplot' with Associated Information
* [arules](https://cran.r-project.org/web/packages/arules/index.html): Mining Association Rules and Frequent Itemsets
* [arulesViz](https://cran.r-project.org/web/packages/arulesViz/index.html): Visualizing Association Rules and Frequent Itemsets
* [autoplotly](https://cran.r-project.org/web/packages/autoplotly/index.html): Automatic Generation of Interactive Visualizations for Statistical Results
* [bbplot](https://github.com/bbc/bbplot): R package that helps create and export ggplot2 charts in the style used by the BBC News data team
* [beepr](https://cran.r-project.org/web/packages/beepr/index.html): Easily Play Notification Sounds on any Platform
* [bestglm](https://cran.r-project.org/web/packages/bestglm/index.html): Best Subset GLM and Regression Utilities
* [bibliometrix](https://cran.r-project.org/web/packages/bibliometrix/index.html): Comprehensive Science Mapping Analysis
* [bigstatsr](https://cran.r-project.org/web/packages/bigstatsr/index.html): Statistical Tools for Filebacked Big Matrices
* [BiocFileCache](https://bioconductor.org/packages/release/bioc/html/BiocFileCache.html): Manage Files Across Sessions
* [BioInstaller](https://cran.r-project.org/web/packages/BioInstaller/index.html): Integrator of Bioinformatics Resources
* [BiocParallel](https://bioconductor.org/packages/release/bioc/html/BiocParallel.html): Bioconductor facilities for parallel evaluation
* [Biostrings](https://bioconductor.org/packages/release/bioc/html/Biostrings.html): Efficient manipulation of biological strings
* [biscale](https://cran.r-project.org/web/packages/biscale/index.html): Tools and Palettes for Bivariate Thematic Mapping
* [blastula](https://cran.r-project.org/web/packages/blastula/index.html): Easily Send HTML Email Messages
* [blogdown](https://cran.r-project.org/web/packages/blogdown/index.html): Create Blogs and Websites with R Markdown
* [bluster](https://www.bioconductor.org/packages/release/bioc/html/bluster.html): Clustering Algorithms for Bioconductor
* [bnlearn](https://cran.r-project.org/web/packages/bnlearn/index.html): Bayesian Network Structure Learning, Parameter Learning and Inference
* [bookdown](https://cran.rstudio.com/web/packages/bookdown/index.html): Authoring Books and Technical Documents with R Markdown
* [Boruta](https://cran.r-project.org/web/packages/Boruta/index.html): Wrapper Algorithm for All Relevant Feature Selection
* [brickr](https://cran.r-project.org/web/packages/brickr/index.html): Emulate LEGO Bricks in 2D and 3D
* [broom](https://cran.r-project.org/web/packages/broom/index.html): Convert Statistical Objects into Tidy Tibbles
* [broom.mixed](https://cran.r-project.org/web/packages/broom.mixed/index.html): Tidying Methods for Mixed Models
* [brranching](https://cran.r-project.org/web/packages/brranching/index.html): Fetch 'Phylogenies' from Many Sources
* [bruceR](https://cran.r-project.org/web/packages/bruceR/index.html): Broadly Useful Convenient and Efficient R Functions
* [bslib](https://cran.r-project.org/web/packages/bslib/index.html): Custom 'Bootstrap' 'Sass' Themes for 'shiny' and 'rmarkdown'
* [bulletchartr](https://github.com/ACDIVOCATech/bulletchartr): This package builds bullet charts suitable for tracking KPIs, and used by ACDI/VOCA to track progress against indicators
* [Cairo](https://cran.r-project.org/web/packages/Cairo/index.html): R Graphics Device using Cairo Graphics Library for Creating High-Quality Bitmap (PNG, JPEG, TIFF), Vector (PDF, SVG, PostScript) and Display (X11 and Win32) Output
* [calendR](https://cran.r-project.org/web/packages/calendR/index.html): Ready to Print Monthly and Yearly Calendars Made with 'ggplot2'
* [canvasXpress](https://cran.r-project.org/web/packages/canvasXpress/index.html): Visualization Package for CanvasXpress in R
* [car](https://cran.r-project.org/web/packages/car/index.html): Companion to Applied Regression
* [caret](https://cran.r-project.org/web/packages/caret/index.html): Classification and Regression Training; Misc functions for training and plotting classification and regression models.
* [cartography](https://cran.r-project.org/web/packages/cartography/index.html): Thematic Cartography
* [caTools](https://cran.r-project.org/web/packages/caTools/index.html): Tools: Moving Window Statistics, GIF, Base64, ROC AUC, etc
* [cellbaseR](https://www.bioconductor.org/packages/release/bioc/html/cellbaseR.html): Querying annotation data from the high performance Cellbase web
* [ChIPseeker](https://www.bioconductor.org/packages/release/bioc/html/ChIPseeker.html): ChIPseeker for ChIP peak Annotation, Comparison, and Visualization
* [ChromHeatMap](http://bioconductor.org/packages/release/bioc/html/ChromHeatMap.html): Heat map plotting by genome coordinate
* [chromoMap](https://cran.r-project.org/web/packages/chromoMap/index.html): Interactive Visualization and Mapping of Chromosomes
* [circlize](https://github.com/jokergoo/circlize): circular visualization in R
* [clipr](https://cran.r-project.org/web/packages/clipr/index.html): Read and Write from the System Clipboard
* [clock](https://cran.r-project.org/web/packages/clock/index.html): Date-Time Types and Tools
* [cluster](https://cran.r-project.org/web/packages/cluster/index.html): "Finding Groups in Data": Cluster Analysis Extended Rousseeuw et al.
* [clusterProfiler](https://bioconductor.org/packages/release/bioc/html/clusterProfiler.html): statistical analysis and visualization of functional profiles for genes and gene clusters
* [cmprsk](https://cran.r-project.org/web/packages/cmprsk/index.html): Subdistribution Analysis of Competing Risks
* [cola](https://www.bioconductor.org/packages/release/bioc/html/cola.html): A Framework for Consensus Partitioning
* [colorfindr](https://github.com/zumbov2/colorfindr): Extracts colors from various image types, plots treemaps and 3D scatterplots of color compositions, create color palettes
* [colorfulVennPlot](https://cran.r-project.org/web/packages/colorfulVennPlot/index.html): Plot and add custom coloring to Venn diagrams for 2-dimensional, 3-dimensional and 4-dimensional data
* [colourpicker](https://cran.r-project.org/web/packages/colourpicker/index.html): A Colour Picker Tool for Shiny and for Selecting Colours in Plots
* [combinat](https://cran.r-project.org/web/packages/combinat/index.html): combinatorics utilities
* [ComplexHeatmap](http://bioconductor.org/packages/release/bioc/html/ComplexHeatmap.html): Make Complex Heatmaps
* [ComplexUpset](https://cran.r-project.org/web/packages/ComplexUpset/index.html): Create Complex UpSet Plots Using 'ggplot2' Components
* [concaveman](https://cran.r-project.org/web/packages/concaveman/index.html): A Very Fast 2D Concave Hull Algorithm
* [conflicted](https://cran.r-project.org/web/packages/conflicted/index.html): An Alternative Conflict Resolution Strategy
* [correlation](https://cran.r-project.org/web/packages/correlation/index.html): Methods for Correlation Analysis
* [corrgram](https://cran.r-project.org/web/packages/corrgram/index.html): Plot a Correlogram
* [corrplot](https://cran.r-project.org/web/packages/corrplot/index.html): Visualization of a Correlation Matrix
* [cowplot](https://cran.r-project.org/web/packages/cowplot/index.html): Streamlined Plot Theme and Plot Annotations for 'ggplot2'
* [CRANsearcher](https://cran.r-project.org/web/packages/CRANsearcher/index.html): RStudio Addin for Searching Packages in CRAN Database Based on Keywords
* [crayon](https://cran.r-project.org/web/packages/crayon/index.html): Colored Terminal Output
* [CrossICC](https://www.bioconductor.org/packages/release/bioc/html/CrossICC.html): An Interactive Consensus Clustering Framework for Multi-platform Data Analysis
* [crosslink](https://github.com/zzwch/crosslink): Network Visualization Tailed for Grouped Nodes
* [ctv](https://cran.r-project.org/web/packages/ctv/index.html): CRAN Task Views
* [curatedTCGAData](https://bioconductor.org/packages/release/data/experiment/html/curatedTCGAData.html): Curated Data From The Cancer Genome Atlas (TCGA) as MultiAssayExperiment Objects
* [customLayout](https://cran.r-project.org/web/packages/customLayout/index.html): Arrange Elements on the R's Drawing Area or Inside the PowerPoint's Slide
* [data.table](https://cran.r-project.org/web/packages/data.table/index.html): Extension of 'data.frame'
* [DataExplorer](https://cran.r-project.org/web/packages/DataExplorer/index.html): Automate Data Exploration and Treatment
* [datapasta](https://github.com/MilesMcBain/datapasta): is about reducing resistance associated with copying and pasting data to and from R
* [datarium](https://cran.r-project.org/web/packages/datarium/index.html): Data Bank for Statistical Analysis and Visualization
* [datasauRus](https://cran.r-project.org/web/packages/datasauRus/index.html): Datasets from the Datasaurus Dozen
* [dendextend](https://cran.r-project.org/web/packages/dendextend/index.html): Extending 'dendrogram' Functionality in R
* [dendsort](https://cran.rstudio.com/web/packages/dendsort/index.html): Modular Leaf Ordering Methods for Dendrogram Nodes
* [descriptr](https://cran.r-project.org/web/packages/descriptr/index.html): Generate Descriptive Statistics
* [DescrTab2](https://cran.r-project.org/web/packages/DescrTab2/index.html): Publication Quality Descriptive Statistics Tables
* [DescTools](https://cran.r-project.org/web/packages/DescTools/index.html): Tools for Descriptive Statistics
* [dlookr](https://cran.r-project.org/web/packages/dlookr/index.html): Tools for Data Diagnosis, Exploration, Transformation
* [DNAshapeR](https://www.bioconductor.org/packages/release/bioc/html/DNAshapeR.html): High-throughput prediction of DNA shape features
* [diffdf](https://cran.r-project.org/web/packages/diffdf/index.html): Dataframe Difference Tool
* [distill](https://cran.r-project.org/web/packages/distill/index.html): 'R Markdown' Format for Scientific and Technical Writing
* [doMC](https://cran.r-project.org/web/packages/doMC/index.html): Foreach Parallel Adaptor for 'parallel'
* [doParallel](https://cran.r-project.org/web/packages/doParallel/index.html): Foreach Parallel Adaptor for the 'parallel' Package
* [dorothea](https://bioconductor.org/packages/release/data/experiment/html/dorothea.html): Collection Of Human And Mouse TF Regulons
* [doSNOW](https://cran.r-project.org/web/packages/doSNOW/index.html): Foreach Parallel Adaptor for the 'snow' Package
* [drake](https://cran.r-project.org/web/packages/drake/index.html): A Pipeline Toolkit for Reproducible Computation at Scale
* [drawProteins](https://bioconductor.org/packages/release/bioc/html/drawProteins.html): Package to Draw Protein Schematics from Uniprot API output
* [dslabs](https://cran.r-project.org/web/packages/dslabs/index.html): Data Science Labs
* [DT](https://cran.r-project.org/web/packages/DT/index.html): A Wrapper of the JavaScript Library 'DataTables'
* [dynamicTreeCut](https://cran.r-project.org/web/packages/dynamicTreeCut/index.html): Methods for Detection of Clusters in Hierarchical Clustering Dendrograms
* [e1071](https://cran.r-project.org/web/packages/e1071/index.html): Functions for latent class analysis, short time Fourier transform, fuzzy clustering, support vector machines, shortest path computation, bagged clustering, naive Bayes classifier, ...
* [easyalluvial](https://cran.r-project.org/web/packages/easyalluvial/index.html): Generate Alluvial Plots with a Single Line of Code
* [easylabel](https://cran.r-project.org/web/packages/easylabel/index.html): Interactive Scatter Plot and Volcano Plot Labels
* [easyPubMed](https://cran.r-project.org/web/packages/easyPubMed/index.html): Search and Retrieve Scientific Publication Records from PubMed
* [EasyStat](https://github.com/taowenmicro/EasyStat): Â∑ÆÂºÇÂàÜÊûêÂÆåÊï¥Ëß£ÂÜ≥ÊñπÊ°à
* [EBImage](https://bioconductor.org/packages/release/bioc/html/EBImage.html): Image processing and analysis toolbox for R
* [echarts4r](https://cran.r-project.org/web/packages/echarts4r/index.html): Create Interactive Graphs with 'Echarts JavaScript' Version 5
* [egg](https://cran.r-project.org/web/packages/egg/index.html): Extensions for 'ggplot2': Custom Geom, Custom Themes, Plot Alignment, Labelled Panels, Symmetric Scales, and Fixed Panel Size
* [embed](https://cran.r-project.org/web/packages/embed/index.html): Extra Recipes for Encoding Categorical Predictors
* [emmeans](https://cran.r-project.org/web/packages/emmeans/index.html): Estimated Marginal Means, aka Least-Squares Means
* [emojifont](https://cran.r-project.org/web/packages/emojifont/index.html): Emoji and Font Awesome in Graphics
* [enrichplot](https://bioconductor.org/packages/release/bioc/html/enrichplot.html): Visualization of Functional Enrichment Result
* [EnhancedVolcano](https://bioconductor.org/packages/release/bioc/html/EnhancedVolcano.html): Publication-ready volcano plots with enhanced colouring and labeling
* [EnrichedHeatmap](https://bioconductor.org/packages/release/bioc/html/EnrichedHeatmap.html): Making Enriched Heatmaps
* [eoffice](https://cran.r-project.org/web/packages/eoffice/index.html): Export or Graph and Tables to 'Microsoft' Office and Import Figures and Tables
* [esquisse](https://cran.r-project.org/web/packages/esquisse/index.html): Explore and Visualize Your Data Interactively
* [eulerr](https://cran.r-project.org/web/packages/eulerr/index.html): Area-Proportional Euler and Venn Diagrams with Ellipses
* [exomePeak2](https://bioconductor.org/packages/release/bioc/html/exomePeak2.html): Bias-aware Peak Calling and Quantification for MeRIP-Seq
* [explore](https://cran.r-project.org/web/packages/explore/index.html): Simplifies Exploratory Data Analysis
* [export](https://github.com/tomwenseleers/export): easily export active R graphs and statistical output in publication quality to Microsoft Office (Word, PowerPoint and Excel), HTML and Latex
* [expss](https://cran.r-project.org/web/packages/expss/index.html): Tables, Labels and Some Useful Functions from Spreadsheets and 'SPSS' Statistics
* [extrafont](https://cran.r-project.org/web/packages/extrafont/index.html): Tools for using fonts
* [ExPosition](https://cran.r-project.org/web/packages/ExPosition/index.html): Exploratory Analysis with the Singular Value Decomposition
* [factoextra](https://cran.r-project.org/web/packages/factoextra/index.html): Extract and Visualize the Results of Multivariate Data Analyses
* [FactoInvestigate](https://cran.r-project.org/web/packages/FactoInvestigate/index.html): Automatic Description of Factorial Analysis
* [FactoMineR](https://cran.r-project.org/web/packages/FactoMineR/index.html): Multivariate Exploratory Data Analysis and Data Mining
* [fgsea](https://bioconductor.org/packages/release/bioc/html/fgsea.html): Fast Gene Set Enrichment Analysis
* [fields](https://mran.microsoft.com/snapshot/2017-02-04/web/packages/fields/index.html): Tools for Spatial Data
* [FindIT2](https://bioconductor.org/packages/devel/bioc/html/FindIT2.html): find influential TF and Target based on multi-omics data
* [filehash](https://cran.r-project.org/web/packages/filehash/index.html): Simple Key-Value Database
* [finalfit](https://cran.r-project.org/web/packages/finalfit/index.html): Quickly Create Elegant Regression Results Tables and Plots when Modelling
* [flextable](https://cran.r-project.org/web/packages/flextable/index.html): Functions for Tabular Reporting
* [forcats](https://cran.r-project.org/web/packages/forcats/index.html): Tools for Working with Categorical Variables (Factors)
* [foreach](https://cran.r-project.org/web/packages/foreach/index.html): Provides Foreach Looping Construct
* [forecast](https://cran.r-project.org/web/packages/forecast/index.html): Forecasting Functions for Time Series and Linear Models
* [forestplot](https://cran.r-project.org/web/packages/forestplot/index.html): Advanced Forest Plot Using 'grid' Graphics
* [formattable](https://cran.r-project.org/web/packages/formattable/index.html): Create 'Formattable' Data Structures
* [fpc](https://cran.r-project.org/web/packages/fpc/index.html): Flexible Procedures for Clustering
* [fpp3](https://cran.r-project.org/web/packages/fpp3/index.html): Data for "Forecasting: Principles and Practice" (3rd Edition)
* [framecleaner](https://cran.r-project.org/web/packages/framecleaner/index.html): Clean Data Frames
* [FunnelPlotR](https://cran.r-project.org/web/packages/FunnelPlotR/index.html): Funnel Plots for Comparing Institutional Performance
* [future](https://cran.r-project.org/web/packages/future/index.html): Unified Parallel and Distributed Processing in R for Everyone
* [future.apply](https://cran.r-project.org/web/packages/future.apply/index.html): Apply Function to Elements in Parallel using Futures
* [ganttrify](https://github.com/giocomai/ganttrify): Create beautiful Gantt charts with ggplot2
* [gapminder](https://cran.r-project.org/web/packages/gapminder/index.html): Data from Gapminder
* [gdata](https://cran.r-project.org/web/packages/gdata/index.html): Various R Programming Tools for Data Manipulation
* [GDCRNATools](https://bioconductor.org/packages/release/bioc/html/GDCRNATools.html): an R/Bioconductor package for integrative analysis of lncRNA, mRNA, and miRNA data in GDC
* [GenomicRanges](https://bioconductor.org/packages/release/bioc/html/GenomicRanges.html): Representation and manipulation of genomic intervals
* [geomtextpath](https://cran.r-project.org/web/packages/geomtextpath/index.html): Curved Text in 'ggplot2'
* [geoR](https://cran.r-project.org/web/packages/geoR/index.html): Analysis of Geostatistical Data
* [geoviz](https://cran.r-project.org/web/packages/geoviz/index.html): Elevation and GPS Data Visualisation
* [GetoptLong](https://cran.r-project.org/web/packages/GetoptLong/index.html): Parsing Command-Line Arguments and Variable Interpolation
* [gg.gap](https://cran.r-project.org/web/packages/gg.gap/index.html): Define Segments in y-Axis for 'ggplot2'
* [ggalluvial](https://cran.r-project.org/web/packages/ggalluvial/index.html): Alluvial Plots in 'ggplot2'
* [GGally](https://cran.r-project.org/web/packages/GGally/index.html): Extension to 'ggplot2'
* [ggalt](https://cran.r-project.org/web/packages/ggalt/index.html): Extra Coordinate Systems, 'Geoms', Statistical Transformations, Scales and Fonts for 'ggplot2'
* [gganatogram](https://github.com/jespermaag/gganatogram): Create anatogram images for different organisms using ggplot2
* [gganimate](https://cran.r-project.org/web/packages/gganimate/index.html): A Grammar of Animated Graphics
* [ggannotate](https://github.com/MattCowgill/ggannotate): Interactively annotate ggplots
* [ggbeeswarm](https://github.com/eclarke/ggbeeswarm): Beeswarm-style plots with ggplot2
* [ggbio](https://bioconductor.org/packages/release/bioc/html/ggbio.html): Visualization tools for genomic data
* [ggbreak](https://cran.r-project.org/web/packages/ggbreak/index.html): Set Axis Break for 'ggplot2'
* [ggbump](https://cran.r-project.org/web/packages/ggbump/index.html): Bump Chart and Sigmoid Curves
* [ggchicklet](https://github.com/hrbrmstr/ggchicklet): Create Chicklet (Rounded Segmented Column) Charts
* [ggchromatic](https://github.com/teunbrand/ggchromatic): Colour and fill scales for 'ggplot2' using colour spaces.
* [ggcor](https://github.com/houyunhuang/ggcor): Extended tools for correlation analysis and visualization
* [ggcorrplot](https://cran.r-project.org/web/packages/ggcorrplot/index.html): Visualization of a Correlation Matrix using 'ggplot2'
* [ggCyberPunk](https://github.com/delabj/ggCyberPunk): A "Geom" for creating charts with a cyberpunk aesthetic
* [ggcyto](https://www.bioconductor.org/packages/release/bioc/html/ggcyto.html): Visualize Cytometry data with ggplot
* [ggDCA](https://cran.r-project.org/web/packages/ggDCA/index.html): Calculate and Plot Decision Curve
* [ggdendro](https://cran.r-project.org/web/packages/ggdendro/index.html): Create Dendrograms and Tree Diagrams Using 'ggplot2'
* [ggdist](https://cran.r-project.org/web/packages/ggdist/index.html): Visualizations of Distributions and Uncertainty
* [ggeasy](https://cran.r-project.org/web/packages/ggeasy/index.html): Easy Access to 'ggplot2' Commands
* [ggeconodist](https://github.com/hrbrmstr/ggeconodist): Create Diminutive Distribution Charts
* [ggExtra](https://cran.r-project.org/web/packages/ggExtra/index.html): Add Marginal Histograms to 'ggplot2', and More 'ggplot2' Enhancements
* [ggfittext](https://cran.r-project.org/web/packages/ggfittext/index.html): Fit Text Inside a Box in 'ggplot2'
* [ggforce](https://cran.r-project.org/web/packages/ggforce/index.html): Accelerating 'ggplot2'
* [ggforestplot](https://github.com/NightingaleHealth/ggforestplot): An R package to create forestplots of measures of effects with their confidence intervals.
* [ggfortify](https://cran.r-project.org/web/packages/ggfortify/index.html): Data Visualization Tools for Statistical Analysis Results
* [gggenes](https://cran.r-project.org/web/packages/gggenes/index.html): Draw Gene Arrow Maps in 'ggplot2'
* [gggibbous](https://cran.r-project.org/web/packages/gggibbous/index.html): Moon Charts, a Pie Chart Alternative
* [ggh4x](https://cran.rstudio.com/web/packages/ggh4x/index.html): Hacks for 'ggplot2'
* [gghalves](https://cran.r-project.org/web/packages/gghalves/index.html): Compose Half-Half Plots Using Your Favourite Geoms
* [ggheatmap](https://cran.r-project.org/web/packages/ggheatmap/index.html): Plot Heatmap
* [ggHoriPlot](https://cran.r-project.org/web/packages/ggHoriPlot/index.html): Horizon Plots for 'ggplot2'
* [ggiraph](https://cran.r-project.org/web/packages/ggiraph/index.html): Make 'ggplot2' Graphics Interactive
* [ggimage](https://cran.r-project.org/web/packages/ggimage/index.html): Use Image in 'ggplot2'
* [ggimg](https://cran.r-project.org/web/packages/ggimg/index.html): Graphics Layers for Plotting Image Data with 'ggplot2'
* [gglmannotate](https://github.com/wilkox/gglmannotate): üìàAnnotate a ggplot with a description of a linear model
* [ggmap](https://cran.r-project.org/web/packages/ggmap/index.html): Spatial Visualization with ggplot2
* [ggmcmc](https://cran.r-project.org/web/packages/ggmcmc/index.html): Tools for Analyzing MCMC Simulations from Bayesian Inference
* [ggmosaic](https://cran.r-project.org/web/packages/ggmosaic/index.html): Mosaic Plots in the 'ggplot2' Framework
* [ggmsa](https://cran.r-project.org/web/packages/ggmsa/index.html): Plot Multiple Sequence Alignment using 'ggplot2'
* [ggnetwork](https://cran.r-project.org/web/packages/ggnetwork/index.html): Geometries to Plot Networks with 'ggplot2'
* [ggnewscale](https://cran.r-project.org/web/packages/ggnewscale/index.html): Multiple Fill and Colour Scales in 'ggplot2'
* [ggord](https://github.com/fawda123/ggord): a take on ordination plots using ggplot2
* [ggparliament](https://github.com/RobWHickman/ggparliament): Simple parliament plots using ggplot2
* [ggpattern](https://github.com/coolbutuseless/ggpattern): ggplot geoms with pattern fills
* [ggplot.multistats](https://cran.r-project.org/web/packages/ggplot.multistats/index.html): Multiple Summary Statistics for Binned Stats/Geometries
* [ggplotgui](https://cran.r-project.org/web/packages/ggplotgui/index.html): Create Ggplots via a Graphical User Interface
* [ggplotify](https://cran.r-project.org/web/packages/ggplotify/index.html): Convert Plot to 'grob' or 'ggplot' Object
* [ggpmisc](https://cran.r-project.org/web/packages/ggpmisc/index.html): Miscellaneous Extensions to 'ggplot2'
* [ggpol](https://cran.r-project.org/web/packages/ggpol/index.html): Visualizing Social Science Data with 'ggplot2'
* [ggprism](https://github.com/csdaw/ggprism): The `ggprism` package provides various themes, palettes, and other useful functions to customise ggplots and give them the *‚ÄòGraphPad Prism‚Äô* look.
* [ggpubr](https://cran.r-project.org/web/packages/ggpubr/index.html): 'ggplot2' Based Publication Ready Plots
* [ggradar](https://github.com/ricardo-bion/ggradar): radar charts with ggplot2
* [ggraph](https://cran.r-project.org/web/packages/ggraph/index.html): An Implementation of Grammar of Graphics for Graphs and Networks
* [ggrepel](https://cran.r-project.org/web/packages/ggrepel/index.html): Automatically Position Non-Overlapping Text Labels with 'ggplot2'
* [ggridges](https://cran.r-project.org/web/packages/ggridges/index.html): Ridgeline Plots in 'ggplot2'
* [ggsankey](https://github.com/davidsjoberg/ggsankey): The goal of ggsankey is to make beautiful sankey, alluvial and sankey bump plots in ggplot2
* [ggsci](https://cran.r-project.org/web/packages/ggsci/index.html): Scientific Journal and Sci-Fi Themed Color Palettes for 'ggplot2'
* [ggseqlogo](https://cran.r-project.org/web/packages/ggseqlogo/index.html): A 'ggplot2' Extension for Drawing Publication-Ready Sequence Logos
* [ggside](https://cran.r-project.org/web/packages/ggside/index.html): Side Grammar Graphics
* [ggsignif](https://cran.r-project.org/web/packages/ggsignif/index.html): Significance Brackets for 'ggplot2'
* [ggsn](https://cran.r-project.org/web/packages/ggsn/index.html): North Symbols and Scale Bars for Maps Created with 'ggplot2' or 'ggmap'
* [ggspatial](https://cran.r-project.org/web/packages/ggspatial/index.html): Spatial Data Framework for ggplot2
* [ggstance](https://cran.r-project.org/web/packages/ggstance/index.html): Horizontal 'ggplot2' Components
* [ggstar](https://cran.rstudio.com/web/packages/ggstar/index.html): Multiple Geometric Shape Point Layer for 'ggplot2'
* [ggstatsplot](https://cran.r-project.org/web/packages/ggstatsplot/index.html): 'ggplot2' Based Plots with Statistical Details
* [ggstream](https://github.com/davidsjoberg/ggstream): create a simple but powerful implementation of streamplot/streamgraph in `ggplot2`
* [ggsymbol](https://github.com/xiangpin/ggsymbol): symbols layer based on ggplot2 and ggstar
* [ggtext](https://cran.r-project.org/web/packages/ggtext/index.html): Improved Text Rendering Support for 'ggplot2'
* [ggThemeAssist](https://cran.r-project.org/web/packages/ggThemeAssist/index.html): Add-in to Customize 'ggplot2' Themes
* [ggthemes](https://cran.r-project.org/web/packages/ggthemes/index.html): Extra Themes, Scales and Geoms for 'ggplot2'
* [ggthemr](https://github.com/Mikata-Project/ggthemr): Themes for ggplot2.
* [ggTimeSeries](https://cran.r-project.org/web/packages/ggTimeSeries/index.html): Time Series Visualisations Using the Grammar of Graphics
* [ggtree](https://www.bioconductor.org/packages/release/bioc/html/ggtree.html): an R package for visualization of tree and annotation data
* [ggtreeExtra](https://bioconductor.org/packages/release/bioc/html/ggtreeExtra.html): An R Package To Add Geometric Layers On Circular Or Other Layout Tree Of "ggtree"
* [ggupset](https://github.com/const-ae/ggupset): Combination matrix axis for 'ggplot2' to create 'UpSet' plots
* [ggvenn](https://cran.r-project.org/web/packages/ggvenn/index.html): Draw Venn Diagram by 'ggplot2'
* [ggVennDiagram](https://cran.r-project.org/web/packages/ggVennDiagram/index.html): A 'ggplot2' Implement of Venn Diagram
* [ggvis](https://cran.r-project.org/web/packages/ggvis/index.html): Interactive Grammar of Graphics
* [ggvwline](https://github.com/houyunhuang/ggvwline): Extension to ggplot2 for drawing variable-width lines
* [ggwaffle](https://github.com/liamgilbey/ggwaffle): Creating waffle charts in a ggplot friendly way
* [ghibli](https://cran.r-project.org/web/packages/ghibli/index.html): Studio Ghibli Colour Palettes
* [glmnet](https://cran.r-project.org/web/packages/glmnet/index.html): Lasso and Elastic-Net Regularized Generalized Linear Models
* [glmSparseNet](https://www.bioconductor.org/packages/release/bioc/html/glmSparseNet.html): Network Centrality Metrics for Elastic-Net Regularized Models
* [glue](https://cran.r-project.org/web/packages/glue/index.html): Interpreted String Literals
* [GOplot](http://wencke.github.io/): Visualization of Functional Analysis Data
* [GOSemSim](https://bioconductor.org/packages/release/bioc/html/GOSemSim.html): GO-terms Semantic Similarity Measures
* [gplots](https://cran.r-project.org/web/packages/gplots/index.html): Various R Programming Tools for Plotting Data
* [grafify](https://github.com/ashenoy-cmbi/grafify): Easy grafs, ANOVAs and posthoc comparisons.
* [grDevices](https://www.rdocumentation.org/packages/grDevices/versions/3.6.2): The R Graphics Devices and Support for Colours and Fonts
* [GSEABase](https://bioconductor.org/packages/release/bioc/html/GSEABase.html): Gene set enrichment data structures and methods
* [GSVA](https://www.bioconductor.org/packages/release/bioc/html/GSVA.html): Gene Set Variation Analysis for microarray and RNA-seq data
* [gt](https://cran.r-project.org/web/packages/gt/index.html): Easily Create Presentation-Ready Display Tables
* [gtools](https://cran.r-project.org/web/packages/gtools/index.html): Various R Programming Tools
* [Guitar](https://www.bioconductor.org/packages/release/bioc/html/Guitar.html): The package is designed for visualization of RNA-related genomic features with respect to the landmarks of RNA transcripts, i.e., transcription starting site, start codon, stop codon and transcription ending site.
* [Gviz](https://bioconductor.org/packages/release/bioc/html/Gviz.html): Plotting data and annotation information along genomic coordinates
* [h2o](https://cran.r-project.org/web/packages/h2o/index.html): R Interface for the 'H2O' Scalable Machine Learning Platform
* [hchinamap](https://cran.r-project.org/web/packages/hchinamap/index.html): Mapping China and Its Provinces
* [here](https://cran.r-project.org/web/packages/here/index.html): A Simpler Way to Find Your Files
* [HGNChelper](https://cran.r-project.org/web/packages/HGNChelper/index.html): Identify and Correct Invalid HGNC Human Gene Symbols and MGI Mouse Gene Symbols
* [highcharter](https://cran.r-project.org/web/packages/highcharter/index.html): A Wrapper for the 'Highcharts' Library
* [HilbertCurve](https://www.bioconductor.org/packages/release/bioc/html/HilbertCurve.html): Making 2D Hilbert Curve
* [Hmisc](https://cran.r-project.org/web/packages/Hmisc/index.html): Harrell Miscellaneous
* [hrbrthemes](https://cran.r-project.org/web/packages/hrbrthemes/index.html): Additional Themes, Theme Components and Utilities for 'ggplot2'
* [httr](https://cran.r-project.org/web/packages/httr/index.html): Tools for Working with URLs and HTTP
* [huxtable](https://cran.r-project.org/web/packages/huxtable/index.html): Easily Create and Style Tables for LaTeX, HTML and Other Formats
* [igraph](https://cran.r-project.org/web/packages/igraph/index.html): Network Analysis and Visualization
* [iheatmapr](https://cran.r-project.org/web/packages/iheatmapr/index.html): Interactive, Complex Heatmaps
* [infer](https://cran.r-project.org/web/packages/infer/index.html): Tidy Statistical Inference
* [intansv](https://bioconductor.org/packages/release/bioc/html/intansv.html): Integrative analysis of structural variations
* [ipred](https://cran.r-project.org/web/packages/ipred/index.html): Improved Predictors
* [iterators](https://cran.r-project.org/web/packages/iterators/index.html): Provides Iterator Construct
* [janitor](https://cran.r-project.org/web/packages/janitor/index.html): Simple Tools for Examining and Cleaning Dirty Data
* [jiebaR](https://cran.r-project.org/web/packages/jiebaR/index.html): Chinese Text Segmentation
* [jiebaRD](https://cran.r-project.org/web/packages/jiebaRD/index.html): Chinese Text Segmentation Data for jiebaR Package
* [kableExtra](https://cran.r-project.org/web/packages/kableExtra/index.html): Construct Complex Table with 'kable' and Pipe Syntax
* [karyoploteR](https://bioconductor.org/packages/release/bioc/html/karyoploteR.html): Plot customizable linear genomes displaying arbitrary data
* [KEGGREST](https://bioconductor.org/packages/release/bioc/html/KEGGREST.html): Client-side REST access to the Kyoto Encyclopedia of Genes and Genomes (KEGG)
* [KMunicate](https://cran.r-project.org/web/packages/KMunicate/index.html): KMunicate-Style Kaplan‚ÄìMeier Plots
* [LaCroixColoR](https://github.com/johannesbjork/LaCroixColoR): LaCroix Color Palettes for R
* [latex2exp](https://cran.r-project.org/web/packages/latex2exp/index.html): Use LaTeX Expressions in Plots
* [leaflet](https://cran.r-project.org/web/packages/leaflet/index.html): Create Interactive Web Maps with the JavaScript 'Leaflet' Library
* [leafletCN](https://cran.r-project.org/web/packages/leafletCN/index.html): An R Gallery for China and Other Geojson Choropleth Map in Leaflet
* [learnr](https://cran.r-project.org/web/packages/learnr/index.html): Interactive Tutorials for R
* [legendMap](https://github.com/3wen/legendMap): Provides functions to add a North arrow and/or a scale bar to maps done using ggplot2
* [lime](https://cran.r-project.org/web/packages/lime/index.html): Local Interpretable Model-Agnostic Explanations
* [lindia](https://cran.r-project.org/web/packages/lindia/index.html): Automated Linear Regression Diagnostic
* [LinkageMapView](https://cran.r-project.org/web/packages/LinkageMapView/index.html): Plot Linkage Group Maps with Quantitative Trait Loci
* [magick](https://cran.r-project.org/web/packages/magick/index.html): Advanced Graphics and Image-Processing in R
* [manhattanly](https://cran.r-project.org/web/packages/manhattanly/index.html): Interactive Q-Q and Manhattan Plots Using 'plotly.js'
* [Map2NCBI](https://cran.r-project.org/web/packages/Map2NCBI/index.html): Mapping Markers to the Nearest Genomic Feature
* [mapdeck](https://cran.r-project.org/web/packages/mapdeck/index.html): Interactive Maps Using 'Mapbox GL JS' and 'Deck.gl'
* [maps](https://cran.r-project.org/web/packages/maps/index.html): Draw Geographical Maps
* [mapsf](https://cran.r-project.org/web/packages/mapsf/index.html): Thematic Cartography
* [maptools](https://cran.r-project.org/web/packages/maptools/index.html): Tools for Handling Spatial Objects
* [mcb](https://cran.r-project.org/web/packages/mcb/index.html): Model Confidence Bounds
* [mclust](https://cran.r-project.org/web/packages/mclust/index.html): Gaussian Mixture Modelling for Model-Based Clustering, Classification, and Density Estimation
* [MetaboSignal](https://bioconductor.org/packages/release/bioc/html/MetaboSignal.html): a network-based approach to overlay and explore metabolic and signaling KEGG pathways
* [MEGENA](https://cran.r-project.org/web/packages/MEGENA/index.html): Multiscale Clustering of Geometrical Network
* [memes](https://www.bioconductor.org/packages/release/bioc/html/memes.html): motif matching, comparison, and de novo discovery using the MEME Suite
* [MeSHDbi](https://www.bioconductor.org/packages/release/bioc/html/MeSHDbi.html): DBI to construct MeSH-related package from sqlite file
* [meshes](https://bioconductor.org/packages/release/bioc/html/meshes.html): MeSH Enrichment and Semantic analyses
* [metafor](https://cran.r-project.org/web/packages/metafor/index.html): Meta-Analysis Package for R
* [MetBrewer](https://github.com/BlakeRMills/MetBrewer/tree/main): Color palette package in R inspired by works at the Metropolitan Museum of Art in New York
* [Mfuzz](https://www.bioconductor.org/packages/release/bioc/html/Mfuzz.html): Soft clustering of time series gene expression data
* [mice](https://cran.r-project.org/web/packages/mice/index.html): Multivariate Imputation by Chained Equations
* [mindr](https://cran.r-project.org/web/packages/mindr/index.html): Convert Files Between Markdown or R Markdown Files and Mind Maps
* [mixOmics](http://www.bioconductor.org/packages/release/bioc/html/mixOmics.html): Omics Data Integration Project
* [MixSim](https://cran.r-project.org/web/packages/MixSim/index.html): Simulating Data to Study Performance of Clustering Algorithms
* [mlr](https://cran.r-project.org/web/packages/mlr/index.html): Machine Learning in R
* [mlr3](https://cran.r-project.org/web/packages/mlr3/index.html): Machine Learning in R - Next Generation
* [mlr3verse](https://cran.rstudio.com/web/packages/mlr3verse/index.html): Easily Install and Load the 'mlr3' Package Family
* [modelROC](https://cran.r-project.org/web/packages/modelROC/index.html): Model Based ROC Analysis
* [modeltime](https://cran.r-project.org/web/packages/modeltime/index.html): The Tidymodels Extension for Time Series Modeling
* [moderndive](https://cran.r-project.org/web/packages/moderndive/index.html): Tidyverse-Friendly Introductory Linear Regression
* [MOFA2](https://www.bioconductor.org/packages/release/bioc/html/MOFA2.html): Multi-Omics Factor Analysis v2
* [motifStack](https://bioconductor.org/packages/release/bioc/html/motifStack.html): Plot stacked logos for single or multiple DNA, RNA and amino acid sequence
* [msa](https://bioconductor.org/packages/release/bioc/html/msa.html): Multiple Sequence Alignment
* [msigdbr](https://cran.r-project.org/web/packages/msigdbr/index.html): MSigDB Gene Sets for Multiple Organisms in a Tidy Data Format
* [mstate](https://cran.r-project.org/web/packages/mstate/index.html): Data Preparation, Estimation and Prediction in Multi-State Models
* [multcomp](https://cran.r-project.org/web/packages/multcomp/index.html): Simultaneous Inference in General Parametric Models
* [multiMiR](https://bioconductor.org/packages/release/bioc/html/multiMiR.html): Integration of multiple microRNA-target databases with their disease and drug associations
* [musicatk](https://bioconductor.org/packages/release/bioc/html/musicatk.html): Mutational Signature Comprehensive Analysis Toolkit
* [NbClust](https://cran.r-project.org/web/packages/NbClust/index.html): Determining the Best Number of Clusters in a Data Set
* [network](https://cran.r-project.org/web/packages/network/index.html): Classes for Relational Data
* [networkD3](https://cran.r-project.org/web/packages/networkD3/index.html): D3 JavaScript Network Graphs from R
* [nlme](https://cran.r-project.org/web/packages/nlme/index.html): Linear and Nonlinear Mixed Effects Models
* [NMF](https://cran.r-project.org/web/packages/NMF/index.html): Algorithms and Framework for Nonnegative Matrix Factorization (NMF)
* [nparcomp](https://cran.r-project.org/web/packages/nparcomp/index.html): Multiple Comparisons and Simultaneous Confidence Intervals
* [officedown](https://cran.r-project.org/web/packages/officedown/index.html): Enhanced 'R Markdown' Format for 'Word' and 'PowerPoint'
* [officer](https://cran.r-project.org/web/packages/officer/index.html): Manipulation of Microsoft Word and PowerPoint Documents
* [OmicCircos](https://bioconductor.org/packages/release/bioc/html/OmicCircos.html): High-quality circular visualization of omics data
* [ontologyIndex](https://cran.r-project.org/web/packages/ontologyIndex/index.html): Functions for Reading Ontologies into R
* [openair](https://cran.r-project.org/web/packages/openair/index.html): Tools for the Analysis of Air Pollution Data
* [openxlsx](https://cran.r-project.org/web/packages/openxlsx/index.html): Read, Write and Edit xlsx Files
* [optparse](https://cran.r-project.org/web/packages/optparse/index.html): Command Line Option Parser
* [overviewR](https://cran.r-project.org/web/packages/overviewR/index.html): Easily Extracting Information About Your Data
* [pacman](https://cran.r-project.org/web/packages/pacman/index.html): Package Management Tool
* [pak](https://cran.r-project.org/web/packages/pak/index.html): Another Approach to Package Installation
* [paletteer](https://cran.r-project.org/web/packages/paletteer/index.html): Comprehensive Collection of Color Palettes
* [parttree](https://github.com/grantmcdermott/parttree): Simple package for plotting decision tree partitions in R
* [patchwork](https://cran.r-project.org/web/packages/patchwork/index.html): The Composer of ggplots
* [pathview](https://www.bioconductor.org/packages/release/bioc/html/pathview.html): a tool set for pathway based data integration and visualization
* [patternplot](https://cran.r-project.org/web/packages/patternplot/index.html): Versatile Pie Charts, Ring Charts, Bar Charts and Box Plots using Patterns, Colors and Images
* [PCAtools](https://bioconductor.org/packages/release/bioc/html/PCAtools.html): Everything Principal Components Analysis
* [pdftables](https://cran.r-project.org/web/packages/pdftables/index.html): Programmatic Conversion of PDF Tables
* [pdftools](https://cran.r-project.org/web/packages/pdftools/index.html): Text Extraction, Rendering and Converting of PDF Documents
* [phytools](https://cran.r-project.org/web/packages/phytools/index.html): Phylogenetic Tools for Comparative Biology (and Other Things)
* [Pi](https://www.bioconductor.org/packages/release/bioc/html/Pi.html): Leveraging Genetic Evidence to Prioritise Drug Targets at the Gene and Pathway Level
* [pivottabler](https://cran.r-project.org/web/packages/pivottabler/index.html): Create Pivot Tables
* [pkgdown](https://cran.r-project.org/web/packages/pkgdown/index.html): Make Static HTML Documentation for a Package
* [plot3D](https://cran.r-project.org/web/packages/plot3D/index.html): Plotting Multi-Dimensional Data
* [plotrix](https://cran.r-project.org/web/packages/plotrix/index.html): Various Plotting Functions
* [PMCMRplus](https://cran.r-project.org/web/packages/PMCMRplus/index.html): Calculate Pairwise Multiple Comparisons of Mean Rank Sums Extended
* [PNWColors](https://cran.r-project.org/web/packages/PNWColors/index.html): Color Palettes Inspired by Nature in the US Pacific Northwest
* [poppr](https://cran.r-project.org/web/packages/poppr/index.html): Genetic Analysis of Populations with Mixed Reproduction
* [precrec](https://cran.r-project.org/web/packages/precrec/index.html): Calculate Accurate Precision-Recall and ROC (Receiver Operator Characteristics) Curves
* [presenter](https://cran.r-project.org/web/packages/presenter/index.html): Present Data with Style
* [prettydoc](https://cran.r-project.org/web/packages/prettydoc/index.html): Creating Pretty Documents from R Markdown
* [prismatic](https://cran.r-project.org/web/packages/prismatic/index.html): Color Manipulation Tools
* [pROC](https://cran.r-project.org/web/packages/pROC/index.html): Display and Analyze ROC Curves
* [psych](https://cran.r-project.org/web/packages/psych/index.html): Procedures for Psychological, Psychometric, and Personality Research
* [qgraph](https://cran.r-project.org/web/packages/qgraph/index.html): Graph Plotting Methods, Psychometric Data Visualization and Graphical Model Estimation
* [qqboxplot](https://cran.rstudio.com/web/packages/qqboxplot/index.html): Implementation of the Q-Q Boxplot
* [qqman](https://cran.r-project.org/web/packages/qqman/index.html): Q-Q and Manhattan Plots for GWAS Data
* [qqplotr](https://cran.r-project.org/web/packages/qqplotr/index.html): Quantile-Quantile Plot Extensions for 'ggplot2'[CRAN - Package xkcd](https://cran.r-project.org/web/packages/xkcd/index.html)
* [quanteda](https://cran.r-project.org/web/packages/quanteda/index.html): Quantitative Analysis of Textual Data
* [quantmod](https://cran.r-project.org/web/packages/quantmod/index.html): Quantitative Financial Modelling Framework
* [quantreg](https://cran.r-project.org/web/packages/quantreg/index.html): Quantile Regression
* [qvalue](http://www.bioconductor.org/packages/release/bioc/html/qvalue.html): Q-value estimation for false discovery rate control
* [r2rtf](https://cran.r-project.org/web/packages/r2rtf/index.html): Easily Create Production-Ready Rich Text Format (RTF) Table and Figure
* [R.utils](https://cran.r-project.org/web/packages/R.utils/index.html): Various Programming Utilities
* [RainCloudPlots](https://github.com/RainCloudPlots/RainCloudPlots): Code and tutorials to visualise your data that is both beautiful *and* statistically valid
* [raincloudplots](https://github.com/jorvlan/raincloudplots): The easy way to create raincloud plots
* [rainette](https://cran.r-project.org/web/packages/rainette/index.html): The Reinert Method for Textual Data Clustering
* [randomcoloR](https://cran.r-project.org/web/packages/randomcoloR/index.html): Generate Attractive Random Colors
* [randomForest](https://cran.r-project.org/web/packages/randomForest/index.html): Breiman and Cutler's Random Forests for Classification and Regression
* [randtoolbox](https://mirrors.sjtug.sjtu.edu.cn/cran/web/packages/randtoolbox/index.html): Toolbox for Pseudo and Quasi Random Number Generation and Random Generator Tests
* [raster](https://cran.r-project.org/web/packages/raster/index.html): Geographic Data Analysis and Modeling
* [rasterVis](https://cran.r-project.org/web/packages/rasterVis/index.html): Visualization Methods for Raster Data
* [rattle](https://cran.r-project.org/web/packages/rattle/index.html): Graphical User Interface for Data Science in R
* [rayshader](https://cran.r-project.org/web/packages/rayshader/index.html): Create Maps and Visualize Data in 2D and 3D
* [rCharts](https://github.com/ramnathv/rCharts): Interactive JS Charts from R
* [RCircos](https://cran.r-project.org/web/packages/RCircos/index.html): Circos 2D Track Plot
* [RcisTarget](https://www.bioconductor.org/packages/release/bioc/html/RcisTarget.html): RcisTarget Identify transcription factor binding motifs enriched on a list of genes or genomic regions
* [RColorBrewer](https://cran.r-project.org/web/packages/RColorBrewer/index.html): ColorBrewer Palettes
* [reactable](https://cran.r-project.org/web/packages/reactable/index.html): Interactive Data Tables Based on 'React Table'
* [recipes](https://cran.r-project.org/web/packages/recipes/index.html): Preprocessing Tools to Create Design Matrices
* [regplot](https://cran.r-project.org/web/packages/regplot/index.html): Enhanced Regression Nomogram Plot
* [remotes](https://cran.r-project.org/web/packages/remotes/index.html): R Package Installation from Remote Repositories, Including 'GitHub'
* [report](https://cran.r-project.org/web/packages/report/index.html): Automated Reporting of Results and Statistical Models
* [reprex](https://cran.r-project.org/web/packages/reprex/index.html): Prepare Reproducible Example Code via the Clipboard
* [Rfast](https://cran.r-project.org/web/packages/Rfast/index.html): A Collection of Efficient and Extremely Fast R Functions
* [rfPermute](https://cran.r-project.org/web/packages/rfPermute/index.html): Estimate Permutation p-Values for Random Forest Importance Metrics
* [rgdal](https://cran.r-project.org/web/packages/rgdal/index.html): Bindings for the 'Geospatial' Data Abstraction Library
* [rgl](https://cran.r-project.org/web/packages/rgl/index.html): 3D Visualization Using OpenGL
* [rhub](https://github.com/r-hub/rhub): a multi-platform build and check service for R packages
* [RImagePalette](https://cran.r-project.org/web/packages/RImagePalette/index.html): Extract the Colors from Images
* [rio](https://cran.r-project.org/web/packages/rio/index.html): A Swiss-Army Knife for Data I/O
* [riskRegression](https://cran.r-project.org/web/packages/riskRegression/index.html): Risk Regression Models and Prediction Scores for Survival Analysis with Competing Risks
* [RIdeogram](https://cran.r-project.org/web/packages/RIdeogram/index.html): Drawing SVG Graphics to Visualize and Map Genome-Wide Data on Idiograms
* [riskRegression](https://cran.r-project.org/web/packages/riskRegression/index.html): Risk Regression Models and Prediction Scores for Survival Analysis with Competing Risks
* [riverplot](https://cran.r-project.org/web/packages/riverplot/index.html): Sankey or Ribbon Plots
* [rmdformats](https://cran.r-project.org/web/packages/rmdformats/index.html): HTML Output Formats and Templates for 'rmarkdown' Documents
* [rms](https://cran.r-project.org/web/packages/rms/index.html): Regression Modeling Strategies
* [RNASeqR](https://bioconductor.org/packages/release/bioc/html/RNASeqR.html): an R package for automated two-group RNA-Seq analysis workflow
* [RNAseqStat](https://gitmemory.cn/index.php/repo/jisnoalia/RNAseqStat): ‰∏ÄË°å‰ª£Á†ÅÂÆåÊàêËΩ¨ÂΩïÁªÑ‰∏ãÊ∏∏ÂàÜÊûê
* [ROCit](https://cran.r-project.org/web/packages/ROCit/index.html): Performance Assessment of Binary Classifier with Visualization
* [ROCR](https://cran.r-project.org/web/packages/ROCR/index.html): Visualizing the Performance of Scoring Classifiers
* [ropls](https://bioconductor.org/packages/release/bioc/html/ropls.html): PCA, PLS(-DA) and OPLS(-DA) for multivariate analysis and feature selection of omics data
* [RobustRankAggreg](https://cran.r-project.org/web/packages/RobustRankAggreg/index.html): Methods for robust rank aggregation
* [rpart](https://cran.r-project.org/web/packages/rpart/index.html): Recursive Partitioning and Regression Trees
* [rpivotTable](https://cran.r-project.org/web/packages/rpivotTable/index.html): Build Powerful Pivot Tables and Dynamically Slice & Dice your Data
* [RSelenium](https://cran.r-project.org/web/packages/RSelenium/index.html): R Bindings for 'Selenium WebDriver'
* [rstan](https://cran.r-project.org/web/packages/rstan/index.html): R Interface to Stan
* [rstatix](https://cran.r-project.org/web/packages/rstatix/index.html): Pipe-Friendly Framework for Basic Statistical Tests
* [RTCGA](https://www.bioconductor.org/packages/release/bioc/html/RTCGA.html): The Cancer Genome Atlas Data Integration
* [rticles](https://cran.r-project.org/web/packages/rticles/index.html): Article Formats for R Markdown
* [rtracklayer](https://bioconductor.org/packages/release/bioc/html/rtracklayer.html): R interface to genome annotation files and the UCSC genome browser
* [Rtsne](https://cran.r-project.org/web/packages/Rtsne/index.html): T-Distributed Stochastic Neighbor Embedding using a Barnes-Hut Implementation
* [RVA](https://cran.r-project.org/web/packages/RVA/index.html): RNAseq Visualization Automation
* [rvcheck](https://cran.rstudio.com/web/packages/rvcheck/index.html): R/Package Version Check, Check latest release version of R and R package (both in 'CRAN', 'Bioconductor' or 'Github')
* [rvest](https://cran.r-project.org/web/packages/rvest/index.html): Easily Harvest (Scrape) Web Pages
* [sankeywheel](https://cran.r-project.org/web/packages/sankeywheel/index.html): Create Dependency Wheels and Sankey Diagrams
* [scales](https://cran.r-project.org/web/packages/scales/index.html): Scale Functions for Visualization
* [scatterpie](https://cran.r-project.org/web/packages/scatterpie/index.html): Scatter Pie Plot
* [scatterplot3d](https://cran.r-project.org/web/packages/scatterplot3d/index.html): 3D Scatter Plot
* [scatterPlotMatrix](https://cran.r-project.org/web/packages/scatterPlotMatrix/index.html): 'Htmlwidget' for a Scatter Plot Matrix & Multiple Scatter Plot
* [scholar](https://cran.r-project.org/web/packages/scholar/index.html): Analyse Citation Data from Google Scholar
* [SCORPIUS](https://cran.rstudio.com/web/packages/SCORPIUS/index.html): Inferring Developmental Chronologies from Single-Cell RNA Sequencing Data
* [see](https://cran.r-project.org/web/packages/see/index.html): Visualisation Toolbox for 'easystats' and Extra Geoms, Themes and Color Palettes for 'ggplot2'
* [seqinr](https://cran.r-project.org/web/packages/seqinr/index.html): Biological Sequences Retrieval and Analysis
* [sf](https://cran.r-project.org/web/packages/sf/index.html): Simple Features for R
* [shadowtext](https://cran.r-project.org/web/packages/shadowtext/index.html): Shadow Text Grob and Layer
* [shiny](https://cran.r-project.org/web/packages/shiny/index.html): Web Application Framework for R
* [shinythemes](https://cran.r-project.org/web/packages/shinythemes/index.html): Themes for Shiny
* [showtext](https://cran.r-project.org/web/packages/showtext/index.html): Using Fonts More Easily in R Graphs
* [sigminer](https://cran.r-project.org/web/packages/sigminer/index.html): Extract, Analyze and Visualize Mutational Signatures for Genomic Variations
* [simplifyEnrichment](https://bioconductor.org/packages/release/bioc/html/simplifyEnrichment.html): Simplify Functional Enrichment Results
* [sjlabelled](https://cran.r-project.org/web/packages/sjlabelled/index.html): Labelled Data Utility Functions
* [sjmisc](https://cran.r-project.org/web/packages/sjmisc/index.html): Data and Variable Transformation Functions
* [sjPlot](https://cran.r-project.org/web/packages/sjPlot/index.html): Data Visualization for Statistics in Social Science
* [skimr](https://cran.r-project.org/web/packages/skimr/index.html): Compact and Flexible Summaries of Data
* [SnowballC](https://cran.r-project.org/web/packages/SnowballC/index.html): Snowball Stemmers Based on the C 'libstemmer' UTF-8 Library
* [sp](https://cran.r-project.org/web/packages/sp/index.html): Classes and Methods for Spatial Data
* [spatstat](https://cran.r-project.org/web/packages/spatstat/index.html): Spatial Point Pattern Analysis, Model-Fitting, Simulation, Tests
* [spiralize](https://github.com/jokergoo/spiralize): Visualize data on spirals
* [splines2](https://cran.r-project.org/web/packages/splines2/index.html): Regression Spline Functions and Classes
* [stargazer](https://cran.r-project.org/web/packages/stargazer/index.html): Well-Formatted Regression and Summary Statistics Tables
* [statmod](https://cran.r-project.org/web/packages/statmod/index.html): Statistical Modeling
* [stm](https://cran.r-project.org/web/packages/stm/index.html): Estimation of the Structural Topic Model
* [STRINGdb](https://www.bioconductor.org/packages/release/bioc/html/STRINGdb.html): Search Tool for the Retrieval of Interacting proteins database
* [sparklyr](https://spark.rstudio.com/): R interface for Apache Spark
* [stringdist](https://cran.r-project.org/web/packages/stringdist/index.html): Approximate String Matching, Fuzzy Text Search, and String Distance Functions
* [survival](https://cran.r-project.org/web/packages/survival/index.html): Survival Analysis
* [survivalAnalysis](https://cran.r-project.org/web/packages/survivalAnalysis/index.html): High-Level Interface for Survival Analysis and Associated Plots
* [survminer](https://cran.r-project.org/web/packages/survminer/index.html): Drawing Survival Curves using 'ggplot2'
* [survRM2](https://cran.r-project.org/web/packages/survRM2/index.html): Comparing Restricted Mean Survival Time
* [Sushi](https://bioconductor.org/packages/release/bioc/html/Sushi.html): Tools for visualizing genomics data
* [systemPipeR](https://bioconductor.org/packages/release/bioc/html/systemPipeR.html): NGS workflow and report generation environment
* [systemPipeTools](https://bioconductor.org/packages/release/bioc/html/systemPipeTools.html): Tools for data visualization
* [tagger](https://github.com/eliocamp/tagger): Adds tags to 'ggpot2' facets
* [TCGAbiolinks](https://bioconductor.org/packages/release/bioc/html/TCGAbiolinks.html): An R/Bioconductor package for integrative analysis with GDC data
* [TCGAutils](https://bioconductor.org/packages/release/bioc/html/TCGAutils.html): A suite of helper functions for checking and manipulating TCGA data including data obtained from the curatedTCGAData experiment package. These functions aim to simplify and make working with TCGA data more manageable.
* [TCseq](https://bioconductor.org/packages/release/bioc/html/TCseq.html): Time course sequencing data analysis
* [terra](https://cran.r-project.org/web/packages/terra/index.html): Spatial Data Analysis
* [tesseract](https://cran.r-project.org/web/packages/tesseract/index.html): Open Source OCR Engine
* [textir](https://cran.r-project.org/web/packages/textir/index.html): Inverse Regression for Text Analysis
* [tidybayes](https://cran.r-project.org/web/packages/tidybayes/index.html): Tidy Data and 'Geoms' for Bayesian Models
* [tidybulk](https://www.bioconductor.org/packages/release/bioc/html/tidybulk.html): Brings transcriptomics to the tidyverse
* [tidyfst](https://cran.r-project.org/web/packages/tidyfst/index.html): Tidy Verbs for Fast Data Manipulation
* [tidygraph](https://cran.r-project.org/web/packages/tidygraph/index.html): A Tidy API for Graph Manipulation
* [tidymodels](https://cran.r-project.org/web/packages/tidymodels/index.html): Easily Install and Load the 'Tidymodels' Packages
* [tidyquant](https://cran.r-project.org/web/packages/tidyquant/index.html): Tidy Quantitative Financial Analysis
* [tidytext](https://cran.r-project.org/web/packages/tidytext/index.html): Text Mining using 'dplyr', 'ggplot2', and Other Tidy Tools
* [tidytidbits](https://cran.r-project.org/web/packages/tidytidbits/index.html): A Collection of Tools and Helpers Extending the Tidyverse
* [tidyverse](https://cran.r-project.org/web/packages/tidyverse/index.html): Easily Install and Load the 'Tidyverse'
* [timeROC](https://cran.r-project.org/web/packages/timeROC/index.html): Time-Dependent ROC Curve and AUC for Censored Survival Data
* [timetk](https://cran.r-project.org/web/packages/timetk/index.html): A Tool Kit for Working with Time Series in R
* [tm](https://cran.r-project.org/web/packages/tm/index.html): Text Mining Package
* [tmap](https://cran.r-project.org/web/packages/tmap/index.html): Thematic Maps
* [TreeAndLeaf](https://bioconductor.org/packages/release/bioc/html/TreeAndLeaf.html): Displaying binary trees with focus on dendrogram leaves
* [treemapify](https://cran.r-project.org/web/packages/treemapify/index.html): Draw Treemaps in 'ggplot2'
* [trend](https://cran.r-project.org/web/packages/trend/index.html): Non-Parametric Trend Tests and Change-Point Detection
* [TSstudio](https://cran.r-project.org/web/packages/TSstudio/index.html): Functions for Time Series Analysis and Forecasting
* [tufte](https://cran.r-project.org/web/packages/tufte/index.html): Tufte's Styles for R Markdown Documents
* [tvthemes](https://cran.r-project.org/web/packages/tvthemes/index.html): TV Show Themes and Color Palettes for 'ggplot2' Graphics
* [TwoSampleMR](https://github.com/MRCIEU/TwoSampleMR): Mendelian randomization with GWAS summary data
* [UCSCXenaShiny](https://cran.r-project.org/web/packages/UCSCXenaShiny/index.html): Interactive Analysis of UCSC Xena Data
* [UCSCXenaTools](https://cran.r-project.org/web/packages/UCSCXenaTools/index.html): Download and Explore Datasets from UCSC Xena Data Hubs
* [UpSetR](https://cran.r-project.org/web/packages/UpSetR/index.html): A More Scalable Alternative to Venn and Euler Diagrams for Visualizing Intersecting Sets
* [VariantAnnotation](https://bioconductor.org/packages/release/bioc/html/VariantAnnotation.html): Annotation of Genetic Variants
* [vcfR](https://cran.r-project.org/web/packages/vcfR/index.html): Manipulate and Visualize VCF Data
* [vegan](https://cran.r-project.org/web/packages/vegan/index.html): Community Ecology Package
* [venn](https://cran.r-project.org/web/packages/venn/index.html): Draw Venn Diagrams
* [VennDiagram](https://cran.r-project.org/web/packages/VennDiagram/index.html): Generate High-Resolution Venn and Euler Plots
* [viewenhance](https://github.com/kieranjmartin/viewenhance): An add in for RStudio which allows the user to filter and select columns interactively
* [ViewPipeSteps](https://github.com/daranzolin/ViewPipeSteps): Create tabs of View() output for each chained pipe
* [vidger](https://www.bioconductor.org/packages/release/bioc/html/vidger.html): Create rapid visualizations of RNAseq data in R
* [VIM](https://cran.r-project.org/web/packages/VIM/index.html): Visualization and Imputation of Missing Values
* [viper](https://bioconductor.org/packages/release/bioc/html/viper.html): Virtual Inference of Protein-activity by Enriched Regulon analysis
* [viridis](https://cran.r-project.org/web/packages/viridis/index.html): Default Color Maps from 'matplotlib'
* [visdat](https://cran.r-project.org/web/packages/visdat/index.html): Preliminary Visualisation of Data
* [visNetwork](https://cran.r-project.org/web/packages/visNetwork/index.html): Network Visualization using 'vis.js' Library
* [visreg](https://pbreheny.github.io/visreg/): An R package for the visualization of regression models
* [vissE](https://bioconductor.org/packages/release/bioc/html/vissE.html): Visualising Set Enrichment Analysis Results
* [vroom](https://cran.r-project.org/web/packages/vroom/index.html): Read and Write Rectangular Text Data Quickly
* [VSURF](https://cran.r-project.org/web/packages/VSURF/index.html): Variable Selection Using Random Forests
* [waldo](https://cran.r-project.org/web/packages/waldo/index.html): Find Differences Between R Objects
* [wesanderson](https://cran.r-project.org/web/packages/wesanderson/index.html): A Wes Anderson Palette Generator
* [wget](https://github.com/YuLab-SMU/wget): Setting Download Method to "wget -c"
* [wordcloud](https://cran.r-project.org/web/packages/wordcloud/index.html): Word Clouds
* [workflowr](https://cran.r-project.org/web/packages/workflowr/index.html): A Framework for Reproducible and Collaborative Data Science
* [xaringan](https://cran.r-project.org/web/packages/xaringan/index.html): Presentation Ninja
* [xgboost](https://cran.r-project.org/web/packages/xgboost/index.html): Extreme Gradient Boosting
* [xlsx](https://cran.r-project.org/web/packages/xlsx/index.html): Read, Write, Format Excel 2007 and Excel 97/2000/XP/2003 Files
* [xkcd](https://cran.r-project.org/web/packages/xkcd/index.html): Plotting ggplot2 Graphics in an XKCD Style
* [XML](https://cran.r-project.org/web/packages/XML/index.html): Tools for Parsing and Generating XML Within R and S-Plus
* [xts](https://cran.r-project.org/web/packages/xts/index.html): eXtensible Time Series
* [ymlthis](https://cran.r-project.org/web/packages/ymlthis/index.html): Write 'YAML' for 'R Markdown', 'bookdown', 'blogdown', and More

---

| R ÂåÖÂêçÁß∞        | ‰∏ªË¶ÅÂäüËÉΩ                       |
| --------------- | ------------------------------ |
| devtools        | R ÂåÖÂÆâË£Ö                       |
| BiocManager     | R ÂåÖÂÆâË£Ö                       |
| tidyverse       | Êï∞ÊçÆÊï¥ÁêÜÂèäÂèØËßÜÂåñÂ∑•ÂÖ∑ÂåÖÈõÜÂêà     |
| future          | ÊîØÊåÅÂπ∂Ë°åËÆ°ÁÆóÁöÑÂåÖ               |
| future.apply    | ÊîØÊåÅÂπ∂Ë°åËÆ°ÁÆóÁöÑÂåÖ               |
| Seurat          | ÂçïÁªÜËÉûÂàÜÊûêÂü∫Á°ÄÂ∑•ÂÖ∑ÂåÖ           |
| SeuratData      | ÂçïÁªÜËÉûÊï∞ÊçÆ                     |
| SeuratWrappers  | Êèê‰æõÂÖ∂‰ªñ R ÂåÖ‰∏é Seurat ÁöÑÊé•Âè£  |
| SingleR         | ÁªÜËÉûÈâ¥ÂÆö                       |
| batchelor       | ÊâπÊ¨°Ê†°Ê≠£                       |
| harmony         | ÊâπÊ¨°Ê†°Ê≠£                       |
| liger           | ÊâπÊ¨°Ê†°Ê≠£ÔºåÊï∞ÊçÆÊï¥Âêà             |
| limma           | Â∑ÆÂºÇÂàÜÊûê                       |
| edgeR           | Â∑ÆÂºÇÂàÜÊûê                       |
| DESeq2          | Â∑ÆÂºÇÂàÜÊûê                       |
| MAST            | Â∑ÆÂºÇÂàÜÊûê                       |
| GSVA            | Âü∫Âõ†ÈõÜÂèòÂºÇÂàÜÊûê                 |
| msigdbr         | Êèê‰æõÈùû‰∫∫Áâ©ÁßçÁöÑ MSigDB Âü∫Âõ†ÈõÜ   |
| clusterProfiler | Âü∫Âõ†ÈõÜÂØåÈõÜÂàÜÊûê                 |
| scRepertoire    | ÂÖçÁñ´ÁªÑÂ∫ìÂàÜÊûê                   |
| survival        | ÁîüÂ≠òÂàÜÊûê                       |
| survminer       | ÁîüÂ≠òÂàÜÊûê                       |
| CellChat        | ÁªÜËÉûÈÄöËÆØ                       |
| monocle         | ÊãüÊó∂ËΩ®ËøπÂàÜÊûê                   |
| monocle3        | ÊãüÊó∂ËΩ®ËøπÂàÜÊûê                   |
| velocyto.R      | RNA ÈÄüÁéáÂàÜÊûêÔºåÁ±ª‰ººÊãüÊó∂ËΩ®ËøπÂàÜÊûê |
| infercnv        | ÂçïÁªÜËÉû CNV ÂàÜÊûê                |
| nichenetr       | ÁªÜËÉûÈÄöËÆØ                       |
| garnett         | ÁªÜËÉûÂàÜÁ±ª                       |
| DoubletFinder   | ËØÜÂà´ doublets                  |
| WGCNA           | ÂÖ±Ë°®ËææÊùÉÈáçÁΩëÁªúÂàÜÊûê             |
| Signac          | scATAC ÂàÜÊûê                    |
| biomaRt         | Âü∫Âõ† ID ÂèäÂêåÊ∫êÂü∫Âõ†ËΩ¨Êç¢         |
| CytoTRACE       | ËΩ®ËøπÂàÜÊûê                       |
| slingshot       | ËΩ®ËøπÂàÜÊûê                       |
| copykat         | ÂçïÁªÜËÉû CNV ÂàÜÊûê                |
| xCell           | bulkRNA ÂéªÂç∑ÁßØ                 |
| MuSiC           | bulkRNA ÂéªÂç∑ÁßØ                 |
| SCDC            | bulkRNA ÂéªÂç∑ÁßØ                 |
| SPOTlight       | Á©∫Èó¥ËΩ¨ÂΩïÁªÑÂàÜÊûê                 |
| ggpubr          | ÁîªÂõæ                           |

## TeX
* [texshade](https://ctan.org/pkg/texshade): Package for setting nucleotide and peptide alignments

## Perl
* [Data::Dumper](https://metacpan.org/pod/Data::Dumper): stringified perl data structures, suitable for both printing and eval
* [Data::Printer](https://metacpan.org/pod/Data::Printer): colored pretty-print of Perl data structures and objects
* [Getopt::Long](https://metacpan.org/pod/Getopt::Long): Extended processing of command line options
* [List::AllUtils](https://metacpan.org/pod/List::AllUtils): Combines List::Util, List::SomeUtils and List::UtilsBy in one bite-sized package
* [List::MoreUtils](https://metacpan.org/pod/List::MoreUtils): Provide the stuff missing in List::Util 
* [List::Util](https://metacpan.org/pod/List::Util): A selection of general-utility list subroutines
* [List::UtilsBy](https://metacpan.org/pod/List::UtilsBy): higher-order list utility functions 
* [Parallel::ForkManager](https://metacpan.org/pod/Parallel::ForkManager): A simple parallel processing fork manager
* [Path::Tiny](https://metacpan.org/pod/Path::Tiny): File path utility
* [Try::Tiny](https://metacpan.org/pod/Try::Tiny): Minimal try/catch with proper preservation of $@




## Tools
* [bioawk](https://github.com/lh3/bioawk): BWK awk modified for biological data
* [GraPhlAn](https://bitbucket.org/nsegata/graphlan/wiki/Home): a software tool for producing high-quality circular representations of taxonomic and phylogenetic trees
* [IQ-TREE](http://www.iqtree.org/): Efficient software for phylogenomic inference
* [Recharts](http://recharts.org/en-US/): A composable charting library built on React components
* [pyGenomeTracks](https://github.com/deeptools/pyGenomeTracks): Standalone program and library to plot beautiful genome browser tracks
* [miRWalk](http://mirwalk.umm.uni-heidelberg.de/)
* [miRSystem](http://mirsystem.cgm.ntu.edu.tw/): miRNA integration system for Target gene prediction
* [Heatmapper](http://www.heatmapper.ca/expression/): a freely available web server that allows users to interactively visualize their data in the form of heat maps through an easy-to-use graphical interface
* [jvenn](http://jvenn.toulouse.inra.fr/app/index.html): an interactive Venn diagram viewer
* [webtools-Venn](http://bioinformatics.psb.ugent.be/webtools/Venn/): draw venn diagram
* [EVenn](http://www.ehbio.com/test/venn/#/): Easy to create repeatable, editable, and statistically measurable Venn diagrams online.
* [NetworkAnalyst](https://www.networkanalyst.ca/): comprehensive gene expression analysis, meta-analysis & network biology
* [BioJupies](https://amp.pharm.mssm.edu/biojupies/): Automatically Generates RNA-seq Data Analysis Notebooks
* [bioDBnet](https://biodbnet-abcc.ncifcrf.gov/): biological DataBase network
* [iTOL](https://itol.embl.de/): Interactive Tree Of Life
* [ETE](http://etetoolkit.org/treeview/): Phylogenetic tree (newick) viewer, Âú®Á∫øÂèØËßÜÂåñËøõÂåñÊ†ëÂíåÂ§öÂ∫èÂàóÊØîÂØπ
* [easyGEO](https://easygeo.cn/#/): ‰∏ÄÁ´ôÂºè GEO Êï∞ÊçÆÂàÜÊûêÂπ≥Âè∞
* [Quickomics](http://quickomics.bxgenomics.com/): Ë°®ËææÊï∞ÊçÆÂàÜÊûêÂ∑•ÂÖ∑
* [EvolView](https://www.evolgenius.info/evolview/#login): a webserver for visualization, annotation, and management of phylogenetic trees
* [kingfisher-download](https://github.com/wwood/kingfisher-download): Easier download/extract of FASTA/Q read data from the ENA, NCBI, AWS or GCP.
* [PathwayMapper](https://www.pathwaymapper.org/): a collaborative visual web editor for cancer pathways and genomic data
* [kingfisher-download](https://github.com/wwood/kingfisher-download): Easier download/extract of FASTA/Q read data and metadata from the ENA, NCBI, AWS or GCP.
* [parallel-fastq-dump](https://github.com/rvalieris/parallel-fastq-dump): parallel fastq-dump wrapper
* [Refgenie](http://refgenie.databio.org/en/latest/): Refgenie manages storage, access, and transfer of reference genome resources. It provides command-line and Python interfaces to *download* pre-built reference genome "assets", like indexes used by bioinformatics tools. It can also *build* assets for custom genome assemblies. Refgenie provides programmatic access to a standard genome folder structure, so software can swap from one genome to another.

## scRNA
* [**awesome-single-cell**](https://github.com/seandavi/awesome-single-cell)**: Community-curated list of software packages and data resources for single-cell, including RNA-seq, ATAC-seq, etc.**
* [**awesome-multi-omics**](https://github.com/mikelove/awesome-multi-omics)**: List of software packages for multi-omics analysis**
* [**scRNA-tools**](https://www.scrna-tools.org/table): The scRNA-tools database is a catalogue of tools for analysing single-cell RNA sequencing data.
* [scater](https://bioconductor.org/packages/release/bioc/html/scater.html): Single-Cell Analysis Toolkit for Gene Expression Data in R
* [Seurat](https://cran.r-project.org/web/packages/Seurat/index.html): Tools for Single Cell Genomics
* [monocle](https://www.bioconductor.org/packages/release/bioc/html/monocle.html): Clustering, differential expression, and trajectory analysis for single- cell RNA-Seq
* [cellassign](https://github.com/irrationone/cellassign): Automated, probabilistic assignment of cell types in scRNA-seq data
* [DRAGON](https://github.com/arc85/dragonsc): Deterministic annealing gaussian mixtures models for single cell RNAseq
* [singleseqgset](https://github.com/arc85/singleseqgset): Gene set enrichment analysis for single-cell RNAseq data
* [celltalker](https://github.com/arc85/celltalker): Inference of cell-cell communication from single-cell RNAseq data
* [DPT](https://bioinformaticshome.com/tools/rna-seq/descriptions/DPT.html): estimate the temporal order of differentiating cell in single-cell RNA-seq (scRNA-seq) data
* [diffusionMap](https://github.com/rcannood/diffusionMap)
* [destiny](http://www.bioconductor.org/packages/release/bioc/html/destiny.html): Creates diffusion maps
* [iCellR](https://cran.r-project.org/web/packages/iCellR/index.html): Analyzing High-Throughput Single Cell Sequencing Data
* [presto](https://github.com/immunogenomics/presto): Presto performs a fast Wilcoxon rank sum test and auROC analysis
* [harmony](https://github.com/immunogenomics/harmony): Fast, sensitive and accurate integration of single-cell data with Harmony
* [ESTIMATE](https://bioinformatics.mdanderson.org/public-software/estimate/): Estimation of STromal and Immune cells in MAlignant Tumor tissues using Expression data
* [xCell](https://github.com/dviraran/xCell): cell types enrichment analysis
* [SingleR](https://bioconductor.org/packages/release/bioc/html/SingleR.html): Reference-Based Single-Cell RNA-Seq Annotation
* [CIBERSORTx]( impute gene expression profiles and provide an estimation of the abundances of member cell types in a mixed cell population, using gene expression data)
* [CellPhoneDB](https://github.com/Teichlab/cellphonedb): CellPhoneDB can be used to search for a particular ligand/receptor, or interrogate your own single-cell transcriptomics data.
* [clustermole](https://cran.r-project.org/web/packages/clustermole/index.html): Unbiased Single-Cell Transcriptomic Data Cell Type Identification
* [CellChat](https://github.com/sqjin/CellChat/): Inference and analysis of cell-cell communication
* [iTALK](https://github.com/Coolgenome/iTALK): is an R toolkit for characterizing and illustrating intercellular communication
* [SciBet](http://scibet.cancer-pku.cn/): Single Cell Identificator Based on E-test (SciBet) is a computational tool for predicting cell identity for any randomly sequenced cell by single cell RNA sequencing technique
* [CHETAH](https://github.com/jdekanter/CHETAH): a selective, hierarchical cell type identification method for single-cell RNA sequencing
* [dyno](https://github.com/dynverse/dyno): Inferring trajectories with 60 TI methods
* [Scanpy](https://github.com/theislab/scanpy): Single-Cell Analysis in Python
* [singleseqgset](https://github.com/arc85/singleseqgset): package for gene set enrichment analysis for single-cell RNAseq data
* [clustree](https://cran.r-project.org/web/packages/clustree/index.html): Visualise Clusterings at Different Resolutions
* [batchelor](https://bioconductor.org/packages/release/bioc/html/batchelor.html): Single-Cell Batch Correction Methods
* [presto](https://github.com/immunogenomics/presto): Presto performs a fast Wilcoxon rank sum test and auROC analysis. Latest benchmark ran 1 million observations, 1K features, and 10 groups in 16 seconds (sparse input) and 85 seconds (dense input).
* [SoupX](https://cran.r-project.org/web/packages/SoupX/index.html): Single Cell mRNA Soup eXterminator
* [liger](https://github.com/MacoskoLab/liger): Linked Inference of Genomic Experimental Relationships, LIGER (`liger`) is a package for integrating and analyzing multiple single-cell datasets
* [RcisTarget](https://www.bioconductor.org/packages/release/bioc/html/RcisTarget.html): Identify transcription factor binding motifs enriched on a gene list
* [AUCell](http://bioconductor.org/packages/release/bioc/html/AUCell.html): Analysis of 'gene set' activity in single-cell RNA-seq data (e.g. identify cells with specific gene signatures)
* [Giotto](http://spatialgiotto.rc.fas.harvard.edu/): The Giotto package consists of two modules, Giotto Analyzer and Viewer, which provide tools to process, analyze and visualize single-cell spatial expression data. https://rubd.github.io/Giotto_site/index.html
* [BayesSpace](https://bioconductor.org/packages/release/bioc/html/BayesSpace.html): Clustering and Resolution Enhancement of Spatial Transcriptomes
* [spdep](https://cran.r-project.org/web/packages/spdep/index.html): Spatial Dependence: Weighting Schemes, Statistics
* [nichenetr](https://github.com/saeyslab/nichenetr): the R implementation of the NicheNet method
* [SPOTlight](https://github.com/MarcElosua/SPOTlight): The goal of **SPOTlight** is to provide a tool that enables the deconvolution of cell types and cell type proportions present within each capture locations comprising mixtures of cells, originally developed for 10X‚Äôs Visium - spatial trancsiptomics- technology, it can be used for all technologies returning mixtures of cells.
* [iCellR](https://cran.r-project.org/web/packages/iCellR/index.html): Analyzing High-Throughput Single Cell Sequencing Data
* [ncborcherding/escape](https://github.com/ncborcherding/escape): Easy single cell analysis platform for enrichment
* [xCell](https://github.com/dviraran/xCell): cell types enrichment analysis
* [stLearn](https://stlearn.readthedocs.io/en/latest/):  a downstream analysis toolkit for Spatial Transcriptomics data
* [SPATA](https://themilolab.github.io/SPATA/index.html): A Toolbox for Spatial Gene Expression Analysis
* [celldex](https://bioconductor.org/packages/release/data/experiment/html/celldex.html): Reference Index for Cell Types
* [scCancer](https://github.com/wguo-research/scCancer): A package for automated processing of single cell RNA-seq data in cancer
* [UCell](https://github.com/carmonalab/UCell): An R package for single-cell gene signature scoring
* [metacell](https://github.com/tanaylab/metacell): Metacell - Single-cell mRNA Analysis
* [RCTD](https://github.com/dmcable/RCTD): Robust Cell Type Decomposition
* [RCAv2](https://github.com/prabhakarlab/RCAv2): R package for robust clustering of single cell RNA sequencing data
* [seurat-wrappers](https://github.com/satijalab/seurat-wrappers): Community-provided extensions to Seurat
* [seurat-disk](https://github.com/mojaveazure/seurat-disk): Interfaces for HDF5-based Single Cell File Formats
* [scVelo](https://scvelo.readthedocs.io/): RNA velocity generalized through dynamical modeling
* [DeepSEM](https://github.com/HantaoShu/DeepSEM): a deep-learning-based approach with novel neural network architecture that can infer gene regulatory network, embed scRNA-seq data, and simulate realistic scRNA-seq data by interpreting different modules
* [velocyto.R](https://github.com/velocyto-team/velocyto.R): RNA velocity estimation in R
* [SPOTlight](https://github.com/MarcElosua/SPOTlight): Spatial Transcriptomics Capture Location Deconvolution
* [stereoscope](https://github.com/almaan/stereoscope): Spatial mapping of cell types by integration of transcriptomics data
* [ArchR]: Analysis of Regulatory Chromatin in R (www.ArchRProject.com)
* [SnapATAC](https://github.com/r3fang/SnapATAC): Analysis Pipeline for Single Cell ATAC-seq
* [signac](https://github.com/timoast/signac): R toolkit for the analysis of single-cell chromatin data
* [dynplot](https://github.com/dynverse/dynplot): Visualising single-cell trajectories, including comparisons between two models üìà
* [dorothea](https://bioconductor.org/packages/release/data/experiment/html/dorothea.html): Collection Of Human And Mouse TF Regulons

## [Cancer/Survival-related, ÁîüÂ≠òÂàÜÊûêÂú®Á∫øÂ∑•ÂÖ∑ 2020 Âπ¥ 4 ÊúàÂêêË°ÄÊï¥ÁêÜÔºÅ](https://mp.weixin.qq.com/s?__biz=Mzg4NjA5Mzg2Mw==&mid=2247488689&idx=2&sn=92818d38ffba4f97fd295dcf51f05857&chksm=cf9fb5f9f8e83cefc35b6a980c125fe8be1b5860f66211594cb27d7419be02f5adcb95588c56&mpshare=1&scene=1&srcid=&sharer_sharetime=1587258755943&sharer_shareid=49bb68e4d4ad9f65af077f4e54025da0&key=7024fc3958d21a49f38d5962a504384cb7aa02cfec7d99b587fbbaad39d6f014de6891e0d0d0a1c51311259cc5116c5fda4311d0c4a8bf171a88b52cae8649f6495594a0becdfce753717e6f209ed145&ascene=1&uin=MjEyMzUzNDk2MQ%3D%3D&devicetype=Windows+XP&version=62060841&lang=zh_CN&exportkey=Af7QoGlOZcxdXwTntJ6mQfA%3D&pass_ticket=egSQZf3ZmjmNSBYxFfxeMhxks6Fcdfesee2EK45qb6wqbW1FRDSL1ND6vhbrpF2H)
* [Kaplan Meier-plotter](http://kmplot.com/analysis/index.php)
* [PROGgene](http://genomics.jefferson.edu/proggene/)
* [SurvExpress](http://bioinformatica.mty.itesm.mx:8080/Biomatec/SurvivaX.jsp)
* [KM-express](http://ec2-52-201-246-161.compute-1.amazonaws.com/kmexpress/index.php)
* [PrognoScan](http://dna00.bio.kyutech.ac.jp/PrognoScan/index.html)
* [lnCAR: lncRNA Explorer](https://lncar.renlab.org/)
* [UALCAN](http://ualcan.path.uab.edu/analysis.html)
* [OncoLnc](http://www.oncolnc.org/)
* [OncomiR](http://www.oncomir.org)
* [MethSurv](https://biit.cs.ut.ee/methsurv/)
* [BloodSpot](http://servers.binf.ku.dk/bloodspot/)
* [GEPIA 2](http://gepia2.cancer-pku.cn/#survival)
* [GenomicScape](http://www.genomicscape.com/microarray/survival.php)
* [ExSurv](https://exsurv.soic.iupui.edu/)
* [LOGpc](http://bioinfo.henu.edu.cn/DatabaseList.jsp)
* [CBioportal](https://www.cbioportal.org/)
* [UCSC Xena](https://xena.ucsc.edu/)
* [GDC portal](https://portal.gdc.cancer.gov/)
* [SurvMicro](http://bioinformatica.mty.itesm.mx:8080/Biomatec/Survmicro.jsp)
* [DriverDBv3](http://driverdb.tms.cmu.edu.tw/)
* [LncACTdb 2.0](http://www.bio-bigdata.net/LncACTdb/)
* [TCGAportal](http://tumorsurvival.org/index.html)
* [Cancer RNA-Seq Nexus](http://syslab4.nchu.edu.tw/)
* [iCoMut](http://firebrowse.org/iCoMut/): ÁôåÁóáÁöÑsomatic Á™ÅÂèò‰ø°ÊÅØÂÖ®ÊôØÂõæ
* [UALCAN](http://ualcan.path.uab.edu/index.html)

## Shiny-related

* [nanxstats/awesome-shiny-extensions](https://github.com/nanxstats/awesome-shiny-extensions): üêù Awesome R packages that offer extended UI or server components for the R web framework Shiny

ÂèØ‰ª•ËÆ©ÁΩëÈ°µÁúãËµ∑Êù•Êõ¥‰∏ì‰∏öÁöÑ R ÂåÖÔºö

* shinythemes
* dashboardthemes
* shinydashboard
* shinydashboardPlus
* shinyjs
* shinyWidgets
* shinycssloaders
* shinyalert
* shinydisconnect
* shinybusy
* DT
* waiter
* slickR
* plotly
* heatmaply
* htmlwidgets

# Long read sequencing
* [LRS-DB](https://long-read-tools.org/): A catalogue of long read sequencing data analysis tools

# Scholar search
* [Ëô´ÈÉ®ËêΩÊêúÁ¥¢](https://scholar.chongbuluo.com/): ÈõÜÂêà‰∫ÜÂ∏ÇÈù¢‰∏äÊúÄ‰∏∫Â∏∏ËßÅÁöÑÂ≠¶ÊúØÊêúÁ¥¢ÂºïÊìéÔºåËÆ©‰Ω†‰∏ÄÁ´ôÂºèÊêúÂà∞Â∫ï
* [Open-i](https://openi.nlm.nih.gov): ÁæéÂõΩÂõΩÁ´ãÂç´ÁîüÁ†îÁ©∂Èô¢ÂºÄÂèëÁöÑÁßëÁ†îÂõæÁâáÊï∞ÊçÆÂ∫ìÔºåÁîüÁâ©ÂíåÂåªÂ≠¶Á±ªÈ¶ñÊé®ÁöÑÂõæÁâáÊêúÁ¥¢ÁΩëÁ´ô
* [Viziometrics](http://viziometrics.org/search/): The First Visual Search Engine for Scientific Diagrams
* [CNKI Â≠¶ÊúØÂõæÁâáÂ∫ì](http://image.cnki.net/)
* [rcrossref](https://cran.r-project.org/web/packages/rcrossref/index.html): Client for Various 'CrossRef' 'APIs'
* [crminer](https://cran.r-project.org/web/packages/crminer/index.html): Fetch 'Scholary' Full Text from 'Crossref'

# Chrome
* [Ë∞∑Á≤í - Chrome Êèí‰ª∂Ëã±ÈõÑÊ¶ú](https://github.com/zhaoolee/ChromeAppHeroes): ‰∏∫‰ºòÁßÄÁöÑ Chrome Êèí‰ª∂ÂÜô‰∏ÄÊú¨‰∏≠ÊñáËØ¥Êòé‰π¶ÔºåËÆ© Chrome Êèí‰ª∂Ëã±ÈõÑ‰ª¨ÈÄ†Á¶è‰∫∫Á±ª

# Shell

* [starship](https://starship.rs/): The minimal, blazing-fast, and infinitely customizable prompt for any shell!



# Others

* [E Venn](http://www.ehbio.com/test/venn/#/)
* [Visinets](https://mp.weixin.qq.com/s?__biz=MzIwMzA1MzU5Mg==&mid=2652152715&idx=2&sn=24ee771dd63400cf5d84a94f01313fa6&chksm=8d354cfcba42c5ea90e5ec36c7fb75fc1d41e9b471fb93c3f05432acd9fc553aec6ac5268336&mpshare=1&scene=1&srcid=0213XFRKAbcWYa9igdvlKnCM&sharer_sharetime=1613191433944&sharer_shareid=49bb68e4d4ad9f65af077f4e54025da0&key=22735c9487e13afecff1211788e4d10128063c1442d91be53d61ed44a7fbd1a74abf3c3bdfe67590d74965a684fb45241c20309b52a70d34fcb5be5266b0cfa907062a9dd0528aa1922b293d2771056f40d061593fe631f38e1bb453218a0fa3dcc473a3a10746f1eb7d239d072e7ad34c101c3d6a28a16852b1691e003a9f6f&ascene=1&uin=MjEyMzUzNDk2MQ%3D%3D&devicetype=Windows+7&version=62090529&lang=en&exportkey=AcSdXDNN0VcaW0EE2WlGimU%3D&pass_ticket=bdtbHUindyL5pXtSUK8%2Ft0VMzBFMRgoZkxBxHnWZRa6qQ9jOt%2FDXFdHtbLQtDK1b&wx_header=0)Ôºö‰∏Ä‰∏™ÂèØ‰ª•ËÆ©‰Ω†ÁöÑ‰ø°Âè∑ÈÄöË∑ØÂõæÂä®Ëµ∑Êù•ÁöÑÁΩëÁ´ô
* [AutoGluon: AutoML for Text, Image, and Tabular Data](https://github.com/awslabs/autogluon)

---

- MACS2 ËøõË°å Peak calleing
- csaw ËøõË°åÂ∑ÆÂºÇ Peak ÂàÜÊûê
- MEME suite ËøõË°å motif Ê£ÄÊµãÂíåÂØåÈõÜ
- ChIPseeker ËøõË°åÊ≥®ÈáäÂíåÂèØËßÜÂåñ
- HMMRATAC ËøõË°åÊ†∏Â∞è‰ΩìÊ£ÄÊµã
- HINT-ATAC ËøõË°åË∂≥ËøπÂàÜÊûê

# Reproducibility

* [ÊûÑÂª∫ÂèØÈáçÂ§çÁöÑÂçïÁªÜËÉûÊï∞ÊçÆÂàÜÊûêÊµÅÁ®ã](https://www.jianshu.com/p/675335f3fe4a)
* [Reproducibility in Science](http://ropensci.github.io/reproducibility-guide/): A Guide to enhancing reproducibility in scientific results and writing
* [Top ten reasons to not share your code (and why you should anyway)](https://www.software.ac.uk/blog/2016-10-07-top-ten-reasons-not-share-your-code-and-why-you-should-anyway)
* [Does your code stand up to scrutiny?](https://www.nature.com/articles/d41586-018-02741-4)
* [Excuse me, do you have a moment to talk about version control?](https://peerj.com/preprints/3159v2/)
* [[As a reviewer, can I justify requesting data and code be made available even if the journal does not?](https://stats.stackexchange.com/questions/14416/as-a-reviewer-can-i-justify-requesting-data-and-code-be-made-available-even-if)]
* [Table Convert Online - table to markdown,csv,json,latex table,excel,sql,xml](https://tableconvert.com/)

# Single-cell databases

| Êï∞ÊçÆÂ∫ìÂêçÁß∞                   | ÁΩëÂùÄ                                           |
| ---------------------------- | ---------------------------------------------- |
| CellMarker                   | http://biocc.hrbmu.edu.cn/CellMarker/index.jsp |
| Mouse Cell Atlas(MCAÔºâ       | http://bis.zju.edu.cn/MCA/index.html           |
| PanglaoDB                    | https://panglaodb.se/index.html                |
| Single Cell Expression Atlas | https://www.ebi.ac.uk/gxa/sc/home              |
| Single Cell PORTAL           | https://portals.broadinstitute.org/single_cell |
| SCPortalen                   | http://single-cell.clst.riken.jp/              |
| CancerSEA                    | http://biocc.hrbmu.edu.cn/CancerSEA/home.jsp   |
| SpatialDB                    | https://www.spatialomics.org/SpatialDB         |
| Human Cell LandscapeÔºàHCLÔºâ  | http://bis.zju.edu.cn/HCL/                     |